<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.38">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Species trees and delimitation with SVDQuartets, PICL, and DELINEATE – Zosterops Methods and Results</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-b663525e4b92bb903b2199e2d401bf36.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-95a2ced3787879bf47d026b70c7a7c79.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Zosterops Methods and Results</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">Home</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./about.html"> 
<span class="menu-text">Introduction</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./ZosteropsMaps.html"> 
<span class="menu-text">Sampling Maps</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Summary_Sanger.html"> 
<span class="menu-text">Sanger sequencing summary statistics and trees</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Zosterops_StarBEAST2.html"> 
<span class="menu-text">StarBEAST2</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./ZosteropsBPP.html"> 
<span class="menu-text">BPP and GDI based species delimitation</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./ZosteropsSNPs_analysis.html"> 
<span class="menu-text">Distance trees, PCA, UMAP, and DAPC analyses</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./Zosterops_SNPFst.html"> 
<span class="menu-text">Genetic differentiation and biogeographic breaks</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./Zosterops_SVDQ.html" aria-current="page"> 
<span class="menu-text">Species trees and delimitation with SVDQuartets, PICL, and DELINEATE</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./ZosteropsMorphometrics.html"> 
<span class="menu-text">Linear morphometrics</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#goals" id="toc-goals" class="nav-link active" data-scroll-target="#goals">Goals</a></li>
  <li><a href="#getting-started" id="toc-getting-started" class="nav-link" data-scroll-target="#getting-started">Getting started</a>
  <ul class="collapse">
  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data">Data</a></li>
  <li><a href="#software" id="toc-software" class="nav-link" data-scroll-target="#software">Software</a></li>
  <li><a href="#r-packages" id="toc-r-packages" class="nav-link" data-scroll-target="#r-packages">R packages</a></li>
  <li><a href="#data-files" id="toc-data-files" class="nav-link" data-scroll-target="#data-files">Data files</a></li>
  </ul></li>
  <li><a href="#svdquartets" id="toc-svdquartets" class="nav-link" data-scroll-target="#svdquartets">SVDQuartets</a>
  <ul class="collapse">
  <li><a href="#svdquartets-tree-and-bootstraps" id="toc-svdquartets-tree-and-bootstraps" class="nav-link" data-scroll-target="#svdquartets-tree-and-bootstraps">SVDQuartets tree and bootstraps</a></li>
  <li><a href="#svdquartets-consensus-tree" id="toc-svdquartets-consensus-tree" class="nav-link" data-scroll-target="#svdquartets-consensus-tree">SVDQuartets consensus tree</a></li>
  <li><a href="#cleaning-up-the-consensus-tree" id="toc-cleaning-up-the-consensus-tree" class="nav-link" data-scroll-target="#cleaning-up-the-consensus-tree">Cleaning up the consensus tree</a></li>
  </ul></li>
  <li><a href="#picl" id="toc-picl" class="nav-link" data-scroll-target="#picl">PICL</a>
  <ul class="collapse">
  <li><a href="#run-1" id="toc-run-1" class="nav-link" data-scroll-target="#run-1">Run 1</a></li>
  <li><a href="#run-2" id="toc-run-2" class="nav-link" data-scroll-target="#run-2">Run 2</a></li>
  </ul></li>
  <li><a href="#delineate" id="toc-delineate" class="nav-link" data-scroll-target="#delineate">Delineate</a>
  <ul class="collapse">
  <li><a href="#run-1-1" id="toc-run-1-1" class="nav-link" data-scroll-target="#run-1-1">Run 1</a></li>
  <li><a href="#run-2-1" id="toc-run-2-1" class="nav-link" data-scroll-target="#run-2-1">Run 2</a></li>
  <li><a href="#run-3-1" id="toc-run-3-1" class="nav-link" data-scroll-target="#run-3-1">Run 3</a></li>
  <li><a href="#run-4-1" id="toc-run-4-1" class="nav-link" data-scroll-target="#run-4-1">Run 4</a></li>
  <li><a href="#run-5-1" id="toc-run-5-1" class="nav-link" data-scroll-target="#run-5-1">Run 5</a></li>
  </ul></li>
  <li><a href="#speciation-rate" id="toc-speciation-rate" class="nav-link" data-scroll-target="#speciation-rate">Speciation rate</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Species trees and delimitation with SVDQuartets, PICL, and DELINEATE</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="goals" class="level2">
<h2 class="anchored" data-anchor-id="goals">Goals</h2>
<p>Included here are phylogenetic analyses of East Asian Zosterops using the single nucleotide polymorphism (SNP) data from our double digest restriction-site associated DNA (ddRAD) library. The goals of these analyses are to determine species tree topology, estimate branch lengths, and to test hypotheses about species limits.</p>
</section>
<section id="getting-started" class="level2">
<h2 class="anchored" data-anchor-id="getting-started">Getting started</h2>
<p>Along with the software listed below we are using R version 4.4.2 and RStudio version 2024.09.1 for the analysis. Quarto version 1.6.39 for Mac OS was used to document the workflow. Set the working directory to one where the data files are accessible.</p>
<section id="data" class="level3">
<h3 class="anchored" data-anchor-id="data">Data</h3>
<p>The dataset used in these analyses is the <code>zost.filtered.snps.vcf</code> file. The filtered SNP data are in variant call format (VCF) (see DeRaad et al.&nbsp;2024 for details on creation of the ddRAD library, sequencing, and filtering the SNP dataset) <span class="citation" data-cites="deraad2024">(<a href="#ref-deraad2024" role="doc-biblioref">DeRaad et al. 2024</a>)</span>.</p>
</section>
<section id="software" class="level3">
<h3 class="anchored" data-anchor-id="software">Software</h3>
<ul>
<li><p><a href="https://github.com/edgardomortiz/vcf2phylip">vcf2phylip</a> - Python script to convert a <code>vcf</code> file to a <code>phy</code> or <code>nexus</code> formatted file <span class="citation" data-cites="ortiz2019">(<a href="#ref-ortiz2019" role="doc-biblioref">Ortiz 2019</a>)</span>.</p></li>
<li><p><a href="https://paup.phylosolutions.com/">PAUP*4.0</a> - Phylogenetic Analyses Using Parsimony (PAUP) <span class="citation" data-cites="wilgenbusch2003">(<a href="#ref-wilgenbusch2003" role="doc-biblioref">Wilgenbusch and Swofford 2003</a>)</span>.</p></li>
<li><p><a href="https://github.com/j-chou/SVDquartets">SVDQuartets</a> - Singular Value Decomposition Quartets (SVDQuartets), implemented in PAUP*4.0 <span class="citation" data-cites="chifman2014">(<a href="#ref-chifman2014" role="doc-biblioref">Chifman and Kubatko 2014</a>)</span>.</p></li>
<li><p><a href="https://github.com/lkubatko/PICL/?tab=readme-ov-file">PICL</a> - Phylogenetic Inference with Composite Likelihood (PICL) <span class="citation" data-cites="kubatko2025 swofford2023a">(<a href="#ref-kubatko2025" role="doc-biblioref">Kubatko et al. 2025</a>; <a href="#ref-swofford2023a" role="doc-biblioref">Swofford and Kubatko 2023</a>)</span>.</p></li>
<li><p><a href="https://beast.community/figtree">FigTree</a> - Software for editing and visualizing trees.</p></li>
<li><p><a href="https://jeetsukumaran.github.io/delineate/">Delineate</a> - Species delimitation using the multipopulation or multispecies coalescent and explicit models of speciation from a tree with branch lengths <span class="citation" data-cites="sukumaran2021">(<a href="#ref-sukumaran2021" role="doc-biblioref">Sukumaran, Holder, and Knowles 2021</a>)</span>.</p></li>
</ul>
</section>
<section id="r-packages" class="level3">
<h3 class="anchored" data-anchor-id="r-packages">R packages</h3>
<ul>
<li><p>ape - For visualizing and manipulating trees <span class="citation" data-cites="ape">(<a href="#ref-ape" role="doc-biblioref">Paradis and Schliep 2019</a>)</span>.</p></li>
<li><p>dplyr - For manipulating data <span class="citation" data-cites="dplyr">(<a href="#ref-dplyr" role="doc-biblioref">Wickham et al. 2023</a>)</span>.</p></li>
<li><p>gt - For creating tables and figures <span class="citation" data-cites="gt">(<a href="#ref-gt" role="doc-biblioref">Iannone et al. 2024</a>)</span>.</p></li>
<li><p>tidyr - For manipulating data <span class="citation" data-cites="tidyr">(<a href="#ref-tidyr" role="doc-biblioref">Wickham, Vaughan, and Girlich 2024</a>)</span>.</p></li>
<li><p>webshot2 - For creating images to export for tables <span class="citation" data-cites="webshot2">(<a href="#ref-webshot2" role="doc-biblioref">Chang 2025</a>)</span>.</p></li>
</ul>
</section>
<section id="data-files" class="level3">
<h3 class="anchored" data-anchor-id="data-files">Data files</h3>
<p>For use in PAUP*, the <code>vcf</code> file containing the SNP data must first be converted into <code>nexus</code> format. The Python script <code>vcf2phylip.py</code> <span class="citation" data-cites="ortiz2019">(<a href="#ref-ortiz2019" role="doc-biblioref">Ortiz 2019</a>)</span> is used for this conversion. Make sure that Python version 3 is installed before running this script. Execute the following command in the terminal in a directory containing the file <code>zost.filtered.snps.vcf</code>. The <code>-i</code> argument identifies the input file and <code>-n</code> specifies conversion to <code>nexus</code> format.</p>
<pre><code>python3 vcf2phylip.py -i zost.filtered.snps.vcf -n </code></pre>
<p>By default <code>vcf2phylip.py</code> requires at least four samples to be present at any given locus and adds <code>min4</code> to the file name. This setting can be changed with the <code>-m</code> argument but the default value is four. The newly created <code>nexus</code> file will now be called <code>zost.filtered.snps.min4.nexus</code>. Many of the sample names have dashes (-) and underscores(_) and these characters create problems in PAUP* and other software packages. Remove these characters from the file with the following commands executed in the terminal.</p>
<pre><code>sed ’s/_//g’ zost.filtered.snps.min4.nexus &gt; zost.filtered.snps.min4edited.nexus
sed ’s/-//g’ zost.filtered.snps.min4edited.nexus &gt; zost.filtered.snps.min4edited2.nexus</code></pre>
<p>The final file used in subsequent SVDQuartets analyses will be <code>zost.filtered.snps.min4edited2.nexus</code>.</p>
<p>For later analyses using Phylogenetic Inference with Composite Likelihood (PICL) a SNP file in phylip (<code>phy)</code> format will be needed. This conversion of the <code>vcf</code> file to a <code>phy</code> file can be performed by changing the <code>-n</code> argument in <code>vcf2phylip.py</code> to <code>-p</code>, or alternatively you may omit the last argument altogether as <code>vcf2phylip.py</code> converts a <code>vcf</code> file to <code>phy</code> format by default.</p>
<pre><code>python3 vcf2phylip.py -i zost.filtered.snps.vcf</code></pre>
<p>The converted <code>phy</code> file can then be cleaned of any _ and - in the individual sample names by executing the <code>sed ’s/_//g’</code> and <code>sed ’s/-//g’</code> commands in the terminal as described above for the <code>nexus</code> file.</p>
<pre><code>sed ’s/_//g’ zost.filtered.snps.min4.phy &gt; zost.filtered.snps.min4edited.phy
sed ’s/-//g’ zost.filtered.snps.min4edited.phy &gt; zost.filtered.snps.min4edited2.phy</code></pre>
<p>In some formats gaps are designated by -, but do not worry you are eliminating gaps from the data when editing the <code>.nex</code> and <code>.phy</code> files. Gaps are not coded with - in <code>vcf</code> files and will appear in the converted <code>phy</code> and <code>nexus</code> files as N along with any other missing data. The lesson here is to not include these characters in sample names to begin with!</p>
<p>The new <code>phy</code> formatted file will be <code>zost.filtered.snps.min4edited2.phy</code> but the file name will have to be changed later for the PICL analysis (see below).</p>
<p>A taxon file will also be needed to match samples with specific taxa and populations. The taxon file we will use first is <code>ZosteropsTaxa_POPS.nexus</code>. This file identifies individual samples according to taxon and geographic population in order of the sample’s appearance in the <code>zost.filtered.snps.min4.nexus</code> file.</p>
<pre><code>begin sets;
    taxpartition ZosteropsTaxa =
        japonicus_loochooensis_KIKAI : 1 13,
        japonicus_loochooensis_AMAMI : 2 9,
        japonicus_loochooensis_MIYAKOJIMA : 3 10,
        japonicus_loochooensis_IRIOMOTE : 4,
        japonicus_loochooensis_TOKUNOSHIMA : 5 6,
        japonicus_loochooensis_YONAGUNI : 7,
        japonicus_loochooensis_KUMEJIMA : 8,
        japonicus_loochooensis_OKINOERABU : 11 12,
        japonicus_japonicus_GEOJE : 14 15,
        japonicus_japonicus_JEJU : 16-18,
        japonicus_japonicus_HEUKSANDO : 19 20 25,
        japonicus_japonicus_HONSHU : 21-23,
        japonicus_japonicus_SHIKOKU : 24,
        japonicus_alani_HAHAJIMA : 26-28,
        japonicus_alani_ANEJIMA : 29-31,
        japonicus_alani_IWOJIMA : 32,
        japonicus_stejnegeri_KOZUSHIMA : 33-35,
        japonicus_stejnegeri_MIYAKEJIMA : 36-38,
        japonicus_insularis_KAGOSHIMA: 39-40,
        japonicus_HAWAII : 120-124,
        simplex_TAIWAN : 41 104,
        simplex_GUANGXI : 42 43 46-49 111,
        simplex_GUANGDONG : 44 45,
        simplex_GUIZHOU : 108 109 112-114,
        simplex_HUNAN : 110,
        simplex_VIETNAM : 78 79,
        simplex_SINGAPORE : 50 51,
        simplex_LOSANGELES : 105-107, 
        palpebrosus_VIETNAM : 80-82 103,
        meyeni_BATAN : 52 89-91,
        meyeni_SABTANG : 92-95,
        meyeni_LANYU : 53-56,
        montanus_PANAY : 57-60,
        montanus_halconensis_MINDORO : 61,
        montanus_whiteheadi_LUZON : 62-66,
        montanus_vulcani_MINDANAO : 67-70,
        montanus_montanus_SULAWESI : 71, 
        montanus_montanus_JAVA : 87 88,
        montanus_parkesi_PALAWAN : 72 73,
        montanus_pectoralis_NEGROS : 83-86 115,
        montanus_diuatae_MINDANAO : 116,
        everetti_basilanicus_CAMIGUINSUR : 117,
        everetti_basilanicus_MINDANAO : 119,
        everetti_boholensis_SAMAR : 118,
        nigrorum_nigrorum_PANAY : 74,
        nigrorum_meyleri_CAMIGUINNORTE : 99,
        nigrorum_catarmanensis_CAMIGUINSUR : 100,
        nigrorum_innominatus_LUZON : 101,
        nigrorum_luzonicus_LUZON : 102,
        erythropleurus_RUSSIA_b : 75-77, 
        erythropleurus_VIETNAM_nb : 96-98;
end;</code></pre>
</section>
</section>
<section id="svdquartets" class="level2">
<h2 class="anchored" data-anchor-id="svdquartets">SVDQuartets</h2>
<p>Open the terminal (Mac OSX) and use the <code>cd</code> command to navigate to the desired directory where the output will be stored. Data and taxon files should be readily accessible from this directory. From the finder drag the executable file <code>paup4a168_osx</code> into the terminal. This should execute the PAUP* program and now you will be able to run PAUP* commands in the terminal window.</p>
<section id="svdquartets-tree-and-bootstraps" class="level3">
<h3 class="anchored" data-anchor-id="svdquartets-tree-and-bootstraps">SVDQuartets tree and bootstraps</h3>
<p>First create and provide a name for a log file by entering the following command. Add a semicolon after each PAUP* command.</p>
<pre><code>log file=Zosterops_populations_SVDQ.log;</code></pre>
<p>Next load the SNP data from the <code>zost.filtered.snps.min4edited2.nexus</code> file into PAUP* with the <code>exe</code> command. For our run the data and taxon files were in the parent directory in the folder containing the PAUP* output in a folder called <code>SNPdata</code>.</p>
<pre><code>exe ../SNPdata/zost.filtered.snps.min4edited2.nexus;</code></pre>
<p>The output from that command should read as follows.</p>
<pre><code>Processing of file "~/Library/Mobile
Documents/com~apple~CloudDocs/Marshall_University/Research/Zosterops/Kansas collaboration/SNP
species tree analysis /SVDQ/SNPdata/zost.filtered.snps.min4edited2.nexus" begins...

Data read in DNA format

Data matrix has 124 taxa, 15704 characters
Valid character-state symbols: ACGT
Missing data identified by 'N'
Gaps identified by '-'
"Equate" macros in effect:
   R,r ==&gt; {AG}
   Y,y ==&gt; {CT}
   M,m ==&gt; {AC}
   K,k ==&gt; {GT}
   S,s ==&gt; {CG}
   W,w ==&gt; {AT}
   H,h ==&gt; {ACT}
   B,b ==&gt; {CGT}
   V,v ==&gt; {ACG}
   D,d ==&gt; {AGT}

Processing of input file "zost.filtered.snps.min4edited2.nexus" completed.</code></pre>
<p>Load the taxon designations with the <code>exe</code> command.</p>
<pre><code>exe ../SNPdata/ZosteropsTaxa_POPS.nexus;</code></pre>
<p>The following output should follow execution of the taxon file.</p>
<pre><code>Processing of file "~/Library/Mobile
Documents/com~apple~CloudDocs/Marshall_University/Research/Zosterops/Kansas collaboration/SNP
species tree analysis /SVDQ/SNPdata/ZosteropsTaxa_POPS.nexus" begins...

Processing of input file "ZosteropsTaxa_POPS.nexus" completed.</code></pre>
<p>Once the taxon file has been processed, specify which taxa will serve as phylogenetic outgroups. Based on prior studies the best available outgroup for the East Asian and Philippine Zosterops species is <em>Zosterops palpebrosus</em> from South and Southeast Asia. Identify the outgroup samples by the order in which they appear in the <code>zost.filtered.snps.min4edited2.nexus</code> file (80-82, and 103).</p>
<pre><code>outgroup 80-82 103;</code></pre>
<p>This will produce the following output confirming the 4 outgroup samples and the remaining 120 ingroup samples.</p>
<pre><code>Outgroup status changed:
  4 taxa transferred to outgroup
  Total number of taxa now in outgroup = 4
  Number of ingroup taxa = 120</code></pre>
<p>Now we are ready to run SVDQuartets with the <code>svdq</code> command. In the <code>nexus</code> taxon file we named the taxon partitions <code>ZosteropsTaxa</code>. Use this for the <code>taxpartition</code> designation in the <code>svdq</code> command line. We will evaluate all quartets (<code>evalquartets=all</code>), produce 700 bootstrap replicates (<code>bootstrap nreps=700</code>), and provide a name for the resulting tree file (<code>treefile=ZosteropsPOPS_SVDQ.tre</code>). 700 bootstrap replicates takes a very long time even on an M2 max MacbookPro with 96Gb RAM and is probably more than is needed. 200-300 replicates would likely perform just as well. But, we had time so we just waited for the biggest bootstrap procedure we could reasonably afford to do. The SVDQuartets analysis of these data was executed with the following command.</p>
<pre><code>svdq taxpartition=ZosteropsTaxa evalquartets=all bootstrap nreps=700 treefile=ZosteropsPOPS_SVDQ.tre;</code></pre>
<p>The following output will appear in the terminal and be added to the log file upon completion of SVDQuartets.</p>
<pre><code>Analysis using SVDQuartets method

  Number of taxa (lineages) in the analysis = 124
  Tips assigned to species using taxon partition 'ZosteropsTaxa'
  Quartet sampling = exhaustive (8383326 quartets)
  Doing standard bootstrap analysis
    Initial bootstrap seed = generated automatically
    Number of bootstrap replicates = 700
  Tree model = multispecies coalescent (expected rank of flattening matrix for true tree = 10)
  Using 2 parallel threads on 8 processors
  Tree search:
    Quartet assembly algorithm = QFM
    Local search = none
  Writing trees for bootstrap replicates to file: ZosteropsPOPS_SVDQ.tre

Quartet assembly completed:
  Total weight of incompatible quartets = 29398.3173 (11.764%)
  Total weight of compatible quartets   = 220501.6875 (88.236%)
  Time used for QFM = 0.05 sec (CPU time = 0.09 sec)

Tree from SVDQuartets analysis (also stored to tree buffer):

                                                                     /----- japonicus loochooensis KIKAI
                                                                 /---+
                                                                 |   \----- japonicus loochooensis OKINOERAB
                                          /----------------------+
                                          |                      |   /----- japonicus loochooensis AMAMI
                                          |                      \---+
                                          |                          \----- japonicus loochooensis TOKUNOSHI
                                          |
                                          |                      /--------- japonicus loochooensis MIYAKOJIM
                                     /----+                      |
                                     |    |   /------------------+   /----- japonicus loochooensis IRIOMOTE
                                     |    |   |                  \---+
                                     |    |   |                      \----- japonicus loochooensis YONAGUNI
                                     |    |   |
                                     |    |   |                      /----- japonicus loochooensis KUMEJIMA
                                     |    \---+    /-----------------+
                                     |        |    |                 \----- japonicus japonicus SHIKOKU
                                     |        |    |
                                     |        |    |                 /----- japonicus japonicus GEOJE
                                     |        |    |             /---+
                                     |        \----+             |   \----- japonicus japonicus HEUKSANDO
                                     |             |        /----+
                                     |             |    /---+    \--------- japonicus japonicus JEJU
                                /----+             |    |   |
                                |    |             |    |   \-------------- japonicus HAWAII
                                |    |             |    |
                                |    |             \----+            /----- japonicus japonicus HONSHU
                                |    |                  |        /---+
                                |    |                  |        |   \----- japonicus insularis KAGOSHIMA
                                |    |                  \--------+
                                |    |                           |   /----- japonicus stejnegeri KOZUSHIMA
                                |    |                           \---+
                                |    |                               \----- japonicus stejnegeri MIYAKEJIMA
                            /---+    |
                            |   |    |                               /----- japonicus alani HAHAJIMA
                            |   |    |                           /---+
                            |   |    |                           |   \----- japonicus alani ANEJIMA
                            |   |    \---------------------------+
                            |   |                                \--------- japonicus alani IWOJIMA
                       /----+   |
                       |    |   |                                    /----- meyeni BATAN
                       |    |   |                                /---+
                       |    |   |                                |   \----- meyeni SABTANG
                       |    |   \--------------------------------+
                   /---+    |                                    \--------- meyeni LANYU
                   |   |    |
                   |   |    \---------------------------------------------- montanus halconensis MINDORO
                   |   |
                   |   \--------------------------------------------------- montanus whiteheadi LUZON
                   |
                   |                                                 /----- montanus PANAY
                   |                                    /------------+
              /----+                                    |            \----- montanus pectoralis NEGROS
              |    |                                    |
              |    |                                    |            /----- montanus vulcani MINDANAO
              |    |                               /----+        /---+
              |    |                               |    |        |   \----- montanus diuatae MINDANAO
              |    |                               |    |   /----+
              |    |                               |    |   |    |   /----- montanus montanus SULAWESI
              |    |                               |    |   |    \---+
              |    \-------------------------------+    \---+        \----- montanus montanus JAVA
              |                                    |        |
              |                                    |        \-------------- nigrorum catarmanensis CAMIGUINS
              |                                    |
              |                                    \----------------------- montanus parkesi PALAWAN
         /----+
         |    |                                                      /----- simplex TAIWAN
         |    |                                                  /---+
         |    |                                                  |   \----- simplex HUNAN
         |    |                                             /----+
         |    |                                             |    \--------- simplex GUANGDONG
         |    |                                         /---+
         |    |                                         |   |        /----- simplex VIETNAM
         |    |                                    /----+   \--------+
     /---+    |                                    |    |            \----- simplex LOSANGELES
     |   |    |                                    |    |
     |   |    |                               /----+    \------------------ simplex GUIZHOU
     |   |    |                               |    |
     |   |    \-------------------------------+    \----------------------- simplex GUANGXI
     |   |                                    |
     |   |                                    \---------------------------- simplex SINGAPORE
     |   |
/----+   |                                                           /----- erythropleurus RUSSIA b
|    |   \-----------------------------------------------------------+
|    |                                                               \----- erythropleurus VIETNAM nb
|    |
|    |                                                           /--------- everetti basilanicus CAMIGUINSUR
|    |                                                           |
|    |                                                  /--------+   /----- everetti basilanicus MINDANAO
|    |                                                  |        \---+
|    |                                                  |            \----- everetti boholensis SAMAR
|    \--------------------------------------------------+
|                                                       |   /-------------- nigrorum nigrorum PANAY
|                                                       |   |
|                                                       \---+        /----- nigrorum meyleri CAMIGUINNORTE
|                                                           |    /---+
|                                                           |    |   \----- nigrorum innominatus LUZON
|                                                           \----+
|                                                                \--------- nigrorum luzonicus LUZON
|
\-------------------------------------------------------------------------- palpebrosus VIETNAM

Bootstrap 50% majority-rule consensus tree
   (plus other groups compatible with this tree)

                                                                             /------ japonicus loochooensis KIKAI(1)
                                                                       /-40--+
                                                                       |     \------ japonicus loochooensis OKINOERAB(8)
                                                                 /--6--+
                                                                 |     |     /------ japonicus loochooensis AMAMI(2)
                                                                 |     \-37--+
                                                                 |           \------ japonicus loochooensis TOKUNOSHI(5)
                                                     /-----1-----+
                                                     |           |           /------ japonicus loochooensis MIYAKOJIM(3)
                                                     |           |     /-28--+
                                                     |           |     |     \------ japonicus loochooensis KUMEJIMA(7)
                                                     |           \--6--+
                                                     |                 |     /------ japonicus loochooensis IRIOMOTE(4)
                                                     |                 \-37--+
                                                     |                       \------ japonicus loochooensis YONAGUNI(6)
                                                     |
                                               /-10--+                       /------ japonicus japonicus GEOJE(9)
                                               |     |                 /-45--+
                                               |     |                 |     \------ japonicus japonicus HEUKSANDO(11)
                                               |     |           /-38--+
                                               |     |     /-18--+     \------------ japonicus japonicus JEJU(10)
                                               |     |     |     |
                                               |     |     |     \------------------ japonicus HAWAII(20)
                                               |     |     |
                                          /-16-+     \-20--+           /------------ japonicus japonicus HONSHU(12)
                                          |    |           |           |
                                          |    |           |     /-36--+     /------ japonicus stejnegeri KOZUSHIMA(17)
                                          |    |           |     |     \-100-+
                                          |    |           \--9--+           \------ japonicus stejnegeri MIYAKEJIMA(18)
                                          |    |                 |
                                    /-100-+    |                 \------------------ japonicus insularis KAGOSHIMA(19)
                                    |     |    |
                                    |     |    \------------------------------------ japonicus japonicus SHIKOKU(13)
                                    |     |
                                    |     |                                  /------ japonicus alani HAHAJIMA(14)
                                    |     |                            /-68--+
                              /-92--+     |                            |     \------ japonicus alani ANEJIMA(15)
                              |     |     \-------------96-------------+
                              |     |                                  \------------ japonicus alani IWOJIMA(16)
                              |     |
                              |     |                                        /------ meyeni BATAN(30)
                              |     |                                  /-93--+
                        /-91--+     |                                  |     \------ meyeni SABTANG(31)
                        |     |     \---------------100----------------+
                        |     |                                        \------------ meyeni LANYU(32)
                        |     |
                        |     |                                              /------ montanus halconensis MINDORO(34)
                        |     \----------------------53----------------------+
                        |                                                    \------ montanus whiteheadi LUZON(35)
                        |
                        |                                                    /------ montanus PANAY(33)
                  /-100-+                                              /-68--+
                  |     |                                              |     \------ montanus pectoralis NEGROS(40)
                  |     |                                        /-48--+
                  |     |                                        |     \------------ nigrorum catarmanensis CAMIGUINS(47)
                  |     |                                        |
                  |     |                                  /-100-+           /------ montanus vulcani MINDANAO(36)
                  |     |                                  |     |     /-99--+
                  |     |                                  |     |     |     \------ montanus diuatae MINDANAO(41)
                  |     |                                  |     \-65--+
                  |     \----------------92----------------+           |     /------ montanus montanus SULAWESI(37)
                  |                                        |           \-96--+
                  |                                        |                 \------ montanus montanus JAVA(38)
                  |                                        |
            /-100-+                                        \------------------------ montanus parkesi PALAWAN(39)
            |     |
            |     |                                                          /------ simplex TAIWAN(21)
            |     |                                                    /-31--+
            |     |                                                    |     \------ simplex HUNAN(25)
            |     |                                              /-28--+
            |     |                                              |     |     /------ simplex VIETNAM(26)
            |     |                                              |     \-38--+
            |     |                                        /-30--+           \------ simplex LOSANGELES(28)
            |     |                                        |     |
      /-100-+     |                                  /-95--+     \------------------ simplex GUANGDONG(23)
      |     |     |                                  |     |
      |     |     |                            /-100-+     \------------------------ simplex GUIZHOU(24)
      |     |     |                            |     |
      |     |     \------------100-------------+     \------------------------------ simplex GUANGXI(22)
      |     |                                  |
      |     |                                  \------------------------------------ simplex SINGAPORE(27)
      |     |
/-----+     |                                                                /------ erythropleurus RUSSIA b(50)
|     |     \------------------------------100-------------------------------+
|     |                                                                      \------ erythropleurus VIETNAM nb(51)
|     |
|     |                                                                /------------ everetti basilanicus CAMIGUINSUR(42)
|     |                                                                |
|     |                                                    /----100----+     /------ everetti basilanicus MINDANAO(43)
|     |                                                    |           \-61--+
|     |                                                    |                 \------ everetti boholensis SAMAR(44)
|     \------------------------100-------------------------+
|                                                          |     /------------------ nigrorum nigrorum PANAY(45)
|                                                          |     |
|                                                          \-97--+           /------ nigrorum meyleri CAMIGUINNORTE(46)
|                                                                |     /-61--+
|                                                                |     |     \------ nigrorum innominatus LUZON(48)
|                                                                \-89--+
|                                                                      \------------ nigrorum luzonicus LUZON(49)
|
\----------------------------------------------------------------------------------- palpebrosus VIETNAM(29)

Bipartitions found in one or more trees and frequency of occurrence (bootstrap support values):

         111111111122222222223333333333444444444455
123456789012345678901234567890123456789012345678901          Freq        %
--------------------------------------------------------------------------
.................................................**           700  100.00%
.........................................***.......           700  100.00%
.........................................*****.**..           700  100.00%
............................*............*****.**..           700  100.00%
....................******.*.......................           700  100.00%
....................********.......................           700  100.00%
....................*********............*****.****           700  100.00%
....................*******************************           700  100.00%
................................*..***.**.....*....           699   99.86%
............................*............*****.****           699   99.86%
.............................***...................           698   99.71%
................**.................................           697   99.57%
...................................*....*..........           694   99.14%
............................................**.**..           681   97.29%
....................................**.............           675   96.43%
.............***...................................           670   95.71%
....................*.****.*.......................           665   95.00%
.............................**....................           653   93.29%
................................*..******.....*....           647   92.43%
....................*********...*******************           646   92.29%
....................*********...*..****************           636   90.86%
.............................................*.**..           621   88.71%
................................*......*...........           478   68.29%
.............**....................................           474   67.71%
...................................***..*..........           458   65.43%
..........................................**.......           424   60.57%
.............................................*.*...           424   60.57%
.................................**................           371   53.00%
................................*......*......*....           339   48.43%
....................*********...*.*****************           328   46.86%
........*.*........................................           315   45.00%
.******.*******************************************           277   39.57%
.........................*.*.......................           269   38.43%
........***........................................           265   37.86%
...*.*.............................................           257   36.71%
.*..*..............................................           256   36.57%
...........*....**.................................           249   35.57%
....................*...*..........................           218   31.14%
....................*.*.**.*.......................           208   29.71%
...............................................**..           204   29.14%
....................*..***.*.......................           201   28.71%
....................*...**.*.......................           197   28.14%
..*...*............................................           194   27.71%
.........................................*.*.......           190   27.14%
...................................***..*.....*....           183   26.14%
........................**.*.......................           175   25.00%
.............*****.................................           174   24.86%
.............*.*...................................           172   24.57%
.....*..............*******************************           157   22.43%
........................*..*.......................           153   21.86%
........****....****...............................           139   19.86%
.*.************************************************           132   18.86%
........**.........................................           129   18.43%
........***........*...............................           128   18.29%
...................................*....*.....*....           120   17.14%
..*..*.............................................           120   17.14%
......*.....*......................................           117   16.71%
........................**.........................           115   16.43%
...........*......*................................           113   16.14%
.............***....*******************************           112   16.00%
.......................***.*.......................           110   15.71%
..*.*..............................................           108   15.43%
............*.......*******************************           107   15.29%
..................**...............................           106   15.14%
................................*.............*....           101   14.43%
................................*..*...**.....*....           100   14.29%
.*................*................................            98   14.00%
.........**........................................            96   13.71%
.........*.........*...............................            91   13.00%
......................****.*.......................            88   12.57%
.......................................*......*....            87   12.43%
.........................................**........            86   12.29%
........*.........*................................            85   12.14%
......................*..*.........................            83   11.86%
......*...........*................................            82   11.71%
....................*.*............................            77   11.00%
.......................**..........................            76   10.86%
...........*.......*...............................            75   10.71%
.......*....****....*******************************            75   10.71%
............****...................................            74   10.57%
....................*...*..*.......................            72   10.29%
................................*..***.**..........            71   10.14%
..**.*.............................................            69    9.86%
............................................**.*...            67    9.57%
............****....*******************************            67    9.57%
.............................................*..*..            65    9.29%
.*.***.********************************************            63    9.00%
.*......****....****...............................            62    8.86%
...........*....***................................            61    8.71%
........***.......**...............................            59    8.43%
......................*.**.*.......................            58    8.29%
........*..........*...............................            56    8.00%
......................*..*.*.......................            54    7.71%
..............**...................................            54    7.71%
....................*************.*****************            52    7.43%
.**.***********************************************            50    7.14%
......................**...........................            47    6.71%
..*.*.*............................................            47    6.71%
.......*.....***...................................            47    6.71%
................................*.*******.....*....            46    6.57%
...........*....****...............................            46    6.57%
.......*.....***....*******************************            45    6.43%
.....*.......***....*******************************            45    6.43%
........****....**.*...............................            45    6.43%
..**.**............................................            45    6.43%
.*.....*...........................................            44    6.29%
...........**......................................            44    6.29%
...*.*..............*******************************            43    6.14%
.*.*.**********************************************            43    6.14%
..**.**.*******************************************            43    6.14%
..*..**............................................            43    6.14%
....................*...**.........................            42    6.00%
...*..*............................................            42    6.00%
........****......**...............................            42    6.00%
....................*......*.......................            41    5.86%
....................*********...*..***.************            40    5.71%
........**.........*...............................            39    5.57%
.......*....*......................................            38    5.43%
...*......*........................................            37    5.29%
.......................*.*.........................            37    5.29%
.*..*..*...........................................            37    5.29%
...*........*......................................            37    5.29%
....................*....*.*.......................            37    5.29%
.......*............*******************************            36    5.14%
........*.......**.................................            35    5.00%
........*.*........*...............................            35    5.00%

1443 groups at (relative) frequency less than 5% not shown

Bootstrap consensus tree stored to tree buffer (replacing previously stored tree)

Trees from bootstrap replicates written to treefile: ~/Library/Mobile
Documents/com~apple~CloudDocs/Marshall_University/Research/Zosterops/Kansas collaboration/SNP species tree analysis
/SVDQ/FINAL SVDQ QAGE/ZosteropsPOPS_SVDQ.tre

Time used for SVDQuartet analysis = 107:58:45 (CPU time = 215:44:13.8)</code></pre>
</section>
<section id="svdquartets-consensus-tree" class="level3">
<h3 class="anchored" data-anchor-id="svdquartets-consensus-tree">SVDQuartets consensus tree</h3>
<p>Now we will create a consensus tree from the trees in the <code>ZosteropsPOPS_svdq.tre</code> file using the majority rule criterion. Close the current terminal window and start a new terminal session. Use the <code>cd</code> command to select the proper directory and drag the executable file <code>paup4a168_osx</code> into the terminal window. You can now start entering PAUP* commands.</p>
<p>First create a new log file for the consensus tree.</p>
<pre><code>log file=ZosteropsPOPS_SVDQcontree.log;</code></pre>
<p>Next load the bootstrap trees from the SVDQuartets analysis with the following command.</p>
<pre><code>gettrees file=ZosteropsPOPS_SVDQ.tre;</code></pre>
<p>The following will appear after the <code>gettrees</code> command is executed.</p>
<pre><code>Processing TREES block from file: ~/Library/Mobile
Documents/com~apple~CloudDocs/Marshall_University/Research/Zosterops/Kansas collaboration/SNP
species tree analysis /SVDQ/FINAL SVDQ QAGE/ZosteropsPOPS_SVDQ.tre
  Keeping: trees from file (replacing any trees already in memory)

The limit of 100 trees (= 'Maxtrees') has been reached.  Do you want to increase Maxtrees?</code></pre>
<p>Respond to the <code>Do you want to increase Maxtrees?</code> query with <code>Y</code>. After the prompt asking for a new value for the maximum number of bootstrap trees enter <code>700</code>.</p>
<pre><code>Do you want to increase 'Maxtrees'? (Y/n)Y

Enter new value for 'Maxtrees' (200):700</code></pre>
<p>PAUP* will then inquire as to what needs to be done if the maximum number of trees is reached. Enter <code>1</code> for <code>Prompt a new value</code>.</p>
<pre><code>Action if limit is hit:
   (1) Prompt for new value
   (2) Automatically increase by 100 (= AUTOINC)
   (3) Leave unchanged, and don't prompt
(1)&gt;1</code></pre>
<p>The following output will indicate the number of trees read from the file and the time taken to execute this operation.</p>
<pre><code>700 trees read from file

Time used = 15.51 sec (CPU time = 0.06 sec)</code></pre>
<p>Specify the taxonomic outgroup from the taxa on the bootstrapped trees with the following command.</p>
<pre><code>outgroup palpebrosus_VIETNAM;</code></pre>
<p>The outgroup will be confirmed following this command with this output.</p>
<pre><code>Outgroup status changed:
  1 taxon transferred to outgroup
  Total number of taxa now in outgroup = 1
  Number of ingroup taxa = 50</code></pre>
<p>Choose <code>outgroup</code> as the root method.</p>
<pre><code>roottrees rootmethod=outgroup;</code></pre>
<p>The bootstrapped trees will be converted to rooted trees and the following output will appear.</p>
<pre><code>700 trees converted from unrooted to rooted using outgroup method</code></pre>
<p>Finally run the <code>contree</code> operation specifying a 50% majority rule consensus using the <code>majRule</code> method and <code>LE50=yes</code> with the following command. Remember to save the bootstrap support as node labels with <code>saveSupport=nodelabels</code>.</p>
<pre><code>contree / majRule=yes LE50=yes treefile=ZosteropsPOPS_SVDQcontree.tre saveSupport=nodelabels;</code></pre>
<p>This command will produce the following output. Ellipses (…) indicate where the output has been abridged for this document. Groups with a frequency &lt;10% are not shown here.</p>
<pre><code>Strict consensus of 700 trees:

                                                       /----------- japonicus loochooensis KIKAI
                                                       |
                                                       +----------- japonicus loochooensis AMAMI
                                                       |
                                                       +----------- japonicus japonicus GEOJE
                                                       |
                                                       +----------- japonicus japonicus HEUKSANDO
                                                       |
                                                       +----------- japonicus loochooensis KUMEJIMA
                                                       |
                                                       +----------- japonicus japonicus JEJU
                                                       |
                                                       +----------- japonicus HAWAII
                                                       |
                                                       +----------- japonicus japonicus HONSHU
                                                       |
                                                       +----------- japonicus insularis KAGOSHIMA
                                                       |
                                                       +----------- japonicus japonicus SHIKOKU
                                            /----------+
                                            |          +----------- japonicus alani HAHAJIMA
                                            |          |
                                            |          +----------- japonicus alani ANEJIMA
                                            |          |
                                            |          +----------- japonicus alani IWOJIMA
                                            |          |
                                            |          +----------- japonicus stejnegeri KOZUSHIMA
                                            |          |
                                            |          +----------- japonicus stejnegeri MIYAKEJIMA
                                            |          |
                                            |          +----------- japonicus loochooensis IRIOMOTE
                                            |          |
                                            |          +----------- japonicus loochooensis YONAGUNI
                                            |          |
                                            |          +----------- japonicus loochooensis MIYAKOJIM
                                            |          |
                                            |          +----------- japonicus loochooensis TOKUNOSHI
                                            |          |
                                            |          \----------- japonicus loochooensis OKINOERAB
                                            |
                                            +---------------------- montanus PANAY
                                            |
                                 /----------+---------------------- montanus pectoralis NEGROS
                                 |          |
                                 |          +---------------------- montanus vulcani MINDANAO
                                 |          |
                                 |          +---------------------- montanus diuatae MINDANAO
                                 |          |
                                 |          +---------------------- montanus montanus SULAWESI
                                 |          |
                                 |          +---------------------- montanus montanus JAVA
                                 |          |
                                 |          +---------------------- nigrorum catarmanensis CAMIGUINS
                                 |          |
                                 |          +---------------------- montanus parkesi PALAWAN
                                 |          |
                                 |          +---------------------- montanus whiteheadi LUZON
                                 |          |
                                 |          +---------------------- montanus halconensis MINDORO
                                 |          |
                                 |          +---------------------- meyeni BATAN
                                 |          |
                      /----------+          +---------------------- meyeni SABTANG
                      |          |          |
                      |          |          \---------------------- meyeni LANYU
                      |          |
                      |          |                     /----------- simplex TAIWAN
                      |          |                     |
                      |          |                     +----------- simplex HUNAN
                      |          |                     |
                      |          |                     +----------- simplex GUANGDONG
                      |          |                     |
                      |          |          /----------+----------- simplex VIETNAM
                      |          |          |          |
                      |          |          |          +----------- simplex LOSANGELES
                      |          |          |          |
                      |          +----------+          +----------- simplex GUIZHOU
                      |          |          |          |
           /----------+          |          |          \----------- simplex GUANGXI
           |          |          |          |
           |          |          |          \---------------------- simplex SINGAPORE
           |          |          |
           |          |          |                     /----------- erythropleurus RUSSIA b
           |          |          \---------------------+
           |          |                                \----------- erythropleurus VIETNAM nb
           |          |
           |          |                                /----------- everetti basilanicus CAMIGUINSUR
           |          |                                |
           |          |                     /----------+----------- everetti basilanicus MINDANAO
-----------+          |                     |          |
           |          |                     |          \----------- everetti boholensis SAMAR
           |          |                     |
           |          |                     +---------------------- nigrorum nigrorum PANAY
           |          \---------------------+
           |                                +---------------------- nigrorum meyleri CAMIGUINNORTE
           |                                |
           |                                +---------------------- nigrorum innominatus LUZON
           |                                |
           |                                \---------------------- nigrorum luzonicus LUZON
           |
           \------------------------------------------------------- palpebrosus VIETNAM

50% Majority-rule consensus of 700 trees

                                                          /---- japonicus loochooensis KIKAI(1)
                                                      /40-+
                                                      |   \---- japonicus loochooensis OKINOERAB(51)
                                                  /-6-+
                                                  |   |   /---- japonicus loochooensis AMAMI(2)
                                                  |   \37-+
                                                  |       \---- japonicus loochooensis TOKUNOSHI(50)
                                         /---1----+
                                         |        |       /---- japonicus loochooensis KUMEJIMA(5)
                                         |        |   /28-+
                                         |        |   |   \---- japonicus loochooensis MIYAKOJIM(49)
                                         |        \-6-+
                                         |            |   /---- japonicus loochooensis IRIOMOTE(16)
                                         |            \37-+
                                         |                \---- japonicus loochooensis YONAGUNI(17)
                                         |
                                     /10-+                /---- japonicus japonicus GEOJE(3)
                                     |   |            /45-+
                                     |   |            |   \---- japonicus japonicus HEUKSANDO(4)
                                     |   |        /38-+
                                     |   |   /-18-+   \-------- japonicus japonicus JEJU(6)
                                     |   |   |    |
                                     |   |   |    \------------ japonicus HAWAII(7)
                                     |   |   |
                                 /16-+   \20-+        /-------- japonicus japonicus HONSHU(8)
                                 |   |       |        |
                                 |   |       |    /36-+   /---- japonicus stejnegeri KOZUSHIMA(14)
                                 |   |       |    |   \100+
                                 |   |       \-9--+       \---- japonicus stejnegeri MIYAKEJIMA(15)
                                 |   |            |
                             /100+   |            \------------ japonicus insularis KAGOSHIMA(9)
                             |   |   |
                             |   |   \------------------------- japonicus japonicus SHIKOKU(10)
                             |   |
                             |   |                        /---- japonicus alani HAHAJIMA(11)
                             |   |                    /68-+
                         /92-+   |                    |   \---- japonicus alani ANEJIMA(12)
                         |   |   \---------96---------+
                         |   |                        \-------- japonicus alani IWOJIMA(13)
                         |   |
                         |   |                            /---- meyeni BATAN(46)
                         |   |                        /93-+
                     /91-+   |                        |   \---- meyeni SABTANG(47)
                     |   |   \----------100-----------+
                     |   |                            \-------- meyeni LANYU(48)
                     |   |
                     |   |                                /---- montanus whiteheadi LUZON(44)
                     |   \---------------53---------------+
                     |                                    \---- montanus halconensis MINDORO(45)
                     |
                     |                                    /---- montanus PANAY(36)
                 /100+                                /68-+
                 |   |                                |   \---- montanus pectoralis NEGROS(37)
                 |   |                            /48-+
                 |   |                            |   \-------- nigrorum catarmanensis CAMIGUINS(42)
                 |   |                            |
                 |   |                       /100-+       /---- montanus vulcani MINDANAO(38)
                 |   |                       |    |   /99-+
                 |   |                       |    |   |   \---- montanus diuatae MINDANAO(39)
                 |   |                       |    \65-+
                 |   \----------92-----------+        |   /---- montanus montanus SULAWESI(40)
                 |                           |        \96-+
                 |                           |            \---- montanus montanus JAVA(41)
                 |                           |
            /100-+                           \----------------- montanus parkesi PALAWAN(43)
            |    |
            |    |                                        /---- simplex TAIWAN(18)
            |    |                                    /31-+
            |    |                                    |   \---- simplex HUNAN(19)
            |    |                                /28-+
            |    |                                |   |   /---- simplex VIETNAM(21)
            |    |                                |   \38-+
            |    |                           /-30-+       \---- simplex LOSANGELES(22)
            |    |                           |    |
        /100+    |                       /95-+    \------------ simplex GUANGDONG(20)
        |   |    |                       |   |
        |   |    |                   /100+   \----------------- simplex GUIZHOU(23)
        |   |    |                   |   |
        |   |    \--------100--------+   \--------------------- simplex GUANGXI(24)
        |   |                        |
        |   |                        \------------------------- simplex SINGAPORE(25)
        |   |
    /100+   |                                             /---- erythropleurus RUSSIA b(34)
    |   |   \---------------------100---------------------+
    |   |                                                 \---- erythropleurus VIETNAM nb(35)
    |   |
    |   |                                             /-------- everetti basilanicus CAMIGUINSUR(27)
    |   |                                             |
    |   |                                    /--100---+   /---- everetti basilanicus MINDANAO(28)
    |   |                                    |        \61-+
    |   |                                    |            \---- everetti boholensis SAMAR(29)
----+   \----------------100-----------------+
    |                                        |    /------------ nigrorum nigrorum PANAY(30)
    |                                        |    |
    |                                        \-97-+       /---- nigrorum meyleri CAMIGUINNORTE(31)
    |                                             |   /61-+
    |                                             |   |   \---- nigrorum innominatus LUZON(32)
    |                                             \89-+
    |                                                 \-------- nigrorum luzonicus LUZON(33)
    |
    \---------------------------------------------------------- palpebrosus VIETNAM(26)

Groups found in one or more trees and frequency of occurrence:

         1         2         3         4         55
123456789012345678901234567890123456789012345678901          Freq        %
--------------------------------------------------------------------------
..........................***......................           700  100.00%
..........................*******..................           700  100.00%
.................................**................           700  100.00%
.................*******...........................           700  100.00%
.................********..........................           700  100.00%
*****************...............................***           700  100.00%
*****************..................****************           700  100.00%
*************************........******************           700  100.00%
*************************.*************************           700  100.00%
...................................*******.........           699   99.86%
*************************..........****************           699   99.86%
.............................................***...           698   99.71%
.............**....................................           697   99.57%
.....................................**............           694   99.14%
.............................****..................           681   97.29%
.......................................**..........           675   96.43%
..........***......................................           670   95.71%
.................******............................           665   95.00%
.............................................**....           653   93.29%
...................................********........           647   92.43%
*****************............................******           646   92.29%
*****************..........................********           636   90.86%
..............................***..................           621   88.71%
...................................**..............           478   68.29%
..........**.......................................           474   67.71%
.....................................****..........           458   65.43%
...........................**......................           424   60.57%
..............................**...................           424   60.57%
...........................................**......           371   53.00%
...................................**....*.........           339   48.43%
*****************...........................*******           328   46.86%
..**...............................................           315   45.00%
*.................................................*           277   39.57%
....................**.............................           269   38.43%
..**.*.............................................           265   37.86%
...............**..................................           257   36.71%
.*...............................................*.           256   36.57%
.......*.....**....................................           249   35.57%
.................**................................           218   31.14%
.................*****.............................           208   29.71%
...............................**..................           204   29.14%
.................**.***............................           201   28.71%
.................**.**.............................           197   28.14%
....*...........................................*..           194   27.71%
..........................*.*......................           190   27.14%
.....................................*****.........           183   26.14%
..................*.**.............................           175   25.00%
..........*****....................................           174   24.86%
..........*.*......................................           172   24.57%
****************................................***           157   22.43%
..................*..*.............................           153   21.86%
..**.****....**....................................           139   19.86%
*...............................................*..           132   18.86%
..*..*.............................................           129   18.43%
..**.**............................................           128   18.29%
.....................................**..*.........           120   17.14%
................*...............................*..           120   17.14%
....*....*.........................................           117   16.71%
..................*.*..............................           115   16.43%
.......**..........................................           113   16.14%
**********...****...............................***           112   16.00%
..................*.***............................           110   15.71%
................................................**.           108   15.43%
*********.*******...............................***           107   15.29%
......*.*..........................................           106   15.14%
...................................*.....*.........           101   14.43%
...................................****..*.........           100   14.29%
.*......*..........................................            98   14.00%
...*.*.............................................            96   13.71%
.....**............................................            91   13.00%
..................*****............................            88   12.57%
....................................*....*.........            87   12.43%
..........................**.......................            86   12.29%
..*.....*..........................................            85   12.14%
...................**..............................            83   11.86%
....*...*..........................................            82   11.71%
.................*.*...............................            77   11.00%
..................*...*............................            76   10.86%
......**...........................................            75   10.71%
*********....****...............................**.            75   10.71%
.........****......................................            74   10.57%
.................**..*.............................            72   10.29%
...................................******..........            71   10.14%

...

Consensus tree(s) written to treefile: ~/Library/Mobile
Documents/com~apple~CloudDocs/Marshall_University/Research/Zosterops/Kansas collaboration/SNP
species tree analysis /SVDQ/FINAL SVDQ QAGE/ZosteropsPOPS_SVDQcontree.tre</code></pre>
</section>
<section id="cleaning-up-the-consensus-tree" class="level3">
<h3 class="anchored" data-anchor-id="cleaning-up-the-consensus-tree">Cleaning up the consensus tree</h3>
<p>Open the file <code>ZosteropsPOPS_SVDQcontree.tre</code> from the SVDQuartets consensus tree results in <a href="https://beast.community/figtree">FigTree</a> and edit as needed for readability and save as a <code>png</code> file. Below is the resulting consensus SVDQuartets tree based on the 50% majority rule method with bootstrap support shown at the nodes.</p>
<p><img src="images/ZosteropsPOPS_SVDQcontree.png" class="img-fluid" width="1400"></p>
</section>
</section>
<section id="picl" class="level2">
<h2 class="anchored" data-anchor-id="picl">PICL</h2>
<p>Here we will use the software package <a href="https://github.com/lkubatko/PICL/?tab=readme-ov-file">Phylogenetic Inference with Composite Likelihood</a> (PICL, pronounced “pickle”) to generate a species tree with branch lengths from the SNP data using the starting tree topology from SVDQuartets <span class="citation" data-cites="kubatko2025">(<a href="#ref-kubatko2025" role="doc-biblioref">Kubatko et al. 2025</a>)</span>. Additionally, PICL will generate confidence intervals for the speciation times using bootstraps to estimate the reliability of each split on the tree.</p>
<section id="run-1" class="level3">
<h3 class="anchored" data-anchor-id="run-1">Run 1</h3>
<p>Each run of PICL requires the following three files in a directory with a copy of the executable <code>picl</code> program. Create a directory titled <code>PICL/Zosterops1</code> with a copy of the executable <code>picl</code> program and copies of these files, remember that the file names must be the same as listed below.</p>
<ul>
<li><p><code>data.phy</code> - The SNP data in phylip format.</p></li>
<li><p><code>treefile.tre</code> - A treefile in Newick format obtained from a collapsed version of the consensus tree in SVDQuartets (see above). Branch support values are removed and dummy initial branch lengths are added (see below).</p></li>
<li><p><code>settings</code> - A settings file with the run settings and a taxon map assigning individual samples to taxa matching those in the <code>treefile.tre</code>.</p></li>
</ul>
<section id="taxa-and-starting-tree" class="level4">
<h4 class="anchored" data-anchor-id="taxa-and-starting-tree">Taxa and starting tree</h4>
<p>For the first PICL analysis we took the topology from SVDQ and collapsed populations into putative taxa and labeled those taxa as follows with abbreviated taxon names as follows.</p>
<ul>
<li><p><em>Zosterops japonicus</em></p>
<ul>
<li><p><code>japonicus</code> - All <em>Z. japonicus</em> subspecies and populations other than <em>Z. japonicus alani</em> in the Ogasawara and Volcano Islands. This taxon includes individual samples from the Korean Peninsula and adjacent islands of the East China Sea and Korean Strait, the main islands of the Japanese archipelago, the Ryukyu Islands, the Izu Islands, and the introduced population in the Hawaiian archipelago.</p></li>
<li><p><code>alani</code> - Populations of the <em>Z. japonicus alani</em> subspecies in the Ogasawara (Chichijima, Hahajima, Anejima, and Imotojima islands) and Volcano islands (Iwojima).</p></li>
</ul></li>
<li><p><em>Zosterops meyeni</em></p>
<ul>
<li><code>meyeni</code> - Lanyu island off the east coast fo Taiwan and the Batan Islands of the Philippines including Batan and Sabtang islands.</li>
</ul></li>
<li><p><em>Zosterops montanus</em></p>
<ul>
<li><p><code>north</code> - <em>Z. montanus</em> from the Northern Philippine islands of Luzon and Mindoro.</p></li>
<li><p><code>south</code> - <em>Z. montanus</em> from the Visayas in the central Philippines including the main islands of Negros, Panay, Cebu, Bohol, Leyte, and Samar and subspecies/populations from the southern most islands of the Philippines including the island of Mindanao. Included with the central and southern Philippine <em>Z. montanus</em> samples is a <em>Z. nigrorum</em> sample (<code>Zni14341</code>). This sample originating from the island of Camiguin Sur off the northern coast of Mindanao was initially assigned to the <em>Z. nigrorum</em> subspecies <em>catarmanensis</em> but in this study was found to cluster strongly with the central and southern Philippine populations of <em>Z. montanus</em> in both Sanger sequencing and SNP datasets.</p></li>
<li><p><code>palawan</code> - Philippine <em>Z. montanus</em> <em>parkesi</em> population on the island of Palawan.</p></li>
<li><p><code>indonesia</code> - <em>Z. montanus</em> <em>montanus</em> populations of the Indonesian archipelago represented in this study by samples from Java and Sulawesi.</p></li>
</ul></li>
<li><p><em>Zosterops simplex</em></p>
<ul>
<li><code>simplex</code> - All <em>Z. simplex</em> populations either from the island of Taiwan or on mainland continental Asia and introduced populations in the southern California region of North America.</li>
</ul></li>
<li><p>Additional outgroup taxa</p>
<ul>
<li><p><code>erythro</code> - <em>Z. erythropleurus</em> including samples originating in the breeding range in the Russian Far East and wintering individuals in Southeast Asia.</p></li>
<li><p><code>everetti</code> - <em>Z. everetti</em> in the Visayas and Southern Philippines.</p></li>
<li><p><code>nigrorum</code> - <em>Z. nigrorum</em> in the Visayas and Northern Philippines.</p></li>
<li><p><code>palpebro</code> - <em>Z. palpebrosus</em> samples from Vietnam.</p></li>
</ul></li>
</ul>
<p>Because run 1 is using a significantly collapsed tree with few taxa it can easily be entered in Newick format manually. Plot the final collapsed tree for PICL run 1 to see if it is correct. Manually save a copy of this as <code>treefile.tre</code> into the directory where PICL run 1 will be executed. The names of the files for a PICL run need to be specific. Dummy branch lengths will need to be added to this Newick formatted tree for PICL to have starting branch lengths to work from. Those branch lengths are arbitrary but made the final <code>treefile.tre</code> tree roughly clock-like and ultrametric. Below is the final starting tree for the PICL run 1 with the initial branch lengths and the final collapsed topology plotted using <code>ape</code>.</p>
<pre><code>((((((((japonicus:0.1,alani:0.1):0.1,meyeni:0.2):0.1,north:0.3):0.1,((indonesia:0.1,south:0.1):0.1,palawan:0.2):0.2):0.1,simplex:0.5):0.1,erythro:0.6):0.1,(everetti:0.4,nigrorum:0.4):0.3):0.1,palpebro:0.8);</code></pre>
<div class="cell">
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ape)</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Define the Newick string</span></span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>topology <span class="ot">&lt;-</span> <span class="st">"((((((((japonicus:0.1,alani:0.1):0.1,meyeni:0.2):0.1,north:0.3):0.1,((indonesia:0.1,south:0.1):0.1,palawan:0.2):0.2):0.1,simplex:0.5):0.1,erythro:0.6):0.1,(everetti:0.4,nigrorum:0.4):0.3):0.1,palpebro:0.8);"</span></span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tree from the Newick string</span></span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>PICL_topology1 <span class="ot">&lt;-</span> <span class="fu">read.tree</span>(<span class="at">text =</span> topology)</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology1, <span class="at">edge.width =</span> <span class="dv">4</span>, <span class="at">label.offset =</span> <span class="fl">0.02</span>, <span class="at">cex =</span> <span class="fl">1.5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Tree topology for run Zosterops1-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="settings" class="level4">
<h4 class="anchored" data-anchor-id="settings">Settings</h4>
<p>The PICL <code>settings</code> file is a text file that establishes the parameters for a particular run and matches individual samples to assigned groups. This file must be saved in the directory where PICL is to be run and named <code>settings</code>. Arguments for PICL are described in detail at <a href="https://github.com/lkubatko/PICL/blob/main/Documentation.md" class="uri">https://github.com/lkubatko/PICL/blob/main/Documentation.md</a>. The <code>Model</code> prompt specifies the type of data being utilized. 3 is entered for <code>Model</code> to indicate SNP data is being used. <code>Theta</code> is determined empirically for these data beforehand but any value that makes sense probably will do. Multiple runs with different entries for <code>Theta</code> would help the user assess the sensitivity of the analysis to this parameter. The <code>Random_tree</code> parameter set to 0 signifies that we are using a predefined tree included in the <code>treefile.tre</code> file (see above). Additionally, since a tree is provided for this analysis the <code>Tree_search</code> setting is also set to 0. The <code>Num_opt</code> setting should be a large number, 1,000,000 seems to work well for this dataset. Between runs the <code>Seed1</code> and <code>Seed2</code> settings should be set to any arbitrary number to set the random number seeds for the analysis. Following the last setting parameter (<code>Verbose</code>) should be a tally of all the taxa defined in the analysis, followed by the names of those taxa on the next line, and following that a list of the taxa and their associated samples separated by a space (the taxon name should come first). All these parameters should match. The individual sample names should match those in the <code>nexus</code> and <code>phy</code> files converted from the original <code>vcf</code> file. The taxon names associated with each sample should match the list of names directly above the sample list. Make sure the count of taxon names is correct and matches what appears in <code>treefile.tre</code>. It is wise to double (and triple) check these entries before starting a run. Additional details on the parameters in the <code>settings</code> file can be found in the PICL documentation.</p>
<pre><code>Model: 3
Gaps: 1
Bootstrap: 100
Theta: 0.03501696
Rate_param: 1.0
Random_tree: 0
Opt_bl: 1
User_bl: 0
Num_opt: 1000000
Seed1: 12345
Seed2: 67890
Num_cat: 4
Tree_search: 0
Num_iter: 30000
Beta: 0.005
Verbose: 1
12
japonicus alani simplex palpebro meyeni indonesia south north palawan everetti nigrorum erythro
japonicus ZJlo012
japonicus ZJlo015
japonicus ZJlo016
japonicus ZJlo017
japonicus ZJlo021
japonicus ZJlo022
japonicus ZJlo024
japonicus ZJlo031
japonicus ZJlo045
japonicus ZJlo046
japonicus ZJlo050
japonicus ZJlo052
japonicus ZJlo055
japonicus ZJja001
japonicus ZJja002
japonicus ZJja003
japonicus ZJja004
japonicus ZJja005
japonicus ZJja009
japonicus ZJja010
japonicus ZJja012
japonicus ZJja013
japonicus ZJja014
japonicus ZJja016
japonicus ZJja030
alani ZJal003
alani  Zjal004
alani  ZJal005
alani  ZJal010
alani  ZJal011
alani  ZJal012
alani  ZJal020
japonicus ZJst007
japonicus ZJst009
japonicus ZJst010
japonicus ZJst012
japonicus ZJst014
japonicus ZJst015
japonicus ZJin001
japonicus ZJin003
simplex ZJsi002
simplex ZJsi017
simplex ZJsi032
simplex ZJsi023
simplex ZJsi024
simplex ZJsi027
simplex ZJsi028
simplex ZJsi029
simplex ZJsi031
simplex ZPxx001
simplex ZPxx002
meyeni ZMxx002
meyeni ZMxx003
meyeni ZMxx004
meyeni ZMxx005
meyeni ZMxx006
south ZMOxx001
south ZMOxx002
south ZMOxx003
south ZMOxx005
north ZMOha001
north ZMOwh002
north ZMOwh003
north ZMOwh004
north ZMOwh005
north ZMOwh006
south ZMOvu002
south ZMOvu003
south ZMOvu004
south ZMOvu005
south ZMOmo001
palawan ZMOpa001
palawan ZMOpa002
nigrorum ZNni001
erythro ZERxx002
erythro ZERxx004
erythro ZERxx005
simplex Zsim23588
simplex Zsim28142
palpebro Zsim30897
palpebro Zpal23498
palpebro Zpal23522
south Zmon20893
south Zmon20892
south Zmon20902
south Zmon20909
indonesia Zsim31166
indonesia Zsim31159
meyeni Zmey17876
meyeni Zmey17877
meyeni Zmey17852
meyeni Zmey17922
meyeni Zmey17920
meyeni Zmey17925
meyeni Zmey17923
erythro Zery28090
erythro Zery28091
erythro Zery28088
nigrorum Zni10863
south Zni14341
nigrorum Zni19650
nigrorum Zni17984
palpebro ZjaDOT10981
simplex ZjaDOT5235
simplex ZLA1228662
simplex ZLA1225772
simplex ZLA1221882
simplex Zsim13809
simplex Zsim13773
simplex Zsim6797
simplex Zsim10336
simplex Zsim11362
simplex Zsim11102
simplex Zsim11220
south Zmon20899
south Zmon28375
everetti Zev13949
everetti Zev31650
everetti Zev28451
japonicus ZHIBRY431
japonicus ZHINAN290
japonicus ZHIWAI087
japonicus ZHIWAI078
japonicus ZHISOL783</code></pre>
</section>
<section id="run" class="level4">
<h4 class="anchored" data-anchor-id="run">Run</h4>
<p>To run the analysis open a terminal session and <code>cd</code> to the <code>PICL/Zosterops1</code> directory that was created earlier. Enter <code>./picl</code> to run the program. Remember a copy of the <code>picl</code> executable file must be present in this directory along with the three needed data (<code>data.phy</code>) and settings files (<code>settings</code> and <code>treefile.tre</code>). After executing the <code>./picl</code> command a summary of the settings should appear in the terminal followed by a list of the taxon designations and the number of individual samples in each. An alert will follow stating <code>Estimation will be carried out using the SNP model</code> followed by a summary of the <code>data.phy</code> file which should contain 124 lineages and 15704 sites. The tree will be read from the <code>treefile.tre</code> file and a summary showing the dummy branch lengths will follow. Once the tree is successfully read a composite likelihood for the initial tree will be calculated and a revised list of branch lengths and a new composite likelihood will follow. The program will state that the tree has been written to the file <code>outtree.tre</code> and the bootstrapping reps will follow. A running tally of the bootstrap reps will appear as each replicate bootstrap run is completed. 100 bootstrap replicates under these settings may take a while (up to several hours or more) depending on the size of the starting tree.</p>
</section>
<section id="results" class="level4">
<h4 class="anchored" data-anchor-id="results">Results</h4>
<p>Three files will be produced following a PICL run.The <code>results</code> file contains the branch lengths in the final estimated tree if <code>Verbose</code> is set to 1 (this is primarily used for debugging and simulation studies, and will not be useful for most users). The <code>outtree.tre</code> file should contain two trees in Newick format with branch lengths in mutation units (substitutions per site per generation) in the first tree and coalescent units (generations in units of twice the effective population size, 2N<sub>e</sub>) for the second tree. The <code>boot.dat</code> file stores the results of the bootstrapping replicates. This file should have n-1 columns corresponding to the internal nodes of the tree where n equals the number of taxa designated in the <code>settings</code> file.</p>
<p>Numbering the tips and internal nodes is a little tricky. Numbering begins consecutively with the tips and is followed by the numbers for the internal nodes. Because there are 12 taxa on this particular tree the internal nodes are numbered beginning at the root with 13.</p>
<p>Using the ape package in R read the <code>outtree.tre</code> upon completion of the PICL run and plot both the tree in mutation units and the tree in coalescent units. Add the correct node labels to the tree. You can make sure they make sense by matching up with the speciation time estimates from the bootstraps (see below).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the outtree file from PICL</span></span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>Zosterops1_outtree <span class="ot">&lt;-</span> <span class="fu">read.tree</span>(<span class="st">"Data/PICL/Zosterops1/outtree.tre"</span>)</span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>node_labels1 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">13</span><span class="sc">:</span><span class="dv">23</span>)</span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the mutation units [1] and coalescent units [2]. </span></span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops1_outtree[[<span class="dv">1</span>]], <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.001</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">1</span>]]<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="fl">0.05</span>), <span class="at">labels =</span> <span class="cn">FALSE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"0.1 substitutions per site per generation"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">1</span>]]<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">2.7</span>)</span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Add labels to the internal nodes of the first tree with an offset</span></span>
<span id="cb30-10"><a href="#cb30-10" aria-hidden="true" tabindex="-1"></a>n_internal_nodes1 <span class="ot">&lt;-</span> <span class="fu">Nnode</span>(Zosterops1_outtree[[<span class="dv">1</span>]])  <span class="co"># Number of internal nodes</span></span>
<span id="cb30-11"><a href="#cb30-11" aria-hidden="true" tabindex="-1"></a><span class="fu">nodelabels</span>(node_labels1[<span class="dv">1</span><span class="sc">:</span>n_internal_nodes1], </span>
<span id="cb30-12"><a href="#cb30-12" aria-hidden="true" tabindex="-1"></a>            <span class="at">node =</span> (<span class="fu">Ntip</span>(Zosterops1_outtree[[<span class="dv">1</span>]]) <span class="sc">+</span> <span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">Ntip</span>(Zosterops1_outtree[[<span class="dv">1</span>]]) <span class="sc">+</span> n_internal_nodes1), </span>
<span id="cb30-13"><a href="#cb30-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">frame =</span> <span class="st">"none"</span>, <span class="at">cex =</span> <span class="fl">0.8</span>, <span class="at">adj =</span> <span class="fu">c</span>(<span class="fl">1.1</span>, <span class="fl">1.2</span>))  <span class="co"># Adjust adj for positioning</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops1 PICL run-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops1_outtree[[<span class="dv">2</span>]], <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.035</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="fl">1.5</span>), <span class="at">labels =</span> <span class="cn">FALSE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"3 coalescent units (2Ne)"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">2.7</span>)</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Add labels to the internal nodes of the second tree with an offset</span></span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>n_internal_nodes2 <span class="ot">&lt;-</span> <span class="fu">Nnode</span>(Zosterops1_outtree[[<span class="dv">2</span>]])  <span class="co"># Number of internal nodes</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a><span class="fu">nodelabels</span>(node_labels1[<span class="dv">1</span><span class="sc">:</span>n_internal_nodes2], </span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>            <span class="at">node =</span> (<span class="fu">Ntip</span>(Zosterops1_outtree[[<span class="dv">2</span>]]) <span class="sc">+</span> <span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">Ntip</span>(Zosterops1_outtree[[<span class="dv">2</span>]]) <span class="sc">+</span> n_internal_nodes2), </span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>            <span class="at">frame =</span> <span class="st">"none"</span>, <span class="at">cex =</span> <span class="fl">0.8</span>, <span class="at">adj =</span> <span class="fu">c</span>(<span class="fl">1.1</span>, <span class="fl">1.2</span>))  <span class="co"># Adjust adj for positioning</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops1 PICL run-2.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>Finally calculate the 95% confidence intervals for each node (speciation or split time) from the <code>boot.dat</code> file. Again, the <code>boot.dat</code> file should have no header and n-1 columns and 100 rows (or whatever value was set in the settings file at the <code>Bootstrap</code> prompt for the number of bootstrap replicates) of numerical estimates of speciation times measured in coalescent units. After calculating the means and 95% confidence interval for each column use the <code>gt</code> package to save the results to a table.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load necessary libraries</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'dplyr'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:ape':

    where</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:stats':

    filter, lag</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union</code></pre>
</div>
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gt)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'gt'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:ape':

    where</code></pre>
</div>
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb40-2"><a href="#cb40-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(webshot2)</span>
<span id="cb40-3"><a href="#cb40-3" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">webshot2.chrome =</span> <span class="st">"/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"</span>)</span>
<span id="cb40-4"><a href="#cb40-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-5"><a href="#cb40-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the text file</span></span>
<span id="cb40-6"><a href="#cb40-6" aria-hidden="true" tabindex="-1"></a>PICLbootstraps1 <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">'Data/PICL/Zosterops1/boot.dat'</span>, <span class="at">header =</span> <span class="cn">FALSE</span>)</span>
<span id="cb40-7"><a href="#cb40-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-8"><a href="#cb40-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Add a header line labeled 1-11</span></span>
<span id="cb40-9"><a href="#cb40-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(PICLbootstraps1) <span class="ot">&lt;-</span> <span class="fu">as.character</span>(<span class="dv">13</span><span class="sc">:</span><span class="dv">23</span>)</span>
<span id="cb40-10"><a href="#cb40-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-11"><a href="#cb40-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Initialize vectors to store means and confidence intervals</span></span>
<span id="cb40-12"><a href="#cb40-12" aria-hidden="true" tabindex="-1"></a>means <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps1))</span>
<span id="cb40-13"><a href="#cb40-13" aria-hidden="true" tabindex="-1"></a>lower_ci <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps1))</span>
<span id="cb40-14"><a href="#cb40-14" aria-hidden="true" tabindex="-1"></a>upper_ci <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps1))</span>
<span id="cb40-15"><a href="#cb40-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-16"><a href="#cb40-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate means and 95% confidence intervals for each column</span></span>
<span id="cb40-17"><a href="#cb40-17" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(PICLbootstraps1)) {</span>
<span id="cb40-18"><a href="#cb40-18" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Remove NA values. NA values should not appear in the bootstraps however. This would mean there is a problem with the code that needs to be investigated. </span></span>
<span id="cb40-19"><a href="#cb40-19" aria-hidden="true" tabindex="-1"></a>    <span class="co"># This is just a precautionary alert for a very beta software.</span></span>
<span id="cb40-20"><a href="#cb40-20" aria-hidden="true" tabindex="-1"></a>    column_data <span class="ot">&lt;-</span> PICLbootstraps1[[i]]</span>
<span id="cb40-21"><a href="#cb40-21" aria-hidden="true" tabindex="-1"></a>    column_data <span class="ot">&lt;-</span> column_data[<span class="sc">!</span><span class="fu">is.na</span>(column_data)]</span>
<span id="cb40-22"><a href="#cb40-22" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb40-23"><a href="#cb40-23" aria-hidden="true" tabindex="-1"></a>    n <span class="ot">&lt;-</span> <span class="fu">length</span>(column_data)  <span class="co"># Sample size</span></span>
<span id="cb40-24"><a href="#cb40-24" aria-hidden="true" tabindex="-1"></a>    means[i] <span class="ot">&lt;-</span> <span class="fu">mean</span>(column_data)  <span class="co"># Sample mean</span></span>
<span id="cb40-25"><a href="#cb40-25" aria-hidden="true" tabindex="-1"></a>    sd_value <span class="ot">&lt;-</span> <span class="fu">sd</span>(column_data)  <span class="co"># Sample standard deviation</span></span>
<span id="cb40-26"><a href="#cb40-26" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb40-27"><a href="#cb40-27" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate the t-value for 95% confidence</span></span>
<span id="cb40-28"><a href="#cb40-28" aria-hidden="true" tabindex="-1"></a>    t_value <span class="ot">&lt;-</span> <span class="fu">qt</span>(<span class="fl">0.975</span>, <span class="at">df =</span> n <span class="sc">-</span> <span class="dv">1</span>)  <span class="co"># 0.975 for two-tailed test</span></span>
<span id="cb40-29"><a href="#cb40-29" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb40-30"><a href="#cb40-30" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate the margin of error</span></span>
<span id="cb40-31"><a href="#cb40-31" aria-hidden="true" tabindex="-1"></a>    margin_of_error <span class="ot">&lt;-</span> t_value <span class="sc">*</span> (sd_value <span class="sc">/</span> <span class="fu">sqrt</span>(n))</span>
<span id="cb40-32"><a href="#cb40-32" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb40-33"><a href="#cb40-33" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate lower and upper confidence intervals</span></span>
<span id="cb40-34"><a href="#cb40-34" aria-hidden="true" tabindex="-1"></a>    lower_ci[i] <span class="ot">&lt;-</span> means[i] <span class="sc">-</span> margin_of_error</span>
<span id="cb40-35"><a href="#cb40-35" aria-hidden="true" tabindex="-1"></a>    upper_ci[i] <span class="ot">&lt;-</span> means[i] <span class="sc">+</span> margin_of_error</span>
<span id="cb40-36"><a href="#cb40-36" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb40-37"><a href="#cb40-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-38"><a href="#cb40-38" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a data frame for the results</span></span>
<span id="cb40-39"><a href="#cb40-39" aria-hidden="true" tabindex="-1"></a>results1 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb40-40"><a href="#cb40-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">Column =</span> <span class="dv">13</span><span class="sc">:</span><span class="dv">23</span>,</span>
<span id="cb40-41"><a href="#cb40-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean =</span> means,</span>
<span id="cb40-42"><a href="#cb40-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">Lower_CI =</span> lower_ci,</span>
<span id="cb40-43"><a href="#cb40-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">Upper_CI =</span> upper_ci</span>
<span id="cb40-44"><a href="#cb40-44" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb40-45"><a href="#cb40-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-46"><a href="#cb40-46" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a gt table</span></span>
<span id="cb40-47"><a href="#cb40-47" aria-hidden="true" tabindex="-1"></a>results_table1 <span class="ot">&lt;-</span> results1 <span class="sc">%&gt;%</span></span>
<span id="cb40-48"><a href="#cb40-48" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb40-49"><a href="#cb40-49" aria-hidden="true" tabindex="-1"></a>    <span class="fu">tab_header</span>(</span>
<span id="cb40-50"><a href="#cb40-50" aria-hidden="true" tabindex="-1"></a>        <span class="at">title =</span> <span class="st">"Zosterops1 Mean and 95% CIs for speciation times"</span></span>
<span id="cb40-51"><a href="#cb40-51" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span></span>
<span id="cb40-52"><a href="#cb40-52" aria-hidden="true" tabindex="-1"></a>    <span class="fu">cols_label</span>(</span>
<span id="cb40-53"><a href="#cb40-53" aria-hidden="true" tabindex="-1"></a>        <span class="at">Column =</span> <span class="st">"Node"</span>,</span>
<span id="cb40-54"><a href="#cb40-54" aria-hidden="true" tabindex="-1"></a>        <span class="at">Mean =</span> <span class="st">"Mean"</span>,</span>
<span id="cb40-55"><a href="#cb40-55" aria-hidden="true" tabindex="-1"></a>        <span class="at">Lower_CI =</span> <span class="st">"Lower CI"</span>,</span>
<span id="cb40-56"><a href="#cb40-56" aria-hidden="true" tabindex="-1"></a>        <span class="at">Upper_CI =</span> <span class="st">"Upper CI"</span></span>
<span id="cb40-57"><a href="#cb40-57" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span></span>
<span id="cb40-58"><a href="#cb40-58" aria-hidden="true" tabindex="-1"></a>    <span class="fu">fmt_number</span>(</span>
<span id="cb40-59"><a href="#cb40-59" aria-hidden="true" tabindex="-1"></a>        <span class="at">columns =</span> <span class="fu">vars</span>(Mean, Lower_CI, Upper_CI),</span>
<span id="cb40-60"><a href="#cb40-60" aria-hidden="true" tabindex="-1"></a>        <span class="at">decimals =</span> <span class="dv">8</span></span>
<span id="cb40-61"><a href="#cb40-61" aria-hidden="true" tabindex="-1"></a>    )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb42"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Display the table</span></span>
<span id="cb42-2"><a href="#cb42-2" aria-hidden="true" tabindex="-1"></a>results_table1</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="zmirjtjclo" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#zmirjtjclo table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#zmirjtjclo thead, #zmirjtjclo tbody, #zmirjtjclo tfoot, #zmirjtjclo tr, #zmirjtjclo td, #zmirjtjclo th {
  border-style: none;
}

#zmirjtjclo p {
  margin: 0;
  padding: 0;
}

#zmirjtjclo .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#zmirjtjclo .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#zmirjtjclo .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#zmirjtjclo .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#zmirjtjclo .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zmirjtjclo .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zmirjtjclo .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#zmirjtjclo .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#zmirjtjclo .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#zmirjtjclo .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#zmirjtjclo .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#zmirjtjclo .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#zmirjtjclo .gt_spanner_row {
  border-bottom-style: hidden;
}

#zmirjtjclo .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#zmirjtjclo .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#zmirjtjclo .gt_from_md > :first-child {
  margin-top: 0;
}

#zmirjtjclo .gt_from_md > :last-child {
  margin-bottom: 0;
}

#zmirjtjclo .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#zmirjtjclo .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#zmirjtjclo .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#zmirjtjclo .gt_row_group_first td {
  border-top-width: 2px;
}

#zmirjtjclo .gt_row_group_first th {
  border-top-width: 2px;
}

#zmirjtjclo .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zmirjtjclo .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#zmirjtjclo .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#zmirjtjclo .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zmirjtjclo .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#zmirjtjclo .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#zmirjtjclo .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#zmirjtjclo .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#zmirjtjclo .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#zmirjtjclo .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zmirjtjclo .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zmirjtjclo .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#zmirjtjclo .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#zmirjtjclo .gt_left {
  text-align: left;
}

#zmirjtjclo .gt_center {
  text-align: center;
}

#zmirjtjclo .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#zmirjtjclo .gt_font_normal {
  font-weight: normal;
}

#zmirjtjclo .gt_font_bold {
  font-weight: bold;
}

#zmirjtjclo .gt_font_italic {
  font-style: italic;
}

#zmirjtjclo .gt_super {
  font-size: 65%;
}

#zmirjtjclo .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#zmirjtjclo .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#zmirjtjclo .gt_indent_1 {
  text-indent: 5px;
}

#zmirjtjclo .gt_indent_2 {
  text-indent: 10px;
}

#zmirjtjclo .gt_indent_3 {
  text-indent: 15px;
}

#zmirjtjclo .gt_indent_4 {
  text-indent: 20px;
}

#zmirjtjclo .gt_indent_5 {
  text-indent: 25px;
}

#zmirjtjclo .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#zmirjtjclo div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="4" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops1 Mean and 95% CIs for speciation times</th>
</tr>
<tr class="gt_col_headings even">
<th id="Column" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Node</th>
<th id="Mean" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Mean</th>
<th id="Lower_CI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Lower CI</th>
<th id="Upper_CI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Upper CI</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_right" headers="Column">13</td>
<td class="gt_row gt_right" headers="Mean">4.00483643</td>
<td class="gt_row gt_right" headers="Lower_CI">3.98403036</td>
<td class="gt_row gt_right" headers="Upper_CI">4.02564251</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">14</td>
<td class="gt_row gt_right" headers="Mean">3.69477670</td>
<td class="gt_row gt_right" headers="Lower_CI">3.67557376</td>
<td class="gt_row gt_right" headers="Upper_CI">3.71397965</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">15</td>
<td class="gt_row gt_right" headers="Mean">2.39617261</td>
<td class="gt_row gt_right" headers="Lower_CI">2.38282684</td>
<td class="gt_row gt_right" headers="Upper_CI">2.40951837</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">16</td>
<td class="gt_row gt_right" headers="Mean">1.76974043</td>
<td class="gt_row gt_right" headers="Lower_CI">1.75950535</td>
<td class="gt_row gt_right" headers="Upper_CI">1.77997551</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">17</td>
<td class="gt_row gt_right" headers="Mean">0.05512250</td>
<td class="gt_row gt_right" headers="Lower_CI">0.05218192</td>
<td class="gt_row gt_right" headers="Upper_CI">0.05806309</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">18</td>
<td class="gt_row gt_right" headers="Mean">0.00000058</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000045</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000071</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">19</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">20</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">21</td>
<td class="gt_row gt_right" headers="Mean">0.02672204</td>
<td class="gt_row gt_right" headers="Lower_CI">0.02198279</td>
<td class="gt_row gt_right" headers="Upper_CI">0.03146129</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">22</td>
<td class="gt_row gt_right" headers="Mean">0.00000033</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000024</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000043</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">23</td>
<td class="gt_row gt_right" headers="Mean">0.79131947</td>
<td class="gt_row gt_right" headers="Lower_CI">0.78042892</td>
<td class="gt_row gt_right" headers="Upper_CI">0.80221002</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb43"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(results_table1, <span class="st">"Zosterops1PICL_mean_CIss.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-2" class="level3">
<h3 class="anchored" data-anchor-id="run-2">Run 2</h3>
<p>Create another directory named <code>PICL/Zosterops2</code> and copy to that directory the executable <code>picl</code> program and the following three files, remember the file names need to be specific.</p>
<ul>
<li><p><code>data.phy</code> - The same SNP data in phylip format used in run 1.</p></li>
<li><p><code>treefile.tre</code> - A treefile in Newick format with support values removed and dummy initial branch lengths added. For run 2 the starting tree will include additional taxa based on the support values from the SVDQ 50% majority consensus tree. To make sure there are no mistakes in creating this tree, the workflow for carefully collapsing branches will be done using the <code>ape</code> package (see below).</p></li>
<li><p><code>settings</code> - The settings file includes the same parameters as in run 1 except new random numbers are selected for the <code>Seed1</code> and <code>Seed2</code> settings and new taxon assignments for the individual samples are needed to match the new <code>treefile.tre</code> file.</p></li>
</ul>
<section id="taxa-and-starting-tree-1" class="level4">
<h4 class="anchored" data-anchor-id="taxa-and-starting-tree-1">Taxa and starting tree</h4>
<p>For the second PICL analysis we took the topology from SVDQuartets and collapsed populations where bootsrap support for a split was less than 60% (mostly, see below). We defined these lineages as putative taxa and labeled those taxa as follows with abbreviated taxon names as follows.</p>
<ul>
<li><p><em>Zosterops japonicus</em></p>
<ul>
<li><p><code>japonicus</code> - Within the subspecies/populations of <em>Z. japonicus</em> in the Ryukyus and the main islands of Japan and their nearby islands only the <em>Z. japonicus stejnegeri</em> populations on Kozushima and Miyakejima in the Izu Islands exhibited bootstrap support greater than 60%. Because these sister populations of <em>Z. japonicus</em> are deeply embedded in a grouping of very poorly supported lineages, all these continental island populations of <em>Z. japonicus</em>, along with those Izu Island populations of <em>Z. japonicus</em>, were collapsed into a single “continental” clade.</p></li>
<li><p><code>alani1</code> - The far flung oceanic populations of <em>Z. japonicus alani</em> in the Ogasawara and Volcano Islands in the Pacific enjoy high support from all the “continental” populations of <em>Z. japonicus</em> in the Ryukyu Islands, Izu Islands, and the main islands of Japan (Honshu, Hokkaido, Shikoku, and Kyushu), and from one another in the SVDQuartets tree. The population of <em>Z. japonicus alani</em> on Hahajima is assigned the <code>alani1</code> designation.</p></li>
<li><p><code>alani2</code> - <em>Z. japonicus alani</em> from Anejima.</p></li>
<li><p><code>alani3</code> - The most geographically isolated population of <em>Z. japonicus alani</em> from the Volcano Island of Iwojima (Iōtō).</p></li>
</ul></li>
<li><p><em>Zosterops meyeni</em></p>
<ul>
<li><p><code>meyeni1</code> - The northernmost population of <em>Z. meyeni</em> in this study collected on the island of Lanyu near Taiwan.</p></li>
<li><p><code>meyeni2</code> - <em>Z. meyeni</em> population on the island of Sabtang in the Batanes Islands in the Luzon Strait.</p></li>
<li><p><code>meyeni3</code> - <em>Z meyeni</em> population on the island of Batan in the Batanes Islands in the Luzon Strait.</p></li>
</ul></li>
<li><p><em>Zosterops montanus</em></p>
<ul>
<li><p><code>montanus1</code> - <em>Z. montanus</em> subspecies in the Northern Philippines including <em>Z. montanus whiteheadi</em> on the island of Luzon and <em>Z. montanus halconensis</em> on Mindoro.</p></li>
<li><p><code>montanus2</code> - <em>Z. montanus</em> on the island of Panay represents a hypothesized but as of yet unnamed subspecies.</p></li>
<li><p><code>montanus3</code> - <em>Z. montanus pectoralis</em> on the island of Negros in the Visayas in the Central Philippines.</p></li>
<li><p><code>montanus4</code> - <em>Z. montanus vulcani</em> in the southern part of the island of Mindanao.</p></li>
<li><p><code>montanus5</code> - <em>Z. montanus diuatae</em> in the northern and northeastern parts of Mindanao.</p></li>
<li><p><code>montanus6</code> - <em>Z. montanus montanus</em> on the Indonesian island of Sulawesi.</p></li>
<li><p><code>montanus7</code>- <em>Z. montanus montanus</em> on the Indonesian island of Java.</p></li>
<li><p><code>montanus8</code> - <em>Z. montanus parkesi</em> on the Philippine island of Palawan.</p></li>
</ul></li>
<li><p><em>Zosterops simplex</em></p>
<ul>
<li><p><code>simplex1</code> - <em>Z. simplex</em> from Singapore.</p></li>
<li><p><code>simplex2</code> - <em>Z. simplex</em> from the Chinese province of Guangxi.</p></li>
<li><p><code>simplex3</code> - A clade of poorly supported <em>Z. simplex</em> populations (all with &lt;40% support) including those from Vietnam, the island of Taiwan, the Chinese provinces of Guangdong, Guizhou and Hunan, and the introduced population in Los Angeles, California, USA.</p></li>
</ul></li>
<li><p>Additional outgroup taxa</p>
<ul>
<li><p><code>erythro1</code> - <em>Z. erythropleurus</em> collected from the breeding range in the Russian Far East.</p></li>
<li><p><code>erythro2</code> - <em>Z. erythropleurus</em> collected from the winter, non-breeding range in Vietnam.</p></li>
<li><p><code>everetti1</code> - <em>Z. everetti basilanicus</em> from the island of Camiguin Sur off the north coast of Mindanao.</p></li>
<li><p><code>everetti2</code> - <em>Z. everetti basilanicus</em> from the Philippine island of Mindanao.</p></li>
<li><p><code>everetti3</code> - <em>Z. everetti boholensis</em> from Philippine island of Samar.</p></li>
<li><p><code>nigrorum1</code> - <em>Z. nigrorum nigrorum</em> from the Philippine island of Panay.</p></li>
<li><p><code>nigrorum2</code> - <em>Z. nigrorum meyleri</em> from the Philippine island of Camiguin Norte off the north coast of Luzon.</p></li>
<li><p><code>nigrorum3</code> - <em>Z. nigrorum innominatus</em> from east-central Luzon.</p></li>
<li><p><code>nigrorum4</code>- <em>Z. nigrorum luzonicus</em> from the south-east Luzon.</p></li>
<li><p><code>nigrorum5</code> - <em>Z. nigrorum catarmanensis</em> from the Philippine island of Camiguin Sur off the north coast of Mindanao. This taxon is represented by a single sample (<code>Zni14341</code>) that is consistently mismatched in every analysis with the <em>Z. montanus</em> clade instead of with <em>Z. nigrorum</em>, usually allied with those populations from either the central or southern Philippines.</p></li>
<li><p><code>palpebro</code> - <em>Z. palpebrosus</em> from Vietnam.</p></li>
</ul></li>
</ul>
<p>Below is the full Zosterops topology from SVDQuartets. The next steps may seem overly tedious but it is good to carefully ensure the right branches are being collapsed. What follows is a step wise procedure for achieving this using the <code>ape</code> package. Only branches with bootstrap values below 60% are being collapsed with two exceptions.</p>
<p>The <em>Z. japonicus stejnegeri</em> populations on the islands of Kozushima an Miyakejima are sister lineages with near 100% bootstrap support, but they are nested in a group of 15 other populations with extremely poor support. For this reason they were lumped into a single “continental” clade of <em>Z. japonicus</em> that includes all <em>Z. japonicus</em> populations except for the most far flung oceanic populations of <em>Z. japonicus alani</em> on the Ogasawara and Volcano Islands.</p>
<p>Also, <em>Z. nigrorum catarmanensis</em> from the Philippine island of Camiguin Sur is strongly nested within a clade of <em>Z. montanus</em> from the central and southern Philippines. Within this clade however it is positioned with poor support (48%) with a branch containing <em>Z. montanus pectoralis</em> from Negros and an unnamed population of <em>Z. montanus</em> from Panay. Because of this unexpected mismatch we kept <em>Z. nigrorum catarmanensis</em> as distinct in the taxon assignments from populations of <em>Z. montanus</em>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb44"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1" aria-hidden="true" tabindex="-1"></a>PICL_topology2 <span class="ot">&lt;-</span> <span class="fu">read.tree</span>(<span class="st">"Data/SVDQ/ZosteropsPOPS_SVDQtopology.tre"</span>)</span>
<span id="cb44-2"><a href="#cb44-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2, <span class="at">label.offset =</span> <span class="fl">0.1</span>, <span class="at">cex =</span> <span class="fl">0.7</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Read full tree for PICL analysis-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>From the root of the tree collapse nodes with relatively poor support (&lt;60, with exceptions noted above) starting with the <em>Zosterops japonicus</em> clade. Do this by removing all but one taxon for the group to be collapsed and then rename the remaining group with the name assigned to the collapsed group, <code>japonicus</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Starting from the root of the tree collapse all the nodes with branch support &lt;60 starting with the japonicus clade. Include the well supported (99.57) split between the Z. japonicus stejnegeri populations on Miyakejima and Kozushima. This is the only well supported split in the entire continental japonicus clade. Collapse all "continental" japonicus into a single branch. </span></span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>ZOSTEROPS_collapse1 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"japonicus_insularis_KAGOSHIMA"</span>, </span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_japonicus_GEOJE"</span>, </span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_japonicus_HEUKSANDO"</span>, </span>
<span id="cb45-5"><a href="#cb45-5" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_japonicus_JEJU"</span>, </span>
<span id="cb45-6"><a href="#cb45-6" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_HAWAII"</span>, </span>
<span id="cb45-7"><a href="#cb45-7" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_japonicus_HONSHU"</span>, </span>
<span id="cb45-8"><a href="#cb45-8" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_stejnegeri_KOZUSHIMA"</span>, </span>
<span id="cb45-9"><a href="#cb45-9" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_stejnegeri_MIYAKEJIMA"</span>,</span>
<span id="cb45-10"><a href="#cb45-10" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_YONAGUNI"</span>,</span>
<span id="cb45-11"><a href="#cb45-11" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_IRIOMOTE"</span>,</span>
<span id="cb45-12"><a href="#cb45-12" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_YONAGUNI"</span>,</span>
<span id="cb45-13"><a href="#cb45-13" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_MIYAKOJIMA"</span>,</span>
<span id="cb45-14"><a href="#cb45-14" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_KUMEJIMA"</span>,</span>
<span id="cb45-15"><a href="#cb45-15" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_TOKUNOSHIMA"</span>,</span>
<span id="cb45-16"><a href="#cb45-16" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_AMAMI"</span>,</span>
<span id="cb45-17"><a href="#cb45-17" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_OKINOERABU"</span>,</span>
<span id="cb45-18"><a href="#cb45-18" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"japonicus_loochooensis_KIKAI"</span>)</span>
<span id="cb45-19"><a href="#cb45-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-20"><a href="#cb45-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "japonicus_japonicus_SHIKOKU" tip to "japonicus"</span></span>
<span id="cb45-21"><a href="#cb45-21" aria-hidden="true" tabindex="-1"></a>PICL_topology2<span class="sc">$</span>tip.label[PICL_topology2<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"japonicus_japonicus_SHIKOKU"</span>] <span class="ot">&lt;-</span> <span class="st">"japonicus"</span></span>
<span id="cb45-22"><a href="#cb45-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-23"><a href="#cb45-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop the specified tips</span></span>
<span id="cb45-24"><a href="#cb45-24" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1 <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(PICL_topology2, ZOSTEROPS_collapse1)</span>
<span id="cb45-25"><a href="#cb45-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-26"><a href="#cb45-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb45-27"><a href="#cb45-27" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed1, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed1"</span>)</span>
<span id="cb45-28"><a href="#cb45-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb45-29"><a href="#cb45-29" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb45-30"><a href="#cb45-30" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Collapse the Zosterops japonicus clade (minus Z. japonicus alani)-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>Because all the <em>Z. japonicus alani</em> populations are supported &gt;60% simply rename each population as <code>alani1</code> for <em>Z. zosterops alani</em> from Hahajima, <code>alani2</code> for the population on Anejima, and <code>alani3</code> for the population on Iwojima (Iōtō). The same process will be performed for the <em>Z. meyeni</em> clade since those three populations are also relatively well supported on the SVQD tree. The <em>Z. meyeni</em> population form Lanyu will become <code>meyeni1</code>, the population of <em>Z. meyeni</em> on Sabtang will be renamed <code>meyeni2</code>, and the <em>Z. meyeni</em> population on Batan will now be <code>meyeni3</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># All the groups in both the Z. japonicus alani clade and the Z. meyeni clade are supported &gt;60. Just rename these taxa. </span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "japonicus_alani_HAHAJIMA" tip to "alani1"</span></span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"japonicus_alani_HAHAJIMA"</span>] <span class="ot">&lt;-</span> <span class="st">"alani1"</span></span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "japonicus_alani_ANEJIMA" tip to "alani2"</span></span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"japonicus_alani_ANEJIMA"</span>] <span class="ot">&lt;-</span> <span class="st">"alani2"</span></span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "japonicus_alani_IWOJIMA" tip to "alani3"</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"japonicus_alani_IWOJIMA"</span>] <span class="ot">&lt;-</span> <span class="st">"alani3"</span></span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a><span class="co"># The meyeni populations all ave strong support on the SVDQ tree (&gt;90) so don't need further collapsing. </span></span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"meyeni_LANYU"</span>] <span class="ot">&lt;-</span> <span class="st">"meyeni1"</span></span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"meyeni_SABTANG"</span>] <span class="ot">&lt;-</span> <span class="st">"meyeni2"</span></span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed1<span class="sc">$</span>tip.label[PICL_topology2_collapsed1<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"meyeni_BATAN"</span>] <span class="ot">&lt;-</span> <span class="st">"meyeni3"</span></span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-14"><a href="#cb46-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Move to the next tree file</span></span>
<span id="cb46-15"><a href="#cb46-15" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed2 <span class="ot">&lt;-</span> PICL_topology2_collapsed1</span>
<span id="cb46-16"><a href="#cb46-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-17"><a href="#cb46-17" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb46-18"><a href="#cb46-18" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed2, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed2"</span>)</span>
<span id="cb46-19"><a href="#cb46-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-20"><a href="#cb46-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb46-21"><a href="#cb46-21" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Rename the Zosterops japonicus alani and meyeni clades-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>Collapse the northern subspecies of Z. montanus in the Philippines, <em>Z. montanus halconensis</em> and <em>Z. montanus whiteheadi</em>, into a group renamed <code>montanus1</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Collapse the Northern Philippine subspecies of whiteheadi and halconensis into a single clade called montanus1.</span></span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a>ZOSTEROPS_collapse3 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"montanus_whiteheadi_LUZON"</span>)</span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_halconensis_MINDORO" tip to "montanus1"</span></span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed2<span class="sc">$</span>tip.label[PICL_topology2_collapsed2<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_halconensis_MINDORO"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus1"</span></span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-7"><a href="#cb47-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop the specified tips</span></span>
<span id="cb47-8"><a href="#cb47-8" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3 <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(PICL_topology2_collapsed2, ZOSTEROPS_collapse3)</span>
<span id="cb47-9"><a href="#cb47-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-10"><a href="#cb47-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb47-11"><a href="#cb47-11" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed3, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed3"</span>)</span>
<span id="cb47-12"><a href="#cb47-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-13"><a href="#cb47-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb47-14"><a href="#cb47-14" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Collapse the Zosterops montanus nothern Philippine clade-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>Collapse the southern Philippine <em>Z. montanus</em> populations. Keep the mismatched <em>Z. nigrorum catarmanensis</em> and rename as <code>nigrorum5</code> and also maintain each <em>Z. montanus</em> population as a separate group; <code>montanus2</code> (Panay), <code>montanus3</code> (Negros), <code>montanus4</code> (Southern Mindanao), <code>montanus5</code> (Northern Mindanao), <code>montanus6</code> (Sulawesi), <code>montanus7</code> (Java), and <code>montanus8</code> (Palawan).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb48"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Rename the Z. montanus groups. No collapses. </span></span>
<span id="cb48-2"><a href="#cb48-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-3"><a href="#cb48-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_PANAY" tip to "montanus2"</span></span>
<span id="cb48-4"><a href="#cb48-4" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_PANAY"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus2"</span></span>
<span id="cb48-5"><a href="#cb48-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_pectoralis_NEGROS" tip to "montanus3"</span></span>
<span id="cb48-6"><a href="#cb48-6" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_pectoralis_NEGROS"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus3"</span></span>
<span id="cb48-7"><a href="#cb48-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "nigrorum_catarmanensis_CAMIGUINSUR" tip to "nigrorum5"</span></span>
<span id="cb48-8"><a href="#cb48-8" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum_catarmanensis_CAMIGUINSUR"</span>] <span class="ot">&lt;-</span> <span class="st">"nigrorum5"</span></span>
<span id="cb48-9"><a href="#cb48-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_vulcani_MINDANAO" tip to "montanus4"</span></span>
<span id="cb48-10"><a href="#cb48-10" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_vulcani_MINDANAO"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus4"</span></span>
<span id="cb48-11"><a href="#cb48-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_diuatae_MINDANAO" tip to "montanus5"</span></span>
<span id="cb48-12"><a href="#cb48-12" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_diuatae_MINDANAO"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus5"</span></span>
<span id="cb48-13"><a href="#cb48-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_montanus_SULAWESI" tip to "montanus6"</span></span>
<span id="cb48-14"><a href="#cb48-14" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_montanus_SULAWESI"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus6"</span></span>
<span id="cb48-15"><a href="#cb48-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_montanus_JAVA" tip to "montanus7"</span></span>
<span id="cb48-16"><a href="#cb48-16" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_montanus_JAVA"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus7"</span></span>
<span id="cb48-17"><a href="#cb48-17" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "montanus_parkesi_PALAWAN" tip to "montanus8"</span></span>
<span id="cb48-18"><a href="#cb48-18" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed3<span class="sc">$</span>tip.label[PICL_topology2_collapsed3<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"montanus_parkesi_PALAWAN"</span>] <span class="ot">&lt;-</span> <span class="st">"montanus8"</span></span>
<span id="cb48-19"><a href="#cb48-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-20"><a href="#cb48-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Move to the next tree file</span></span>
<span id="cb48-21"><a href="#cb48-21" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4 <span class="ot">&lt;-</span> PICL_topology2_collapsed3</span>
<span id="cb48-22"><a href="#cb48-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-23"><a href="#cb48-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb48-24"><a href="#cb48-24" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed4, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed4"</span>)</span>
<span id="cb48-25"><a href="#cb48-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-26"><a href="#cb48-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb48-27"><a href="#cb48-27" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed4)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Rename the Zosterops montanus clades in the Visayas, Mindanao, Palawan, and Indonesia-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>The <em>Z. simplex</em> populations from Singapore and Guangxi each are assigned their own group, <code>simplex1</code> and <code>simplex2</code>, respectively. Lump the poorly supported <em>Z. simplex</em> populations in Guizhou, Guangdong, Hunan, Taiwan, Vietnam, and the introduced population from Los Angeles, California into a single group named <code>simplex3</code>. Also rename the <em>Z. erythropleurus</em> groups from the breeding range in the Russian far east and the non-breeding samples from Vietnam <code>erythro1</code> and <code>erythro2</code>, respectively.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Collapse and rename the Z. simplex clade. </span></span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>ZOSTEROPS_collapse4 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"simplex_GUANGDONG"</span>,</span>
<span id="cb49-3"><a href="#cb49-3" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"simplex_LOSANGELES"</span>,</span>
<span id="cb49-4"><a href="#cb49-4" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"simplex_VIETNAM"</span>,</span>
<span id="cb49-5"><a href="#cb49-5" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"simplex_HUNAN"</span>,</span>
<span id="cb49-6"><a href="#cb49-6" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"simplex_TAIWAN"</span>)</span>
<span id="cb49-7"><a href="#cb49-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-8"><a href="#cb49-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "simplex_GUIZHOU" tip to "simplex3"</span></span>
<span id="cb49-9"><a href="#cb49-9" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4<span class="sc">$</span>tip.label[PICL_topology2_collapsed4<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"simplex_GUIZHOU"</span>] <span class="ot">&lt;-</span> <span class="st">"simplex3"</span></span>
<span id="cb49-10"><a href="#cb49-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "simplex_GUANGXI" tip to "simplex2"</span></span>
<span id="cb49-11"><a href="#cb49-11" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4<span class="sc">$</span>tip.label[PICL_topology2_collapsed4<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"simplex_GUANGXI"</span>] <span class="ot">&lt;-</span> <span class="st">"simplex2"</span></span>
<span id="cb49-12"><a href="#cb49-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "simplex_SINGAPORE" tip to "simplex1"</span></span>
<span id="cb49-13"><a href="#cb49-13" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4<span class="sc">$</span>tip.label[PICL_topology2_collapsed4<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"simplex_SINGAPORE"</span>] <span class="ot">&lt;-</span> <span class="st">"simplex1"</span></span>
<span id="cb49-14"><a href="#cb49-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "erythropleurus_VIETNAM_nb" tip to "erythro2"</span></span>
<span id="cb49-15"><a href="#cb49-15" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4<span class="sc">$</span>tip.label[PICL_topology2_collapsed4<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"erythropleurus_VIETNAM_nb"</span>] <span class="ot">&lt;-</span> <span class="st">"erythro2"</span></span>
<span id="cb49-16"><a href="#cb49-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "erythropleurus_RUSSIA_b" tip to "erythro1"</span></span>
<span id="cb49-17"><a href="#cb49-17" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed4<span class="sc">$</span>tip.label[PICL_topology2_collapsed4<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"erythropleurus_RUSSIA_b"</span>] <span class="ot">&lt;-</span> <span class="st">"erythro1"</span></span>
<span id="cb49-18"><a href="#cb49-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-19"><a href="#cb49-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop the specified tips</span></span>
<span id="cb49-20"><a href="#cb49-20" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5 <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(PICL_topology2_collapsed4, ZOSTEROPS_collapse4)</span>
<span id="cb49-21"><a href="#cb49-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-22"><a href="#cb49-22" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb49-23"><a href="#cb49-23" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed5, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed5"</span>)</span>
<span id="cb49-24"><a href="#cb49-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-25"><a href="#cb49-25" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb49-26"><a href="#cb49-26" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed5)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Collapse the Zosterops simplex and rename erythropleurus clades, for simplex collapse clades with <60 bootstrap support.-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>The branch support for each of the popluations of <em>Z. everetti</em> and <em>Z. nigrorum</em> (minus the mismatched <em>Z. nigroum catarmanensis</em>, see above) were all &gt;60%. No collapsing occurred in this clade but instead each was simply renamed with a more convenient name for further analyses. <em>Z. everetti</em> <em>baslianicus</em> from Camiguin Sur and Mindanao were renamed <code>everetti1</code> and <code>everetti2</code>, respectively, and <em>Z. everetti boholensis</em> from Samar was renamed <code>everetti3</code>. The nominate <em>Z. nigrorum</em> subspecies on Panay was renamed <code>nigrorum1</code>, the <em>Z. nigrorum meyleri</em> subspecies on Camiguin Norte was renamed <code>nigrorum2</code>, <em>Z. nigrorum innominatus</em> was renamed <code>nigrorum3</code>, and <em>Z. nigrorum luzonicus</em> was renamed <code>nigrorum4</code>. The mismatched <em>Z. nigrorum catarmanensis</em> on Camiguin Sur was already renamed <code>nigrorum5</code> in a previous step. Finally, change the name of the <em>Z. palpebrosus</em> outgroup to <code>palpebro</code>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb50"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Rename the Z. everetti and Z. nigrorum clades and Z. palpebrosus. All the banches in this clade however have bootstrap support &gt;60 so no collapses in this clade. </span></span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-3"><a href="#cb50-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "everetti_basilanicus_CAMIGUINSUR" tip to "everetti1"</span></span>
<span id="cb50-4"><a href="#cb50-4" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"everetti_basilanicus_CAMIGUINSUR"</span>] <span class="ot">&lt;-</span> <span class="st">"everetti1"</span></span>
<span id="cb50-5"><a href="#cb50-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "everetti_basilanicus_MINDANAO" tip to "everetti2"</span></span>
<span id="cb50-6"><a href="#cb50-6" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"everetti_basilanicus_MINDANAO"</span>] <span class="ot">&lt;-</span> <span class="st">"everetti2"</span></span>
<span id="cb50-7"><a href="#cb50-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "everetti_boholensis_SAMAR" tip to "everetti3"</span></span>
<span id="cb50-8"><a href="#cb50-8" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"everetti_boholensis_SAMAR"</span>] <span class="ot">&lt;-</span> <span class="st">"everetti3"</span></span>
<span id="cb50-9"><a href="#cb50-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "nigrorum_nigrorum_PANAY" tip to "nigrorum1"</span></span>
<span id="cb50-10"><a href="#cb50-10" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum_nigrorum_PANAY"</span>] <span class="ot">&lt;-</span> <span class="st">"nigrorum1"</span></span>
<span id="cb50-11"><a href="#cb50-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "nigrorum_meyleri_CAMIGUINNORTE" tip to "nigrorum2"</span></span>
<span id="cb50-12"><a href="#cb50-12" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum_meyleri_CAMIGUINNORTE"</span>] <span class="ot">&lt;-</span> <span class="st">"nigrorum2"</span></span>
<span id="cb50-13"><a href="#cb50-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "nigrorum_innominatus_LUZON" tip to "nigrorum3"</span></span>
<span id="cb50-14"><a href="#cb50-14" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum_innominatus_LUZON"</span>] <span class="ot">&lt;-</span> <span class="st">"nigrorum3"</span></span>
<span id="cb50-15"><a href="#cb50-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "nigrorum_luzonicus_LUZON" tip to "nigrorum4"</span></span>
<span id="cb50-16"><a href="#cb50-16" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum_luzonicus_LUZON"</span>] <span class="ot">&lt;-</span> <span class="st">"nigrorum4"</span></span>
<span id="cb50-17"><a href="#cb50-17" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of the "palpebrosus_VIETNAM" tip to "palpebro" just to an abbreviated taxon name</span></span>
<span id="cb50-18"><a href="#cb50-18" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed5<span class="sc">$</span>tip.label[PICL_topology2_collapsed5<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"palpebrosus_VIETNAM"</span>] <span class="ot">&lt;-</span> <span class="st">"palpebro"</span></span>
<span id="cb50-19"><a href="#cb50-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-20"><a href="#cb50-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Move to the next tree file</span></span>
<span id="cb50-21"><a href="#cb50-21" aria-hidden="true" tabindex="-1"></a>PICL_topology2_collapsed6 <span class="ot">&lt;-</span> PICL_topology2_collapsed5</span>
<span id="cb50-22"><a href="#cb50-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-23"><a href="#cb50-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb50-24"><a href="#cb50-24" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(PICL_topology2_collapsed6, <span class="at">file =</span> <span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed6"</span>)</span>
<span id="cb50-25"><a href="#cb50-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-26"><a href="#cb50-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Optionally, plot the resulting tree</span></span>
<span id="cb50-27"><a href="#cb50-27" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology2_collapsed6)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Rename the everetti and nigrorum clades where bootstrap support.-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
<p>Plot the final collapsed tree for PICL run 2 from the file <code>PICL_topology2_collapse6</code> to check and see if it is correct. Manually save a copy of this as <code>treefile.tre</code> into the directory where PICL run 2 will be executed (<code>PICL/Zosterops2</code> in this analysis). Dummy branch lengths are added manually to this Newick formatted file for PICL to have starting branch lengths to work from. Those branch lengths were arbitrary but made the final <code>treefile.tre</code> tree roughly clock-like and ultrametric. This however does not matter for the algorithm but it just makes the tree look prettier when it is plotted out. Below is the final starting tree for the PICL run 2 with the initial branch lengths and the final collapsed topology plotted using <code>ape</code>.</p>
<pre><code>((((((((japonicus:0.3,((alani1:0.1,alani2:0.1):0.1,alani3:0.2):0.1):0.1,((meyeni3:0.1,meyeni2:0.1):0.1,meyeni1:0.2):0.2):0.1,montanus1:0.5):0.3,((((montanus2:0.1,montanus3:0.1):0.1,nigrorum5:0.2):0.2,((montanus4:0.1,montanus5:0.1):0.1,(montanus6:0.1,montanus7:0.1):0.1):0.2):0.1,montanus8:0.5):0.3):0.3,((simplex3:0.1,simplex2:0.1):0.3,simplex1:0.4):0.7):0.6,(erythro1:0.1,erythro2:0.1):1.6):0.1,((everetti1:0.2,(everetti2:0.1,everetti3:0.1):0.1):0.2,(nigrorum1:0.3,((nigrorum2:0.1,nigrorum3:0.1):0.1,nigrorum4:0.2):0.1):0.1):1.4):0.8,palpebro:2.6);</code></pre>
<div class="cell">
<div class="sourceCode cell-code" id="cb52"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Display topology for PICL tree from Zosterops2 run remember that the tree file used in the PICL run will have the dummy branch lengths. </span></span>
<span id="cb52-2"><a href="#cb52-2" aria-hidden="true" tabindex="-1"></a>PICL_topology <span class="ot">&lt;-</span> <span class="fu">read.tree</span>(<span class="st">"Data/PICL/ZosteropsTrees/PICL_topology2_collapsed6"</span>)</span>
<span id="cb52-3"><a href="#cb52-3" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(PICL_topology, <span class="at">label.offset =</span> <span class="fl">0.05</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Tree for run Zosterops2-1.png" class="img-fluid figure-img" width="1152"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="settings-1" class="level4">
<h4 class="anchored" data-anchor-id="settings-1">Settings</h4>
<p>The PICL <code>settings</code> file is a text file that establishes the parameters for a particular run and matches individual samples to assigned groups. This file must be saved in the directory where PICL is to be run and named <code>settings</code>. See the description of the <code>settings</code> file for run 1 above. Arguments for PICL are described in detail at <a href="https://github.com/lkubatko/PICL/blob/main/Documentation.md" class="uri">https://github.com/lkubatko/PICL/blob/main/Documentation.md</a>.</p>
<p>The only difference between the <code>settings</code> files for run 1 and run 2 is that the taxon assignments are different with more taxa in run 2 and the random starting numbers for <code>Seed1</code> and <code>Seed2</code> are changed.</p>
<pre><code>Model: 3
Gaps: 1
Bootstrap: 100
Theta: 0.03501696
Rate_param: 1.0
Random_tree: 0
Opt_bl: 1
User_bl: 0
Num_opt: 1000000
Seed1: 65456
Seed2: 11290
Num_cat: 4
Tree_search: 0
Num_iter: 30000
Beta: 0.005
Verbose: 1
29
japonicus alani1 alani2 alani3 meyeni1 meyeni2 meyeni3 montanus1 montanus2 montanus3 montanus4 montanus5 montanus6 montanus7 montanus8 simplex1 simplex2 simplex3 erythro1 erythro2 everetti1 everetti2 everetti3 nigrorum1 nigrorum2 nigrorum3 nigrorum4 nigrorum5 palpebro
japonicus ZJlo012
japonicus ZJlo015
japonicus ZJlo016
japonicus ZJlo017
japonicus ZJlo021
japonicus ZJlo022
japonicus ZJlo024
japonicus ZJlo031
japonicus ZJlo045
japonicus ZJlo046
japonicus ZJlo050
japonicus ZJlo052
japonicus ZJlo055
japonicus ZJja001
japonicus ZJja002
japonicus ZJja003
japonicus ZJja004
japonicus ZJja005
japonicus ZJja009
japonicus ZJja010
japonicus ZJja012
japonicus ZJja013
japonicus ZJja014
japonicus ZJja016
japonicus ZJja030
alani1 ZJal003
alani1  Zjal004
alani1  ZJal005
alani2  ZJal010
alani2  ZJal011
alani2  ZJal012
alani3  ZJal020
japonicus ZJst007
japonicus ZJst009
japonicus ZJst010
japonicus ZJst012
japonicus ZJst014
japonicus ZJst015
japonicus ZJin001
japonicus ZJin003
simplex3 ZJsi002
simplex2 ZJsi017
simplex2 ZJsi032
simplex3 ZJsi023
simplex3 ZJsi024
simplex2 ZJsi027
simplex2 ZJsi028
simplex2 ZJsi029
simplex2 ZJsi031
simplex1 ZPxx001
simplex1 ZPxx002
meyeni3 ZMxx002
meyeni1 ZMxx003
meyeni1 ZMxx004
meyeni1 ZMxx005
meyeni1 ZMxx006
montanus2 ZMOxx001
montanus2 ZMOxx002
montanus2 ZMOxx003
montanus2 ZMOxx005
montanus1 ZMOha001
montanus1 ZMOwh002
montanus1 ZMOwh003
montanus1 ZMOwh004
montanus1 ZMOwh005
montanus1 ZMOwh006
montanus4 ZMOvu002
montanus4 ZMOvu003
montanus4 ZMOvu004
montanus4 ZMOvu005
montanus6 ZMOmo001
montanus8 ZMOpa001
montanus8 ZMOpa002
nigrorum1 ZNni001
erythro1 ZERxx002
erythro1 ZERxx004
erythro1 ZERxx005
simplex3 Zsim23588
simplex3 Zsim28142
palpebro Zsim30897
palpebro Zpal23498
palpebro Zpal23522
montanus3 Zmon20893
montanus3 Zmon20892
montanus3 Zmon20902
montanus3 Zmon20909
montanus7 Zsim31166
montanus7 Zsim31159
meyeni3 Zmey17876
meyeni3 Zmey17877
meyeni3 Zmey17852
meyeni2 Zmey17922
meyeni2 Zmey17920
meyeni2 Zmey17925
meyeni2 Zmey17923
erythro2 Zery28090
erythro2 Zery28091
erythro2 Zery28088
nigrorum2 Zni10863
nigrorum5 Zni14341
nigrorum3 Zni19650
nigrorum4 Zni17984
palpebro ZjaDOT10981
simplex3 ZjaDOT5235
simplex3 ZLA1228662
simplex3 ZLA1225772
simplex3 ZLA1221882
simplex3 Zsim13809
simplex3 Zsim13773
simplex3 Zsim6797
simplex2 Zsim10336
simplex3 Zsim11362
simplex3 Zsim11102
simplex3 Zsim11220
montanus3 Zmon20899
montanus5 Zmon28375
everetti1 Zev13949
everetti3 Zev31650
everetti2 Zev28451
japonicus ZHIBRY431
japonicus ZHINAN290
japonicus ZHIWAI087
japonicus ZHIWAI078
japonicus ZHISOL783</code></pre>
</section>
<section id="run-3" class="level4">
<h4 class="anchored" data-anchor-id="run-3">Run</h4>
<p>To run the analysis open a terminal session and <code>cd</code> to the <code>PICL/Zosterops2</code> directory that was created earlier. Enter <code>./picl</code> to run the program. Remember a copy of the <code>picl</code> executable file must be present in this directory along with the needed data (<code>data.phy</code>) and settings files (<code>settings</code> and <code>treefile.tre</code>). After executing the <code>./picl</code> command a summary of the settings should appear in the terminal followed by a list of the taxon designations and the number of individual samples in each. An alert will follow stating <code>Estimation will be carried out using the SNP model</code> followed by a summary of the <code>data.phy</code> file which should contain 124 lineages and 15704 sites. The tree will be read from the <code>treefile.tre</code> file and a summary showing the dummy branch lengths will follow. Once the tree is successfully read a composite likelihood for the initial tree will be calculated and a revised list of branch lengths and a new composite likelihood will follow. The program will state that the tree has been written to the file <code>outtree.tre</code> and the bootstrapping reps will follow. A running tally of the bootstrap reps will appear as each replicate bootstrap run is completed. 100 bootstrap replicates under these settings takes maybe 15 hours or so on an M2 max MacBook Pro with 96Gb RAM with this particular starting tree.</p>
</section>
<section id="results-1" class="level4">
<h4 class="anchored" data-anchor-id="results-1">Results</h4>
<p>Three files will be produced following a PICL run.The <code>results</code> file contains the branch lengths in the final estimated tree if <code>Verbose</code> is set to 1 (this is primarily used for debugging and simulation studies, and will not be useful for most users). The <code>outtree.tre</code> file should contain two trees in Newick format with branch lengths in mutation units (substitutions per site per generation) in the first tree and coalescent units (generations in units of twice the effective population size, 2N<sub>e</sub>) for the second tree. The <code>boot.dat</code> file stores the results of the bootstrapping replicates. This file should have n-1 columns corresponding to the internal nodes of the tree where n equals the number of taxa designated in the <code>settings</code> file.</p>
<p>Numbering the tips and internal nodes is a little tricky. Numbering begins consecutively with the tips and is followed by the numbers for the internal nodes. Because there are 29 taxa on this particular tree the internal nodes are numbered beginning at the root with 30.</p>
<p>Using the ape package in R read the <code>outtree.tre</code> upon completion of the PICL run and plot both the tree in mutation units and the tree in coalescent units. Add the correct node labels to the tree. You can make sure they make sense by making sure they match up with the speciation time estimates from the bootstraps (see below).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb54"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the outtree file from PICL</span></span>
<span id="cb54-2"><a href="#cb54-2" aria-hidden="true" tabindex="-1"></a>Zosterops2_outtree <span class="ot">&lt;-</span> <span class="fu">read.tree</span>(<span class="st">"Data/PICL/Zosterops2/outtree.tre"</span>)</span>
<span id="cb54-3"><a href="#cb54-3" aria-hidden="true" tabindex="-1"></a>node_labels2 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">30</span><span class="sc">:</span><span class="dv">57</span>)</span>
<span id="cb54-4"><a href="#cb54-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-5"><a href="#cb54-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the mutation units [1] and coalescent units [2]. </span></span>
<span id="cb54-6"><a href="#cb54-6" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops2_outtree[[<span class="dv">1</span>]], <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.001</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb54-7"><a href="#cb54-7" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(Zosterops2_outtree[[<span class="dv">1</span>]]<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="fl">0.1</span>), <span class="at">labels =</span> <span class="cn">FALSE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb54-8"><a href="#cb54-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"0.1 substitutions per site per generation"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">1</span>]]<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">2.5</span>, <span class="at">cex =</span> <span class="dv">2</span>)</span>
<span id="cb54-9"><a href="#cb54-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Add labels to the internal nodes of the first tree with an offset</span></span>
<span id="cb54-10"><a href="#cb54-10" aria-hidden="true" tabindex="-1"></a>n_internal_nodes1 <span class="ot">&lt;-</span> <span class="fu">Nnode</span>(Zosterops2_outtree[[<span class="dv">1</span>]])  <span class="co"># Number of internal nodes</span></span>
<span id="cb54-11"><a href="#cb54-11" aria-hidden="true" tabindex="-1"></a><span class="fu">nodelabels</span>(node_labels2[<span class="dv">1</span><span class="sc">:</span>n_internal_nodes1], </span>
<span id="cb54-12"><a href="#cb54-12" aria-hidden="true" tabindex="-1"></a>            <span class="at">node =</span> (<span class="fu">Ntip</span>(Zosterops2_outtree[[<span class="dv">1</span>]]) <span class="sc">+</span> <span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">Ntip</span>(Zosterops2_outtree[[<span class="dv">1</span>]]) <span class="sc">+</span> n_internal_nodes1), </span>
<span id="cb54-13"><a href="#cb54-13" aria-hidden="true" tabindex="-1"></a>            <span class="at">frame =</span> <span class="st">"none"</span>, <span class="at">cex =</span> <span class="fl">1.5</span>, <span class="at">adj =</span> <span class="fu">c</span>(<span class="fl">1.1</span>, <span class="fl">1.2</span>))  <span class="co"># Adjust adj for positioning</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops2 PICL run-1.png" class="img-fluid figure-img" width="2400"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops2_outtree[[<span class="dv">2</span>]], <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.035</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb55-2"><a href="#cb55-2" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(Zosterops2_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="dv">3</span>), <span class="at">labels =</span> <span class="cn">FALSE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb55-3"><a href="#cb55-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"3 coalescent units (2Ne)"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(Zosterops1_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">3.3</span>, <span class="at">cex =</span> <span class="dv">2</span>)</span>
<span id="cb55-4"><a href="#cb55-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Add labels to the internal nodes of the first tree with an offset</span></span>
<span id="cb55-5"><a href="#cb55-5" aria-hidden="true" tabindex="-1"></a>n_internal_nodes2 <span class="ot">&lt;-</span> <span class="fu">Nnode</span>(Zosterops2_outtree[[<span class="dv">2</span>]])  <span class="co"># Number of internal nodes</span></span>
<span id="cb55-6"><a href="#cb55-6" aria-hidden="true" tabindex="-1"></a><span class="fu">nodelabels</span>(node_labels2[<span class="dv">1</span><span class="sc">:</span>n_internal_nodes2], </span>
<span id="cb55-7"><a href="#cb55-7" aria-hidden="true" tabindex="-1"></a>            <span class="at">node =</span> (<span class="fu">Ntip</span>(Zosterops2_outtree[[<span class="dv">2</span>]]) <span class="sc">+</span> <span class="dv">1</span>)<span class="sc">:</span>(<span class="fu">Ntip</span>(Zosterops2_outtree[[<span class="dv">2</span>]]) <span class="sc">+</span> n_internal_nodes2), </span>
<span id="cb55-8"><a href="#cb55-8" aria-hidden="true" tabindex="-1"></a>            <span class="at">frame =</span> <span class="st">"none"</span>, <span class="at">cex =</span> <span class="fl">1.5</span>, <span class="at">adj =</span> <span class="fu">c</span>(<span class="fl">1.1</span>, <span class="fl">1.2</span>))  <span class="co"># Adjust adj for positioning</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops2 PICL run-2.png" class="img-fluid figure-img" width="2400"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb56"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print existing tip labels to see the order, enter the actual scientific names in the right order, replace the tip labels with the scientific names</span></span>
<span id="cb56-2"><a href="#cb56-2" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(Zosterops2_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>tip.label)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> [1] "palpebro"  "montanus1" "japonicus" "alani3"    "alani1"    "alani2"   
 [7] "meyeni1"   "meyeni2"   "meyeni3"   "montanus8" "nigrorum5" "montanus2"
[13] "montanus3" "montanus4" "montanus5" "montanus6" "montanus7" "simplex1" 
[19] "simplex2"  "simplex3"  "erythro1"  "erythro2"  "everetti1" "everetti2"
[25] "everetti3" "nigrorum1" "nigrorum4" "nigrorum2" "nigrorum3"</code></pre>
</div>
<div class="sourceCode cell-code" id="cb58"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Choose the new names. </span></span>
<span id="cb58-2"><a href="#cb58-2" aria-hidden="true" tabindex="-1"></a>Zosterops2_outtree_species <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. palpebrosus"</span>, <span class="st">"Z. montanus NORTH"</span>, <span class="st">"Z. japonicus CONTINENTAL"</span>, <span class="st">"Z. japonicus alani IWOJIMA"</span>, <span class="st">"Z. japonicus alani HAHAJIMA"</span>, </span>
<span id="cb58-3"><a href="#cb58-3" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Z. japonicus alani ANEJIMA"</span>, <span class="st">"Z. meyeni LANYU"</span>, <span class="st">"Z. meyeni SABTANG"</span>, <span class="st">"Z. meyeni BATAN"</span>, <span class="st">"Z. montanus parkesi"</span>, <span class="st">"Z. nigrorum catarmanensis"</span>,</span>
<span id="cb58-4"><a href="#cb58-4" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Z. montanus PANAY"</span>, <span class="st">"Z. montanus pectoralis"</span>, <span class="st">"Z. montanus vulcani"</span>, <span class="st">"Z. montanus diuatae"</span>, <span class="st">"Z. montanus montanus SULAWESI"</span>, </span>
<span id="cb58-5"><a href="#cb58-5" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Z. montanus montanus JAVA"</span>, <span class="st">"Z. simplex SINGAPORE"</span>, <span class="st">"Z. simplex GUANGXI"</span>, <span class="st">"Z. simplex CHINA, TAIWAN, VIETNAM, USA"</span>, <span class="st">"Z. erythropleurus RUSSIA"</span>, </span>
<span id="cb58-6"><a href="#cb58-6" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Z. erythropleurus VIETNAM"</span>, <span class="st">"Z. everetti basilanicus CAMIGUINSUR"</span>, <span class="st">"Z. everetti basilanicus MINDANAO"</span>, <span class="st">"Z. everetti boholensis SAMAR"</span>, <span class="st">"Z. nigrorum nigrorum"</span>, </span>
<span id="cb58-7"><a href="#cb58-7" aria-hidden="true" tabindex="-1"></a>                                <span class="st">"Z. nigrorum luzonicus"</span>, <span class="st">"Z. nigrorum meyleri"</span>, <span class="st">"Z. nigrorum innominatus"</span>)</span>
<span id="cb58-8"><a href="#cb58-8" aria-hidden="true" tabindex="-1"></a>Zosterops2_outtree[[<span class="dv">2</span>]]<span class="sc">$</span>tip.label <span class="ot">&lt;-</span> Zosterops2_outtree_species  </span>
<span id="cb58-9"><a href="#cb58-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb58-10"><a href="#cb58-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the coalescent units, this will be used later to create a figure</span></span>
<span id="cb58-11"><a href="#cb58-11" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops2_outtree[[<span class="dv">2</span>]], <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.08</span>, <span class="at">cex =</span> <span class="fl">1.2</span>, <span class="at">direction =</span> <span class="st">"upwards"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops2 PICL run-3.png" class="img-fluid figure-img" width="2400"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Remove the branch lengths and plot just the topology, this will be used later to create a figure</span></span>
<span id="cb59-2"><a href="#cb59-2" aria-hidden="true" tabindex="-1"></a>Zosterops2_outtree_topology <span class="ot">&lt;-</span> Zosterops2_outtree[[<span class="dv">2</span>]]</span>
<span id="cb59-3"><a href="#cb59-3" aria-hidden="true" tabindex="-1"></a>Zosterops2_outtree_topology<span class="sc">$</span>edge.length <span class="ot">&lt;-</span> <span class="cn">NULL</span></span>
<span id="cb59-4"><a href="#cb59-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Zosterops2_outtree_topology, <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.3</span>, <span class="at">cex =</span> <span class="fl">1.2</span>, <span class="at">direction =</span> <span class="st">"downwards"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/Species tree from Zosterops2 PICL run-4.png" class="img-fluid figure-img" width="2400"></p>
</figure>
</div>
</div>
</div>
<p>Calculate the 95% confidence intervals for each node (speciation or split time) from the <code>boot.dat</code> file. Again, the <code>boot.dat</code> file should have no header and n-1 columns and 100 rows (or whatever value was set in the settings file at the <code>Bootstrap</code> prompt) of numerical estimates of split times measured in coalescent units. After calculating the means and 95% confidence interval for each column use the <code>gt</code> package to save the results to a table.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb60"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the text file</span></span>
<span id="cb60-2"><a href="#cb60-2" aria-hidden="true" tabindex="-1"></a>PICLbootstraps2 <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">'Data/PICL/Zosterops2/boot.dat'</span>, <span class="at">header =</span> <span class="cn">FALSE</span>)</span>
<span id="cb60-3"><a href="#cb60-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-4"><a href="#cb60-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Add a header line labeled 1-28</span></span>
<span id="cb60-5"><a href="#cb60-5" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(PICLbootstraps2) <span class="ot">&lt;-</span> <span class="fu">as.character</span>(<span class="dv">30</span><span class="sc">:</span><span class="dv">57</span>)</span>
<span id="cb60-6"><a href="#cb60-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-7"><a href="#cb60-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Initialize vectors to store means and confidence intervals</span></span>
<span id="cb60-8"><a href="#cb60-8" aria-hidden="true" tabindex="-1"></a>means <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps2))</span>
<span id="cb60-9"><a href="#cb60-9" aria-hidden="true" tabindex="-1"></a>lower_ci <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps2))</span>
<span id="cb60-10"><a href="#cb60-10" aria-hidden="true" tabindex="-1"></a>upper_ci <span class="ot">&lt;-</span> <span class="fu">numeric</span>(<span class="fu">ncol</span>(PICLbootstraps2))</span>
<span id="cb60-11"><a href="#cb60-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-12"><a href="#cb60-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Calculate means and 95% confidence intervals for each column</span></span>
<span id="cb60-13"><a href="#cb60-13" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(PICLbootstraps2)) {</span>
<span id="cb60-14"><a href="#cb60-14" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Remove NA values. NA values should not appear in the bootstraps however. This would mean there is a problem with the code that needs to be investigated. </span></span>
<span id="cb60-15"><a href="#cb60-15" aria-hidden="true" tabindex="-1"></a>    <span class="co"># This is just a precautionary alert for a very beta software.</span></span>
<span id="cb60-16"><a href="#cb60-16" aria-hidden="true" tabindex="-1"></a>    column_data <span class="ot">&lt;-</span> PICLbootstraps2[[i]]</span>
<span id="cb60-17"><a href="#cb60-17" aria-hidden="true" tabindex="-1"></a>    column_data <span class="ot">&lt;-</span> column_data[<span class="sc">!</span><span class="fu">is.na</span>(column_data)]</span>
<span id="cb60-18"><a href="#cb60-18" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb60-19"><a href="#cb60-19" aria-hidden="true" tabindex="-1"></a>    n <span class="ot">&lt;-</span> <span class="fu">length</span>(column_data)  <span class="co"># Sample size</span></span>
<span id="cb60-20"><a href="#cb60-20" aria-hidden="true" tabindex="-1"></a>    means[i] <span class="ot">&lt;-</span> <span class="fu">mean</span>(column_data)  <span class="co"># Sample mean</span></span>
<span id="cb60-21"><a href="#cb60-21" aria-hidden="true" tabindex="-1"></a>    sd_value <span class="ot">&lt;-</span> <span class="fu">sd</span>(column_data)  <span class="co"># Sample standard deviation</span></span>
<span id="cb60-22"><a href="#cb60-22" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb60-23"><a href="#cb60-23" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate the t-value for 95% confidence</span></span>
<span id="cb60-24"><a href="#cb60-24" aria-hidden="true" tabindex="-1"></a>    t_value <span class="ot">&lt;-</span> <span class="fu">qt</span>(<span class="fl">0.975</span>, <span class="at">df =</span> n <span class="sc">-</span> <span class="dv">1</span>)  <span class="co"># 0.975 for two-tailed test</span></span>
<span id="cb60-25"><a href="#cb60-25" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb60-26"><a href="#cb60-26" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate the margin of error</span></span>
<span id="cb60-27"><a href="#cb60-27" aria-hidden="true" tabindex="-1"></a>    margin_of_error <span class="ot">&lt;-</span> t_value <span class="sc">*</span> (sd_value <span class="sc">/</span> <span class="fu">sqrt</span>(n))</span>
<span id="cb60-28"><a href="#cb60-28" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb60-29"><a href="#cb60-29" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Calculate lower and upper confidence intervals</span></span>
<span id="cb60-30"><a href="#cb60-30" aria-hidden="true" tabindex="-1"></a>    lower_ci[i] <span class="ot">&lt;-</span> means[i] <span class="sc">-</span> margin_of_error</span>
<span id="cb60-31"><a href="#cb60-31" aria-hidden="true" tabindex="-1"></a>    upper_ci[i] <span class="ot">&lt;-</span> means[i] <span class="sc">+</span> margin_of_error</span>
<span id="cb60-32"><a href="#cb60-32" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb60-33"><a href="#cb60-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-34"><a href="#cb60-34" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a data frame for the results</span></span>
<span id="cb60-35"><a href="#cb60-35" aria-hidden="true" tabindex="-1"></a>results2 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb60-36"><a href="#cb60-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">Column =</span> <span class="dv">30</span><span class="sc">:</span><span class="dv">57</span>,</span>
<span id="cb60-37"><a href="#cb60-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean =</span> means,</span>
<span id="cb60-38"><a href="#cb60-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">Lower_CI =</span> lower_ci,</span>
<span id="cb60-39"><a href="#cb60-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">Upper_CI =</span> upper_ci</span>
<span id="cb60-40"><a href="#cb60-40" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb60-41"><a href="#cb60-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-42"><a href="#cb60-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Create a gt table</span></span>
<span id="cb60-43"><a href="#cb60-43" aria-hidden="true" tabindex="-1"></a>results_table2 <span class="ot">&lt;-</span> results2 <span class="sc">%&gt;%</span></span>
<span id="cb60-44"><a href="#cb60-44" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb60-45"><a href="#cb60-45" aria-hidden="true" tabindex="-1"></a>    <span class="fu">tab_header</span>(</span>
<span id="cb60-46"><a href="#cb60-46" aria-hidden="true" tabindex="-1"></a>        <span class="at">title =</span> <span class="st">"Zosterops2 Mean and 95% CIs for speciation times"</span></span>
<span id="cb60-47"><a href="#cb60-47" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span></span>
<span id="cb60-48"><a href="#cb60-48" aria-hidden="true" tabindex="-1"></a>    <span class="fu">cols_label</span>(</span>
<span id="cb60-49"><a href="#cb60-49" aria-hidden="true" tabindex="-1"></a>        <span class="at">Column =</span> <span class="st">"Node"</span>,</span>
<span id="cb60-50"><a href="#cb60-50" aria-hidden="true" tabindex="-1"></a>        <span class="at">Mean =</span> <span class="st">"Mean"</span>,</span>
<span id="cb60-51"><a href="#cb60-51" aria-hidden="true" tabindex="-1"></a>        <span class="at">Lower_CI =</span> <span class="st">"Lower CI"</span>,</span>
<span id="cb60-52"><a href="#cb60-52" aria-hidden="true" tabindex="-1"></a>        <span class="at">Upper_CI =</span> <span class="st">"Upper CI"</span></span>
<span id="cb60-53"><a href="#cb60-53" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span></span>
<span id="cb60-54"><a href="#cb60-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">fmt_number</span>(</span>
<span id="cb60-55"><a href="#cb60-55" aria-hidden="true" tabindex="-1"></a>        <span class="at">columns =</span> <span class="fu">vars</span>(Mean, Lower_CI, Upper_CI),</span>
<span id="cb60-56"><a href="#cb60-56" aria-hidden="true" tabindex="-1"></a>        <span class="at">decimals =</span> <span class="dv">8</span></span>
<span id="cb60-57"><a href="#cb60-57" aria-hidden="true" tabindex="-1"></a>    )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb62"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Display the table</span></span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>results_table2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="tlkdrucseg" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#tlkdrucseg table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#tlkdrucseg thead, #tlkdrucseg tbody, #tlkdrucseg tfoot, #tlkdrucseg tr, #tlkdrucseg td, #tlkdrucseg th {
  border-style: none;
}

#tlkdrucseg p {
  margin: 0;
  padding: 0;
}

#tlkdrucseg .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: auto;
  margin-right: auto;
  color: #333333;
  font-size: 16px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: auto;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#tlkdrucseg .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#tlkdrucseg .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#tlkdrucseg .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#tlkdrucseg .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tlkdrucseg .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tlkdrucseg .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tlkdrucseg .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#tlkdrucseg .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#tlkdrucseg .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#tlkdrucseg .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#tlkdrucseg .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#tlkdrucseg .gt_spanner_row {
  border-bottom-style: hidden;
}

#tlkdrucseg .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#tlkdrucseg .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#tlkdrucseg .gt_from_md > :first-child {
  margin-top: 0;
}

#tlkdrucseg .gt_from_md > :last-child {
  margin-bottom: 0;
}

#tlkdrucseg .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#tlkdrucseg .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#tlkdrucseg .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#tlkdrucseg .gt_row_group_first td {
  border-top-width: 2px;
}

#tlkdrucseg .gt_row_group_first th {
  border-top-width: 2px;
}

#tlkdrucseg .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tlkdrucseg .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#tlkdrucseg .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#tlkdrucseg .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tlkdrucseg .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tlkdrucseg .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#tlkdrucseg .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#tlkdrucseg .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#tlkdrucseg .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tlkdrucseg .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tlkdrucseg .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tlkdrucseg .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tlkdrucseg .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tlkdrucseg .gt_left {
  text-align: left;
}

#tlkdrucseg .gt_center {
  text-align: center;
}

#tlkdrucseg .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#tlkdrucseg .gt_font_normal {
  font-weight: normal;
}

#tlkdrucseg .gt_font_bold {
  font-weight: bold;
}

#tlkdrucseg .gt_font_italic {
  font-style: italic;
}

#tlkdrucseg .gt_super {
  font-size: 65%;
}

#tlkdrucseg .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#tlkdrucseg .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#tlkdrucseg .gt_indent_1 {
  text-indent: 5px;
}

#tlkdrucseg .gt_indent_2 {
  text-indent: 10px;
}

#tlkdrucseg .gt_indent_3 {
  text-indent: 15px;
}

#tlkdrucseg .gt_indent_4 {
  text-indent: 20px;
}

#tlkdrucseg .gt_indent_5 {
  text-indent: 25px;
}

#tlkdrucseg .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#tlkdrucseg div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="4" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops2 Mean and 95% CIs for speciation times</th>
</tr>
<tr class="gt_col_headings even">
<th id="Column" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Node</th>
<th id="Mean" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Mean</th>
<th id="Lower_CI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Lower CI</th>
<th id="Upper_CI" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Upper CI</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_right" headers="Column">30</td>
<td class="gt_row gt_right" headers="Mean">4.50701383</td>
<td class="gt_row gt_right" headers="Lower_CI">4.48543103</td>
<td class="gt_row gt_right" headers="Upper_CI">4.52859663</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">31</td>
<td class="gt_row gt_right" headers="Mean">4.16711517</td>
<td class="gt_row gt_right" headers="Lower_CI">4.14830936</td>
<td class="gt_row gt_right" headers="Upper_CI">4.18592097</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">32</td>
<td class="gt_row gt_right" headers="Mean">2.72914868</td>
<td class="gt_row gt_right" headers="Lower_CI">2.71607279</td>
<td class="gt_row gt_right" headers="Upper_CI">2.74222458</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">33</td>
<td class="gt_row gt_right" headers="Mean">2.06876869</td>
<td class="gt_row gt_right" headers="Lower_CI">2.05516129</td>
<td class="gt_row gt_right" headers="Upper_CI">2.08237609</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">34</td>
<td class="gt_row gt_right" headers="Mean">0.12941373</td>
<td class="gt_row gt_right" headers="Lower_CI">0.12510362</td>
<td class="gt_row gt_right" headers="Upper_CI">0.13372385</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">35</td>
<td class="gt_row gt_right" headers="Mean">0.00000409</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000331</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000487</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">36</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">37</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">38</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">39</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">40</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">41</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">42</td>
<td class="gt_row gt_right" headers="Mean">0.10102479</td>
<td class="gt_row gt_right" headers="Lower_CI">0.09394097</td>
<td class="gt_row gt_right" headers="Upper_CI">0.10810861</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">43</td>
<td class="gt_row gt_right" headers="Mean">0.00019391</td>
<td class="gt_row gt_right" headers="Lower_CI">−0.00017823</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00056605</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">44</td>
<td class="gt_row gt_right" headers="Mean">0.00019076</td>
<td class="gt_row gt_right" headers="Lower_CI">−0.00018080</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00056231</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">45</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000001</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">46</td>
<td class="gt_row gt_right" headers="Mean">0.00000001</td>
<td class="gt_row gt_right" headers="Lower_CI">−0.00000001</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000003</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">47</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">48</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">49</td>
<td class="gt_row gt_right" headers="Mean">0.00005008</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00003858</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00006159</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">50</td>
<td class="gt_row gt_right" headers="Mean">0.00000000</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000000</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">51</td>
<td class="gt_row gt_right" headers="Mean">0.00997728</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00665223</td>
<td class="gt_row gt_right" headers="Upper_CI">0.01330233</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">52</td>
<td class="gt_row gt_right" headers="Mean">0.92572939</td>
<td class="gt_row gt_right" headers="Lower_CI">0.91456738</td>
<td class="gt_row gt_right" headers="Upper_CI">0.93689140</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">53</td>
<td class="gt_row gt_right" headers="Mean">0.00017073</td>
<td class="gt_row gt_right" headers="Lower_CI">−0.00011833</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00045980</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">54</td>
<td class="gt_row gt_right" headers="Mean">0.00000001</td>
<td class="gt_row gt_right" headers="Lower_CI">0.00000000</td>
<td class="gt_row gt_right" headers="Upper_CI">0.00000002</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">55</td>
<td class="gt_row gt_right" headers="Mean">0.70166163</td>
<td class="gt_row gt_right" headers="Lower_CI">0.69110160</td>
<td class="gt_row gt_right" headers="Upper_CI">0.71222167</td>
</tr>
<tr class="odd">
<td class="gt_row gt_right" headers="Column">56</td>
<td class="gt_row gt_right" headers="Mean">0.32407711</td>
<td class="gt_row gt_right" headers="Lower_CI">0.31449242</td>
<td class="gt_row gt_right" headers="Upper_CI">0.33366180</td>
</tr>
<tr class="even">
<td class="gt_row gt_right" headers="Column">57</td>
<td class="gt_row gt_right" headers="Mean">0.32143755</td>
<td class="gt_row gt_right" headers="Lower_CI">0.31165783</td>
<td class="gt_row gt_right" headers="Upper_CI">0.33121727</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(results_table2, <span class="st">"Zosterops2PICL_mean_CIss.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
</section>
<section id="delineate" class="level2">
<h2 class="anchored" data-anchor-id="delineate">Delineate</h2>
<p>Now we have a well supported tree with branch lengths and at least some known, undisputed species we can use the python3 program <a href="https://jeetsukumaran.github.io/delineate/install.html">Delineate</a> <span class="citation" data-cites="sukumaran2021">(<a href="#ref-sukumaran2021" role="doc-biblioref">Sukumaran, Holder, and Knowles 2021</a>)</span> to test hypotheses regarding species limits. Delineate estimates speciation rates and rates of lineage birth from a tree where some lineages belong to known species and have been constrained as such while other lineages on the tree are unconstrained and therefore may be assigned to an existing species or delineated as a new species. Five runs are done with Delineate using the tree created in PICL and five different sets of lineage constraints.</p>
<section id="run-1-1" class="level3">
<h3 class="anchored" data-anchor-id="run-1-1">Run 1</h3>
<section id="constraints" class="level4">
<h4 class="anchored" data-anchor-id="constraints">Constraints</h4>
<p>Delineate needs two files to run. They can be named whatever you wish. The first is a tree file. For this use the final tree produced from PICL run 2 with the branch lengths in coalescent units and exported to a nexus formatted file using <a href="https://beast.community/figtree">Figtree</a> (see above). Because the branch lengths are in coalescent units the speciation completion rates from Delineate will also be in these units. Some of the branch lengths are zero or very near zero with 95% CIs that include zero. Where there was a zero or near zero branch length on the tree the number of significant digits was changed to nine and a final estimated branch length of 1.0 x 10<sup>-9</sup> was used instead. This nexus formatted tree is named <code>ZosteropsPICL_tree.nex</code>.</p>
<pre><code>#NEXUS
begin trees;
    tree tree_2 = [&amp;R] (palpebro:4.495118,(((((montanus1:0.000000001,((japonicus:0.000000001,(alani3:0.000000001,(alani1:0.000000001,alani2:0.000000001):0.000000001):0.000000001):0.000000001,(meyeni1:0.000000001,(meyeni2:0.000000001,meyeni3:0.000000001):0.000000001):0.000000001):0.000000001):0.122653,(montanus8:0.094776,((nigrorum5:0.000000001,(montanus2:0.000000001,montanus3:0.000000001):0.000000001):0.000003,((montanus4:0.000000001,montanus5:0.000000001):0.000000001,(montanus6:0.000000001,montanus7:0.000000001):0.000000001):0.000003):0.094772):0.027878):1.944734,(simplex1:0.000019,(simplex2:0.000000001,simplex3:0.000000001):0.000019):2.067368):0.667965,(erythro1:0.000072,erythro2:0.000072):2.73528):1.424642,((everetti1:0.000029,(everetti2:0.000000001,everetti3:0.000000001):0.000029):0.920545,(nigrorum1:0.700863,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):0.000005):0.371394):0.219711):3.239421):0.335124);
end;</code></pre>
<p>Plot the tree for Delineate.</p>
<div class="cell" data-label="Tree from PICL run2 with branch lengths in coalescent units to use as tree for Delineate.">
<div class="sourceCode cell-code" id="cb65"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the coalescent units</span></span>
<span id="cb65-2"><a href="#cb65-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the outtree file from PICL</span></span>
<span id="cb65-3"><a href="#cb65-3" aria-hidden="true" tabindex="-1"></a>ZosteropsPICL_tree <span class="ot">&lt;-</span> <span class="fu">read.nexus</span>(<span class="st">"Data/Delineate/ZosteropsPICL_tree.nex"</span>)</span>
<span id="cb65-4"><a href="#cb65-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(ZosteropsPICL_tree, <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.04</span>, <span class="at">cex =</span> <span class="fl">1.8</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="2400"></p>
</figure>
</div>
</div>
</div>
<p>In addition to a tree file, Delineate requires that each lineage on the tree be identified with a taxon name and the status of that identification need be either constrained or unconstrained. This is done with a tab delimited text file with three columns and the column headers ‘lineage’, ‘species’, and ‘status’. The entries in the ‘lineage’ column should correspond to the lineages on the tree. ‘Species’ should reflect the species assignment for that particular lineage. ‘Status’ should indicate that this assignment is constrained (1) or unconstrained (0). In this first run all the lineages <em>Z. japonicus</em>, <em>Z. japonicus alani</em>, <em>Z. meyeni</em>, <em>Z. montanus</em> lineages were set as unconstrained. <em>Z. nigrorum nigrorum</em> (nigrorum1) and <em>Z. nigrorum catarmanensis</em> (nigrorum5) were also set as unconstrained due to their relatively long branches or taxonomic mismatch, respectively. The name for this assignment file for the first Delineate run is <code>ZosteropsDELINEATE1_assignments.tsv</code>.</p>
<pre><code>lineage species status
japonicus   japonicus   0
alani1  alani   0
alani2  alani   0
alani3  alani   0
meyeni1 meyeni  0
meyeni2 meyeni  0
meyeni3 meyeni  0
montanus1   montanus    0
montanus2   montanus    0
montanus3   montanus    0
montanus4   montanus    0
montanus5   montanus    0
montanus6   montanus    0
montanus7   montanus    0
montanus8   montanus    0
simplex1    simplex 1
simplex2    simplex 1
simplex3    simplex 1
erythro1    erythropleurus  1
erythro2    erythropleurus  1
everetti1   everetti    1
everetti2   everetti    1
everetti3   everetti    1
nigrorum1   nigrorum    0
nigrorum2   nigrorum    1
nigrorum3   nigrorum    1
nigrorum4   nigrorum    1
nigrorum5   nigrorum    0
palpebro    palpebrosus 1</code></pre>
</section>
<section id="run-4" class="level4">
<h4 class="anchored" data-anchor-id="run-4">Run</h4>
<p>The authors of Delineate recommend running in a dedicated environment with the program and its dependencies installed. So that is what we did. Create a dedicated environment for Delineate using <a href="https://www.anaconda.com/">Anaconda</a>. The authors of Delineate link to instructions for creating environments in their <a href="https://jeetsukumaran.github.io/delineate/install.html">documentation</a>. Once you have created an environment to run Delineate install the program and its dependencies within this environment. Again see the Delineate <a href="https://jeetsukumaran.github.io/delineate/install.html">documentation</a> for installation instructions. In the Anaconda Navigator app navigate to the <strong>Environments</strong> tab and choose the play button next to the <strong>DELINEATE</strong> environment (this is the obvious name we chose for our dedicated Delineate environment) and then from the dropdown menu select <strong>Open Terminal</strong>. You now should have a prompt in the terminal that looks like this.</p>
<pre><code>(DELINEATE) &lt;specific-user-name&gt;-mac Delineate %</code></pre>
<p>Go to the directory created for the Delineate runs containing the assignments (<code>ZosteropsDELINEATE1_assignments.tsv</code>) and the nexus formatted tree (<code>ZosteropsPICL_tree.nex</code>) with the <code>cd</code> command. To run an analysis that will estimate the partitions among the lineages enter the following command. The <code>-t</code> argument specifies the tree file and the <code>-c</code> argument identifies the assignment file used to define the constraints based on known species lineages. The default using <code>delineate-estimate</code> is to estimate all possible partitions and their probabilities. When there are relatively few constraints on the tree, this is an enormous number that will consume a commensurate enormous amount of computing resources and create huge data files for even a moderate number of lineages. The vast majority of these partitions will be associated with infinitesimally small probabilities. The <code>-P</code> argument restricts the report to the partitions that contribute most of the cumulative probability. <code>-P 0.95</code> will cause Delineate to report only the partitions that contribute to 95% of the cumulative probability. The <code>-u</code> argument allows Delineate to better handle the numerical computing challenges when dealing with very small partition probabilities. This option is recommended for larger datasets with fewer <em>a priori</em> constraints. A full list of options and operations in Delineate is available in their <a href="https://jeetsukumaran.github.io/delineate/quickstart.html">documentation</a>.</p>
<pre><code>delineate-estimate partitions -t ZosteropsPICL_tree.nex -c ZosteropsDELINEATE1_assignments.tsv -P 0.95 -u</code></pre>
</section>
<section id="results-2" class="level4">
<h4 class="anchored" data-anchor-id="results-2">Results</h4>
<p>The fewer constraints in the assignments file the more possible partitions Delineate has to explore and the longer it will take. Three files will be created upon completion of a run.</p>
<ul>
<li><p><code>ZosteropsDELINEATE1_assignments.delimitation-results.json</code></p></li>
<li><p><code>ZosteropsDELINEATE1_assignments.delimitation-results.trees</code></p></li>
<li><p><code>ZosteropsDELINEATE1_assignments.delimitation-results.trunc.trees</code></p></li>
</ul>
<p>Notice that each file begins with the name given to the <code>.tsv</code> assignment file. The <code>.json</code> file shows the results in a format that may be accessed in a browser or as a text document. The results in the <code>.json</code> file begin with a summary of the taxa, the tree used, estimates of the birth rate of lineages, the speciation rate, and a list of the constrained (known) and unconstrained (unknown or uncertain) lineages identified in the <code>.tsv</code> file.</p>
<pre><code>    "lineages": [
        "palpebro",
        "montanus1",
        "japonicus",
        "alani3",
        "alani1",
        "alani2",
        "meyeni1",
        "meyeni2",
        "meyeni3",
        "montanus8",
        "nigrorum5",
        "montanus2",
        "montanus3",
        "montanus4",
        "montanus5",
        "montanus6",
        "montanus7",
        "simplex1",
        "simplex2",
        "simplex3",
        "erythro1",
        "erythro2",
        "everetti1",
        "everetti2",
        "everetti3",
        "nigrorum1",
        "nigrorum4",
        "nigrorum2",
        "nigrorum3"
    ],
    "speciation_completion_rate": 0.7115674145319365,
    "speciation_completion_rate_source": "estimated",
    "speciation_completion_rate_estimate_lnl": -1.1529717191683189,
    "speciation_completion_rate_estimation_initial": 0.013202358749198284,
    "speciation_completion_rate_estimation_min": 1e-08,
    "speciation_completion_rate_estimation_max": 13.202358749198284,
    "lineage_tree_birth_rate": 1.3202358749198284,
    "constrained_lineage_tree": "[&amp;R] (palpebro:4.495118,(((simplex1:1.9e-05,(simplex2:1e-09,simplex3:1e-09):1.9e-05):2.7353330000000002,(erythro1:7.2e-05,erythro2:7.2e-05):2.73528):1.424642,((everetti1:2.9e-05,(everetti2:1e-09,everetti3:1e-09):2.9e-05):0.920545,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):5e-06):0.591105):3.239421):0.335124);\n",
    "species_constraints": {
        "constrained_lineages": [
            "simplex1",
            "simplex2",
            "simplex3",
            "erythro1",
            "erythro2",
            "everetti1",
            "everetti2",
            "everetti3",
            "nigrorum2",
            "nigrorum3",
            "nigrorum4",
            "palpebro"
        ],
        "unconstrained_lineages": [
            "japonicus",
            "alani1",
            "alani2",
            "alani3",
            "meyeni1",
            "meyeni2",
            "meyeni3",
            "montanus1",
            "montanus2",
            "montanus3",
            "montanus4",
            "montanus5",
            "montanus6",
            "montanus7",
            "montanus8",
            "nigrorum1",
            "nigrorum5"
        ],</code></pre>
<p>Following this summary in the <code>.json</code> file is a description of the partitions in the analysis in descending order starting with the most probable partition.</p>
<pre><code>        "lineage_species_map": {
            "japonicus": "japonicus",
            "alani1": "alani",
            "alani2": "alani",
            "alani3": "alani",
            "meyeni1": "meyeni",
            "meyeni2": "meyeni",
            "meyeni3": "meyeni",
            "montanus1": "montanus",
            "montanus2": "montanus",
            "montanus3": "montanus",
            "montanus4": "montanus",
            "montanus5": "montanus",
            "montanus6": "montanus",
            "montanus7": "montanus",
            "montanus8": "montanus",
            "simplex1": "simplex",
            "simplex2": "simplex",
            "simplex3": "simplex",
            "erythro1": "erythropleurus",
            "erythro2": "erythropleurus",
            "everetti1": "everetti",
            "everetti2": "everetti",
            "everetti3": "everetti",
            "nigrorum1": "nigrorum",
            "nigrorum2": "nigrorum",
            "nigrorum3": "nigrorum",
            "nigrorum4": "nigrorum",
            "nigrorum5": "nigrorum",
            "palpebro": "palpebrosus"
        },
        "constrained_lineage_species_map": {
            "simplex1": "simplex",
            "simplex2": "simplex",
            "simplex3": "simplex",
            "erythro1": "erythropleurus",
            "erythro2": "erythropleurus",
            "everetti1": "everetti",
            "everetti2": "everetti",
            "everetti3": "everetti",
            "nigrorum2": "nigrorum",
            "nigrorum3": "nigrorum",
            "nigrorum4": "nigrorum",
            "palpebro": "palpebrosus"
        },
        "species_constrained_lineage_map": {
            "simplex": [
                "simplex1",
                "simplex2",
                "simplex3"
            ],
            "erythropleurus": [
                "erythro1",
                "erythro2"
            ],
            "everetti": [
                "everetti1",
                "everetti2",
                "everetti3"
            ],
            "nigrorum": [
                "nigrorum2",
                "nigrorum3",
                "nigrorum4"
            ],
            "palpebrosus": [
                "palpebro"
            ]
        }
    },
    "num_partitions": 53822103,
    "num_partitions_in_confidence_interval": 14,
    "report_mle_only": false,
    "report_constrained_probability_threshold": null,
    "report_constrained_cumulative_probability_threshold": 0.95,
    "partitions":  [
        {
            "lineage_species_name_map": {
                "alani1": "DelineatedSp002",
                "alani2": "DelineatedSp002",
                "alani3": "DelineatedSp002",
                "erythro1": "erythropleurus",
                "erythro2": "erythropleurus",
                "everetti1": "everetti",
                "everetti2": "everetti",
                "everetti3": "everetti",
                "japonicus": "DelineatedSp002",
                "meyeni1": "DelineatedSp002",
                "meyeni2": "DelineatedSp002",
                "meyeni3": "DelineatedSp002",
                "montanus1": "DelineatedSp002",
                "montanus2": "DelineatedSp002",
                "montanus3": "DelineatedSp002",
                "montanus4": "DelineatedSp002",
                "montanus5": "DelineatedSp002",
                "montanus6": "DelineatedSp002",
                "montanus7": "DelineatedSp002",
                "montanus8": "DelineatedSp002",
                "nigrorum1": "DelineatedSp001",
                "nigrorum2": "nigrorum",
                "nigrorum3": "nigrorum",
                "nigrorum4": "nigrorum",
                "nigrorum5": "DelineatedSp002",
                "palpebro": "palpebrosus",
                "simplex1": "simplex",
                "simplex2": "simplex",
                "simplex3": "simplex"
            },
            "species_leafsets": {
                "DelineatedSp001": [
                    "nigrorum1"
                ],
                "DelineatedSp002": [
                    "japonicus",
                    "montanus2",
                    "montanus6",
                    "alani2",
                    "nigrorum5",
                    "montanus5",
                    "meyeni2",
                    "montanus7",
                    "alani1",
                    "alani3",
                    "montanus4",
                    "montanus1",
                    "meyeni1",
                    "meyeni3",
                    "montanus3",
                    "montanus8"
                ],
                "erythropleurus": [
                    "erythro2",
                    "erythro1"
                ],
                "everetti": [
                    "everetti3",
                    "everetti1",
                    "everetti2"
                ],
                "nigrorum": [
                    "nigrorum4",
                    "nigrorum3",
                    "nigrorum2"
                ],
                "palpebrosus": [
                    "palpebro"
                ],
                "simplex": [
                    "simplex2",
                    "simplex1",
                    "simplex3"
                ]
            },
            "constrained_probability": 0.37191378724226637,
            "constrained_cumulative_probability": 0.37191378724226637,
            "is_in_confidence_interval": true,
            "unconstrained_probability": 0.1174121452034233,
            "unconstrained_cumulative_probability": 0.1174121452034233
        },</code></pre>
<p>The <code>.trees</code> files created upon completion of a Delineate run may be opened in <a href="https://beast.community/figtree">Figtree</a> and contain additional information that color branches according to constraints. For run 1 53,822,103 partitions were examined of which 14 were in the confidence interval. Using the assignments described in <code>ZosteropsDELINEATE1_assignments.tsv</code> the best set of partitions exhibited a constrained probability of 0.3719 and an unconstrained probability of 0.1174. For this best supported partition the nominate <em>Z. nigrorum nigrorum</em> subspecies on Panay was delimited as its own species distinct from the other <em>Z. nigrorum</em> populations (<code>DelineatedSp001</code>) and <em>Z. japonicus</em>, <em>Z. japonicus alani</em>, <em>Z. meyeni</em>, <em>Z. montanus</em>, and <em>Z. nigrorum</em> <em>catarmenensis</em> all fell within a single delimited species (<code>DelineatedSp002</code>). Below is the tree edited for readability in <a href="https://beast.community/figtree">Figtree</a>.</p>
<p><img src="images/ZosteropsDELINEATE1_tree1.png" class="img-fluid"></p>
<p>The species limits for the top 5 delimitation schemes were recorded manually from the <code>.json</code> file and entered in a spreadsheet. Species delimitation from the first 5 best supported partitions for run 1. The top 5 out of a total 53,822,103 partitions for run 1 in Delineate have a constrained cumulative probability of 0.8007 and an unconstrained cumulative probability of 0.2528.</p>
<div class="cell" data-label="Species delimitation from the 5 best supoprted partitions in run 1.">
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tab-delimited file without headers</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"Data/Delineate/ZosteropsDELINEATE1_summary.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert the data frame to a gt table</span></span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE1_summary <span class="ot">&lt;-</span> data <span class="sc">%&gt;%</span></span>
<span id="cb71-6"><a href="#cb71-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb71-7"><a href="#cb71-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb71-8"><a href="#cb71-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Zosterops DELINEATE run1 Summary"</span></span>
<span id="cb71-9"><a href="#cb71-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb71-10"><a href="#cb71-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb71-11"><a href="#cb71-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">.list =</span> <span class="cn">NULL</span>  <span class="co"># Set column labels to NULL to remove them</span></span>
<span id="cb71-12"><a href="#cb71-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb71-13"><a href="#cb71-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb71-14"><a href="#cb71-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">vars</span>(<span class="fu">where</span>(is.numeric)), <span class="co"># Format numeric columns</span></span>
<span id="cb71-15"><a href="#cb71-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb71-16"><a href="#cb71-16" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb71-17"><a href="#cb71-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_width</span>(</span>
<span id="cb71-18"><a href="#cb71-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">everything</span>() <span class="sc">~</span> <span class="fu">px</span>(<span class="dv">130</span>)  <span class="co"># Set all columns to the same width (130 pixels)</span></span>
<span id="cb71-19"><a href="#cb71-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb71-20"><a href="#cb71-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_options</span>(</span>
<span id="cb71-21"><a href="#cb71-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.width =</span> <span class="fu">pct</span>(<span class="dv">100</span>), <span class="co"># Set table width</span></span>
<span id="cb71-22"><a href="#cb71-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.font.size =</span> <span class="dv">12</span>,   <span class="co"># Set font size</span></span>
<span id="cb71-23"><a href="#cb71-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.align =</span> <span class="st">"left"</span>     <span class="co"># Align text to the left</span></span>
<span id="cb71-24"><a href="#cb71-24" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print and save the table</span></span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE1_summary</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="ayakcwwfyk" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#ayakcwwfyk table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#ayakcwwfyk thead, #ayakcwwfyk tbody, #ayakcwwfyk tfoot, #ayakcwwfyk tr, #ayakcwwfyk td, #ayakcwwfyk th {
  border-style: none;
}

#ayakcwwfyk p {
  margin: 0;
  padding: 0;
}

#ayakcwwfyk .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: 0;
  margin-right: auto;
  color: #333333;
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#ayakcwwfyk .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#ayakcwwfyk .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#ayakcwwfyk .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#ayakcwwfyk .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ayakcwwfyk .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ayakcwwfyk .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#ayakcwwfyk .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#ayakcwwfyk .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#ayakcwwfyk .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#ayakcwwfyk .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#ayakcwwfyk .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#ayakcwwfyk .gt_spanner_row {
  border-bottom-style: hidden;
}

#ayakcwwfyk .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#ayakcwwfyk .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#ayakcwwfyk .gt_from_md > :first-child {
  margin-top: 0;
}

#ayakcwwfyk .gt_from_md > :last-child {
  margin-bottom: 0;
}

#ayakcwwfyk .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#ayakcwwfyk .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#ayakcwwfyk .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#ayakcwwfyk .gt_row_group_first td {
  border-top-width: 2px;
}

#ayakcwwfyk .gt_row_group_first th {
  border-top-width: 2px;
}

#ayakcwwfyk .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ayakcwwfyk .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#ayakcwwfyk .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#ayakcwwfyk .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ayakcwwfyk .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#ayakcwwfyk .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#ayakcwwfyk .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#ayakcwwfyk .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#ayakcwwfyk .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#ayakcwwfyk .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ayakcwwfyk .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ayakcwwfyk .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#ayakcwwfyk .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#ayakcwwfyk .gt_left {
  text-align: left;
}

#ayakcwwfyk .gt_center {
  text-align: center;
}

#ayakcwwfyk .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#ayakcwwfyk .gt_font_normal {
  font-weight: normal;
}

#ayakcwwfyk .gt_font_bold {
  font-weight: bold;
}

#ayakcwwfyk .gt_font_italic {
  font-style: italic;
}

#ayakcwwfyk .gt_super {
  font-size: 65%;
}

#ayakcwwfyk .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#ayakcwwfyk .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#ayakcwwfyk .gt_indent_1 {
  text-indent: 5px;
}

#ayakcwwfyk .gt_indent_2 {
  text-indent: 10px;
}

#ayakcwwfyk .gt_indent_3 {
  text-indent: 15px;
}

#ayakcwwfyk .gt_indent_4 {
  text-indent: 20px;
}

#ayakcwwfyk .gt_indent_5 {
  text-indent: 25px;
}

#ayakcwwfyk .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#ayakcwwfyk div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="8" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops DELINEATE run1 Summary</th>
</tr>
<tr class="gt_col_headings even">
<th id="Tree.taxa" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tree.taxa</th>
<th id="Putative.species" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Putative.species</th>
<th id="Constraint" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Constraint</th>
<th id="X" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X</th>
<th id="X.1" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.1</th>
<th id="X.2" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.2</th>
<th id="X.3" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.3</th>
<th id="X.4" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.4</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">japonicus</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">alani2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus6</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus7</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus8</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 2</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">simplex2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">erythro1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">erythro2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">everetti2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 3</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 3</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">palpebro</td>
<td class="gt_row gt_left" headers="Putative.species">Z. palpebrosus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.1">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.2">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.3">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.4">Z. palpebrosus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa"></td>
<td class="gt_row gt_left" headers="Putative.species"></td>
<td class="gt_row gt_right" headers="Constraint">NA</td>
<td class="gt_row gt_left" headers="X">0.1174</td>
<td class="gt_row gt_left" headers="X.1">0.0901</td>
<td class="gt_row gt_left" headers="X.2">0.0218</td>
<td class="gt_row gt_left" headers="X.3">0.0133</td>
<td class="gt_row gt_left" headers="X.4">0.0102</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb74"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb74-2"><a href="#cb74-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(ZosteropsDELINEATE1_summary, <span class="st">"ZosteropsDELINEATE1_summary.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-2-1" class="level3">
<h3 class="anchored" data-anchor-id="run-2-1">Run 2</h3>
<section id="constraints-1" class="level4">
<h4 class="anchored" data-anchor-id="constraints-1">Constraints</h4>
<p>The starting tree is provided from the same file as in Delineate run 1 (<code>ZosteropsPICL_tree.nex</code>). The taxon assignments and constraints are provided in the file <code>ZosteropsDELINEATE2_assignments.tsv</code> differ in that the lineages <code>montanus2</code>, <code>montanus3</code>, <code>montanus4</code>, <code>montanus5</code>, <code>montanus6</code>, and <code>montanus7</code> are constrained to inclusion in the <em>Z. montanus</em> species. Below is the species tree <code>ZosteropsPICL_tree.nex</code> (the same tree used in all Delineate runs).</p>
<pre><code>#NEXUS
begin trees;
    tree tree_2 = [&amp;R] (palpebro:4.495118,(((((montanus1:0.000000001,((japonicus:0.000000001,(alani3:0.000000001,(alani1:0.000000001,alani2:0.000000001):0.000000001):0.000000001):0.000000001,(meyeni1:0.000000001,(meyeni2:0.000000001,meyeni3:0.000000001):0.000000001):0.000000001):0.000000001):0.122653,(montanus8:0.094776,((nigrorum5:0.000000001,(montanus2:0.000000001,montanus3:0.000000001):0.000000001):0.000003,((montanus4:0.000000001,montanus5:0.000000001):0.000000001,(montanus6:0.000000001,montanus7:0.000000001):0.000000001):0.000003):0.094772):0.027878):1.944734,(simplex1:0.000019,(simplex2:0.000000001,simplex3:0.000000001):0.000019):2.067368):0.667965,(erythro1:0.000072,erythro2:0.000072):2.73528):1.424642,((everetti1:0.000029,(everetti2:0.000000001,everetti3:0.000000001):0.000029):0.920545,(nigrorum1:0.700863,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):0.000005):0.371394):0.219711):3.239421):0.335124);
end;</code></pre>
<p>And what follows here is the taxon assignments file for run 2, <code>ZosteropsDELINEATE2_assignments.tsv</code>.</p>
<pre><code>lineage species status
japonicus   japonicus   0
alani1  alani   0
alani2  alani   0
alani3  alani   0
meyeni1 meyeni  0
meyeni2 meyeni  0
meyeni3 meyeni  0
montanus1   montanus    0
montanus2   montanus    1
montanus3   montanus    1
montanus4   montanus    1
montanus5   montanus    1
montanus6   montanus    1
montanus7   montanus    1
montanus8   montanus    0
simplex1    simplex 1
simplex2    simplex 1
simplex3    simplex 1
erythro1    erythropleurus  1
erythro2    erythropleurus  1
everetti1   everetti    1
everetti2   everetti    1
everetti3   everetti    1
nigrorum1   nigrorum    0
nigrorum2   nigrorum    1
nigrorum3   nigrorum    1
nigrorum4   nigrorum    1
nigrorum5   nigrorum    0
palpebro    palpebrosus 1</code></pre>
</section>
<section id="run-5" class="level4">
<h4 class="anchored" data-anchor-id="run-5">Run</h4>
<p>Run Delineate for run 2 exactly as in run 1. Start the Delineate environment in Anaconda. Open a terminal session. Navigate to the directory with the <code>ZosteropsPICL_tree.nex</code> and <code>ZosteropsDELINEATE2_assignments.tsv</code> using the <code>cd</code> command. Execute the following command to run Delineate specifying the run parameters and the tree and assignment files.</p>
<pre><code>delineate-estimate partitions -t ZosteropsPICL_tree.nex -c ZosteropsDELINEATE2_assignments.tsv -P 0.95 -u</code></pre>
</section>
<section id="results-3" class="level4">
<h4 class="anchored" data-anchor-id="results-3">Results</h4>
<p>These 3 files will be created upon completion of run 2.</p>
<ul>
<li><p><code>ZosteropsDELINEATE2_assignments.delimitation-results.json</code></p></li>
<li><p><code>ZosteropsDELINEATE2_assignments.delimitation-results.trees</code></p></li>
<li><p><code>ZosteropsDELINEATE2_assignments.delimitation-results.trunc.trees</code></p></li>
</ul>
<p>The speciation completion rate for run 2 is 0.7930 and the lineage birth rate is 1.3203. There are a total of 206,542 partitions in run 2 of which 9 are within the confidence interval. Below is the best tree with a constrained probability of 0.4185 and an unconstrained probability of 0.1247.</p>
<p><img src="images/ZosteropsDELINEATE2_tree1.png" class="img-fluid"></p>
<p>The species limits for the top 5 delimitation schemes were recorded manually from the <code>.json</code> file and entered in a spreadsheet. Below are the summarized results for the best 5 out of 206,542 partitions together with a total constrained cumulative probability of 0.8473 and an unconstrained cumulative probability of 0.2524.</p>
<div class="cell" data-label="Species delimitation from the 5 best supoprted partitions in run 2.">
<div class="sourceCode cell-code" id="cb78"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tab-delimited file without headers</span></span>
<span id="cb78-2"><a href="#cb78-2" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"Data/Delineate/ZosteropsDELINEATE2_summary.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb78-3"><a href="#cb78-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb78-4"><a href="#cb78-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert the data frame to a gt table</span></span>
<span id="cb78-5"><a href="#cb78-5" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE2_summary <span class="ot">&lt;-</span> data <span class="sc">%&gt;%</span></span>
<span id="cb78-6"><a href="#cb78-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb78-7"><a href="#cb78-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb78-8"><a href="#cb78-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Zosterops DELINEATE run2 Summary"</span></span>
<span id="cb78-9"><a href="#cb78-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb78-10"><a href="#cb78-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb78-11"><a href="#cb78-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">.list =</span> <span class="cn">NULL</span>  <span class="co"># Set column labels to NULL to remove them</span></span>
<span id="cb78-12"><a href="#cb78-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb78-13"><a href="#cb78-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb78-14"><a href="#cb78-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">vars</span>(<span class="fu">where</span>(is.numeric)), <span class="co"># Format numeric columns</span></span>
<span id="cb78-15"><a href="#cb78-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb78-16"><a href="#cb78-16" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb78-17"><a href="#cb78-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_width</span>(</span>
<span id="cb78-18"><a href="#cb78-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">everything</span>() <span class="sc">~</span> <span class="fu">px</span>(<span class="dv">130</span>)  <span class="co"># Set all columns to the same width (130 pixels)</span></span>
<span id="cb78-19"><a href="#cb78-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb78-20"><a href="#cb78-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_options</span>(</span>
<span id="cb78-21"><a href="#cb78-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.width =</span> <span class="fu">pct</span>(<span class="dv">100</span>), <span class="co"># Set table width</span></span>
<span id="cb78-22"><a href="#cb78-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.font.size =</span> <span class="dv">12</span>,   <span class="co"># Set font size</span></span>
<span id="cb78-23"><a href="#cb78-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.align =</span> <span class="st">"left"</span>     <span class="co"># Align text to the left</span></span>
<span id="cb78-24"><a href="#cb78-24" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb80"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print and save the table</span></span>
<span id="cb80-2"><a href="#cb80-2" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE2_summary</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="tosxfrwbkj" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#tosxfrwbkj table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#tosxfrwbkj thead, #tosxfrwbkj tbody, #tosxfrwbkj tfoot, #tosxfrwbkj tr, #tosxfrwbkj td, #tosxfrwbkj th {
  border-style: none;
}

#tosxfrwbkj p {
  margin: 0;
  padding: 0;
}

#tosxfrwbkj .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: 0;
  margin-right: auto;
  color: #333333;
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#tosxfrwbkj .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#tosxfrwbkj .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#tosxfrwbkj .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#tosxfrwbkj .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tosxfrwbkj .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tosxfrwbkj .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#tosxfrwbkj .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#tosxfrwbkj .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#tosxfrwbkj .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#tosxfrwbkj .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#tosxfrwbkj .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#tosxfrwbkj .gt_spanner_row {
  border-bottom-style: hidden;
}

#tosxfrwbkj .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#tosxfrwbkj .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#tosxfrwbkj .gt_from_md > :first-child {
  margin-top: 0;
}

#tosxfrwbkj .gt_from_md > :last-child {
  margin-bottom: 0;
}

#tosxfrwbkj .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#tosxfrwbkj .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#tosxfrwbkj .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#tosxfrwbkj .gt_row_group_first td {
  border-top-width: 2px;
}

#tosxfrwbkj .gt_row_group_first th {
  border-top-width: 2px;
}

#tosxfrwbkj .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tosxfrwbkj .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#tosxfrwbkj .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#tosxfrwbkj .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tosxfrwbkj .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#tosxfrwbkj .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#tosxfrwbkj .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#tosxfrwbkj .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#tosxfrwbkj .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#tosxfrwbkj .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tosxfrwbkj .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tosxfrwbkj .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#tosxfrwbkj .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#tosxfrwbkj .gt_left {
  text-align: left;
}

#tosxfrwbkj .gt_center {
  text-align: center;
}

#tosxfrwbkj .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#tosxfrwbkj .gt_font_normal {
  font-weight: normal;
}

#tosxfrwbkj .gt_font_bold {
  font-weight: bold;
}

#tosxfrwbkj .gt_font_italic {
  font-style: italic;
}

#tosxfrwbkj .gt_super {
  font-size: 65%;
}

#tosxfrwbkj .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#tosxfrwbkj .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#tosxfrwbkj .gt_indent_1 {
  text-indent: 5px;
}

#tosxfrwbkj .gt_indent_2 {
  text-indent: 10px;
}

#tosxfrwbkj .gt_indent_3 {
  text-indent: 15px;
}

#tosxfrwbkj .gt_indent_4 {
  text-indent: 20px;
}

#tosxfrwbkj .gt_indent_5 {
  text-indent: 25px;
}

#tosxfrwbkj .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#tosxfrwbkj div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="8" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops DELINEATE run2 Summary</th>
</tr>
<tr class="gt_col_headings even">
<th id="Tree.taxa" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tree.taxa</th>
<th id="Putative.species" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Putative.species</th>
<th id="Constraint" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Constraint</th>
<th id="X" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X</th>
<th id="X.1" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.1</th>
<th id="X.2" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.2</th>
<th id="X.3" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.3</th>
<th id="X.4" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.4</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">japonicus</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">alani2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus6</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus7</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus8</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">simplex2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">erythro1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">erythro2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">everetti2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">palpebro</td>
<td class="gt_row gt_left" headers="Putative.species">Z. palpebrosus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.1">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.2">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.3">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.4">Z. palpebrosus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa"></td>
<td class="gt_row gt_left" headers="Putative.species"></td>
<td class="gt_row gt_right" headers="Constraint">NA</td>
<td class="gt_row gt_left" headers="X">0.1247</td>
<td class="gt_row gt_left" headers="X.1">0.0823</td>
<td class="gt_row gt_left" headers="X.2">0.0192</td>
<td class="gt_row gt_left" headers="X.3">0.0158</td>
<td class="gt_row gt_left" headers="X.4">0.0104</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb81"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb81-2"><a href="#cb81-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(ZosteropsDELINEATE2_summary, <span class="st">"ZosteropsDELINEATE2_summary.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-3-1" class="level3">
<h3 class="anchored" data-anchor-id="run-3-1">Run 3</h3>
<section id="constraints-2" class="level4">
<h4 class="anchored" data-anchor-id="constraints-2">Constraints</h4>
<p>The starting tree is provided from the same file as in Delineate run 1 (<code>ZosteropsPICL_tree.nex</code>). The taxon assignments and constraints are provided in the file <code>ZosteropsDELINEATE3_assignments.tsv</code> differ in that the lineages <code>montanus2</code>, <code>montanus3</code>, <code>montanus4</code>, <code>montanus5</code>, <code>montanus6</code>, and <code>montanus7</code> are constrained to the <em>Z. montanus</em> species and the <em>Z. meyeni</em> lineages <code>meyeni1</code>, <code>meyeni2</code>, and <code>meyeni3</code> are constrained to the <em>Z. meyeni</em> species. Below is the species tree <code>ZosteropsPICL_tree.nex</code> (the same tree used in all Delineate runs).</p>
<pre><code>#NEXUS
begin trees;
    tree tree_2 = [&amp;R] (palpebro:4.495118,(((((montanus1:0.000000001,((japonicus:0.000000001,(alani3:0.000000001,(alani1:0.000000001,alani2:0.000000001):0.000000001):0.000000001):0.000000001,(meyeni1:0.000000001,(meyeni2:0.000000001,meyeni3:0.000000001):0.000000001):0.000000001):0.000000001):0.122653,(montanus8:0.094776,((nigrorum5:0.000000001,(montanus2:0.000000001,montanus3:0.000000001):0.000000001):0.000003,((montanus4:0.000000001,montanus5:0.000000001):0.000000001,(montanus6:0.000000001,montanus7:0.000000001):0.000000001):0.000003):0.094772):0.027878):1.944734,(simplex1:0.000019,(simplex2:0.000000001,simplex3:0.000000001):0.000019):2.067368):0.667965,(erythro1:0.000072,erythro2:0.000072):2.73528):1.424642,((everetti1:0.000029,(everetti2:0.000000001,everetti3:0.000000001):0.000029):0.920545,(nigrorum1:0.700863,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):0.000005):0.371394):0.219711):3.239421):0.335124);
end;</code></pre>
<p>And what follows here is the taxon assignments file for run 3, <code>ZosteropsDELINEATE3_assignments.tsv</code>.</p>
<pre><code>lineage species status
japonicus   japonicus   0
alani1  alani   0
alani2  alani   0
alani3  alani   0
meyeni1 meyeni  1
meyeni2 meyeni  1
meyeni3 meyeni  1
montanus1   montanus    0
montanus2   montanus    1
montanus3   montanus    1
montanus4   montanus    1
montanus5   montanus    1
montanus6   montanus    1
montanus7   montanus    1
montanus8   montanus    0
simplex1    simplex 1
simplex2    simplex 1
simplex3    simplex 1
erythro1    erythropleurus  1
erythro2    erythropleurus  1
everetti1   everetti    1
everetti2   everetti    1
everetti3   everetti    1
nigrorum1   nigrorum    0
nigrorum2   nigrorum    1
nigrorum3   nigrorum    1
nigrorum4   nigrorum    1
nigrorum5   nigrorum    0
palpebro    palpebrosus 1</code></pre>
</section>
<section id="run-6" class="level4">
<h4 class="anchored" data-anchor-id="run-6">Run</h4>
<p>Run Delineate for run 3 exactly as in runs 1 and 2. Start the Delineate environment in Anaconda. Open a terminal session. Navigate to the directory with the <code>ZosteropsPICL_tree.nex</code> and <code>ZosteropsDELINEATE3_assignments.tsv</code> using the <code>cd</code> command. Execute the following command to run Delineate specifying the run parameters and the tree and assignment files.</p>
<pre><code>delineate-estimate partitions -t ZosteropsPICL_tree.nex -c ZosteropsDELINEATE3_assignments.tsv -P 0.95 -u</code></pre>
</section>
<section id="results-4" class="level4">
<h4 class="anchored" data-anchor-id="results-4">Results</h4>
<p>These 3 files will be created upon completion of run 3.</p>
<ul>
<li><p><code>ZosteropsDELINEATE3_assignments.delimitation-results.json</code></p></li>
<li><p><code>ZosteropsDELINEATE3_assignments.delimitation-results.trees</code></p></li>
<li><p><code>ZosteropsDELINEATE3_assignments.delimitation-results.trunc.trees</code></p></li>
</ul>
<p>The speciation completion rate for run 3 is 1.2515 and the lineage birth rate is 1.3202. There are a total of 20,070 partitions in run 3 of which 6 are within the confidence interval. Below is the best tree with a constrained probability of 0.3700 and an unconstrained probability of 0.0239.</p>
<p><img src="images/ZosteropsDELINEATE3_tree1.png" class="img-fluid"></p>
<p>The species limits for the top 5 delimitation schemes were recorded manually from the <code>.json</code> file and entered in a spreadsheet. Below are the summarized results for the best 5 out of 20,070 partitions together with a total constrained cumulative probability of 0.9146 and an unconstrained cumulative probability of 0.0591.</p>
<div class="cell" data-label="Species delimitation from the 5 best supoprted partitions in run 3.">
<div class="sourceCode cell-code" id="cb85"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tab-delimited file without headers</span></span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"Data/Delineate/ZosteropsDELINEATE3_summary.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-4"><a href="#cb85-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert the data frame to a gt table</span></span>
<span id="cb85-5"><a href="#cb85-5" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE3_summary <span class="ot">&lt;-</span> data <span class="sc">%&gt;%</span></span>
<span id="cb85-6"><a href="#cb85-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb85-7"><a href="#cb85-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb85-8"><a href="#cb85-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Zosterops DELINEATE run3 Summary"</span></span>
<span id="cb85-9"><a href="#cb85-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb85-10"><a href="#cb85-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb85-11"><a href="#cb85-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">.list =</span> <span class="cn">NULL</span>  <span class="co"># Set column labels to NULL to remove them</span></span>
<span id="cb85-12"><a href="#cb85-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb85-13"><a href="#cb85-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb85-14"><a href="#cb85-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">vars</span>(<span class="fu">where</span>(is.numeric)), <span class="co"># Format numeric columns</span></span>
<span id="cb85-15"><a href="#cb85-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb85-16"><a href="#cb85-16" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb85-17"><a href="#cb85-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_width</span>(</span>
<span id="cb85-18"><a href="#cb85-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">everything</span>() <span class="sc">~</span> <span class="fu">px</span>(<span class="dv">130</span>)  <span class="co"># Set all columns to the same width (130 pixels)</span></span>
<span id="cb85-19"><a href="#cb85-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb85-20"><a href="#cb85-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_options</span>(</span>
<span id="cb85-21"><a href="#cb85-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.width =</span> <span class="fu">pct</span>(<span class="dv">100</span>), <span class="co"># Set table width</span></span>
<span id="cb85-22"><a href="#cb85-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.font.size =</span> <span class="dv">12</span>,   <span class="co"># Set font size</span></span>
<span id="cb85-23"><a href="#cb85-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.align =</span> <span class="st">"left"</span>     <span class="co"># Align text to the left</span></span>
<span id="cb85-24"><a href="#cb85-24" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb87"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb87-1"><a href="#cb87-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print and save the table</span></span>
<span id="cb87-2"><a href="#cb87-2" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE3_summary</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="frwymladfx" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#frwymladfx table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#frwymladfx thead, #frwymladfx tbody, #frwymladfx tfoot, #frwymladfx tr, #frwymladfx td, #frwymladfx th {
  border-style: none;
}

#frwymladfx p {
  margin: 0;
  padding: 0;
}

#frwymladfx .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: 0;
  margin-right: auto;
  color: #333333;
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#frwymladfx .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#frwymladfx .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#frwymladfx .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#frwymladfx .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#frwymladfx .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#frwymladfx .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#frwymladfx .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#frwymladfx .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#frwymladfx .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#frwymladfx .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#frwymladfx .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#frwymladfx .gt_spanner_row {
  border-bottom-style: hidden;
}

#frwymladfx .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#frwymladfx .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#frwymladfx .gt_from_md > :first-child {
  margin-top: 0;
}

#frwymladfx .gt_from_md > :last-child {
  margin-bottom: 0;
}

#frwymladfx .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#frwymladfx .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#frwymladfx .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#frwymladfx .gt_row_group_first td {
  border-top-width: 2px;
}

#frwymladfx .gt_row_group_first th {
  border-top-width: 2px;
}

#frwymladfx .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#frwymladfx .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#frwymladfx .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#frwymladfx .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#frwymladfx .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#frwymladfx .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#frwymladfx .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#frwymladfx .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#frwymladfx .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#frwymladfx .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#frwymladfx .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#frwymladfx .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#frwymladfx .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#frwymladfx .gt_left {
  text-align: left;
}

#frwymladfx .gt_center {
  text-align: center;
}

#frwymladfx .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#frwymladfx .gt_font_normal {
  font-weight: normal;
}

#frwymladfx .gt_font_bold {
  font-weight: bold;
}

#frwymladfx .gt_font_italic {
  font-style: italic;
}

#frwymladfx .gt_super {
  font-size: 65%;
}

#frwymladfx .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#frwymladfx .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#frwymladfx .gt_indent_1 {
  text-indent: 5px;
}

#frwymladfx .gt_indent_2 {
  text-indent: 10px;
}

#frwymladfx .gt_indent_3 {
  text-indent: 15px;
}

#frwymladfx .gt_indent_4 {
  text-indent: 20px;
}

#frwymladfx .gt_indent_5 {
  text-indent: 25px;
}

#frwymladfx .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#frwymladfx div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="8" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops DELINEATE run3 Summary</th>
</tr>
<tr class="gt_col_headings even">
<th id="Tree.taxa" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tree.taxa</th>
<th id="Putative.species" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Putative.species</th>
<th id="Constraint" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Constraint</th>
<th id="X" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X</th>
<th id="X.1" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.1</th>
<th id="X.2" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.2</th>
<th id="X.3" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.3</th>
<th id="X.4" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.4</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">japonicus</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">alani2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus6</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus7</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus8</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">simplex2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">erythro1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">erythro2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">everetti2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.1">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.2">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. montanus</td>
<td class="gt_row gt_left" headers="X.1">Z. montanus</td>
<td class="gt_row gt_left" headers="X.2">Z. montanus</td>
<td class="gt_row gt_left" headers="X.3">Z. montanus</td>
<td class="gt_row gt_left" headers="X.4">Z. montanus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">palpebro</td>
<td class="gt_row gt_left" headers="Putative.species">Z. palpebrosus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.1">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.2">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.3">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.4">Z. palpebrosus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa"></td>
<td class="gt_row gt_left" headers="Putative.species"></td>
<td class="gt_row gt_right" headers="Constraint">NA</td>
<td class="gt_row gt_left" headers="X">0.0239</td>
<td class="gt_row gt_left" headers="X.1">0.0145</td>
<td class="gt_row gt_left" headers="X.2">0.0082</td>
<td class="gt_row gt_left" headers="X.3">0.0077</td>
<td class="gt_row gt_left" headers="X.4">0.0047</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb88"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb88-2"><a href="#cb88-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(ZosteropsDELINEATE3_summary, <span class="st">"ZosteropsDELINEATE3_summary.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-4-1" class="level3">
<h3 class="anchored" data-anchor-id="run-4-1">Run 4</h3>
<section id="constraints-3" class="level4">
<h4 class="anchored" data-anchor-id="constraints-3">Constraints</h4>
<p>The starting tree is provided from the same file as in Delineate run 1 (<code>ZosteropsPICL_tree.nex</code>). The taxon assignments and constraints are provided in the file <code>ZosteropsDELINEATE4_assignments.tsv</code> differ in that the lineages <code>meyeni1</code>, <code>meyeni2</code>, and <code>meyeni3</code> are constrained to the <em>Z. meyeni</em> species while all the other lineages in the <em>Z. japonicus</em>, <em>Z. japonicus alani</em>, <em>Z. meyeni</em>, and <em>Z. montanus</em> clade are unconstrained. In addition, as in the other runs, <em>Z. nigrorum nigrorum</em> (<code>nigrorum1</code>), <em>Z. nigrorum catarmanensis</em> (<code>nigrorum5</code>) are also unconstrained. Below is the species tree <code>ZosteropsPICL_tree.nex</code> (the same tree used in all Delineate runs).</p>
<pre><code>#NEXUS
begin trees;
    tree tree_2 = [&amp;R] (palpebro:4.495118,(((((montanus1:0.000000001,((japonicus:0.000000001,(alani3:0.000000001,(alani1:0.000000001,alani2:0.000000001):0.000000001):0.000000001):0.000000001,(meyeni1:0.000000001,(meyeni2:0.000000001,meyeni3:0.000000001):0.000000001):0.000000001):0.000000001):0.122653,(montanus8:0.094776,((nigrorum5:0.000000001,(montanus2:0.000000001,montanus3:0.000000001):0.000000001):0.000003,((montanus4:0.000000001,montanus5:0.000000001):0.000000001,(montanus6:0.000000001,montanus7:0.000000001):0.000000001):0.000003):0.094772):0.027878):1.944734,(simplex1:0.000019,(simplex2:0.000000001,simplex3:0.000000001):0.000019):2.067368):0.667965,(erythro1:0.000072,erythro2:0.000072):2.73528):1.424642,((everetti1:0.000029,(everetti2:0.000000001,everetti3:0.000000001):0.000029):0.920545,(nigrorum1:0.700863,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):0.000005):0.371394):0.219711):3.239421):0.335124);
end;</code></pre>
<p>And what follows here is the taxon assignments file for run 4, <code>ZosteropsDELINEATE4_assignments.tsv</code>.</p>
<pre><code>lineage species status
japonicus   japonicus   0
alani1  alani   0
alani2  alani   0
alani3  alani   0
meyeni1 meyeni  1
meyeni2 meyeni  1
meyeni3 meyeni  1
montanus1   montanus    0
montanus2   montanus    0
montanus3   montanus    0
montanus4   montanus    0
montanus5   montanus    0
montanus6   montanus    0
montanus7   montanus    0
montanus8   montanus    0
simplex1    simplex 1
simplex2    simplex 1
simplex3    simplex 1
erythro1    erythropleurus  1
erythro2    erythropleurus  1
everetti1   everetti    1
everetti2   everetti    1
everetti3   everetti    1
nigrorum1   nigrorum    0
nigrorum2   nigrorum    1
nigrorum3   nigrorum    1
nigrorum4   nigrorum    1
nigrorum5   nigrorum    0
palpebro    palpebrosus 1</code></pre>
</section>
<section id="run-7" class="level4">
<h4 class="anchored" data-anchor-id="run-7">Run</h4>
<p>Run Delineate for run 4 exactly as in runs 1-3. Start the Delineate environment in Anaconda. Open a terminal session. Navigate to the directory with the <code>ZosteropsPICL_tree.nex</code> and <code>ZosteropsDELINEATE4_assignments.tsv</code> using the <code>cd</code> command. Execute the following command to run Delineate specifying the run parameters and the tree and assignment files.</p>
<pre><code>delineate-estimate partitions -t ZosteropsPICL_tree.nex -c ZosteropsDELINEATE4_assignments.tsv -P 0.95 -u</code></pre>
</section>
<section id="results-5" class="level4">
<h4 class="anchored" data-anchor-id="results-5">Results</h4>
<p>These 3 files will be created upon completion of run 4.</p>
<ul>
<li><p><code>ZosteropsDELINEATE4_assignments.delimitation-results.json</code></p></li>
<li><p><code>ZosteropsDELINEATE4_assignments.delimitation-results.trees</code></p></li>
<li><p><code>ZosteropsDELINEATE4_assignments.delimitation-results.trunc.trees</code></p></li>
</ul>
<p>The speciation completion rate for run 4 is 0.7930 and the lineage birth rate is 1.3202. There are a total of 5,680,619 partitions in run 4 of which 9 are within the confidence interval. Below is the best tree with a constrained probability of 0.4185 and an unconstrained probability of 0.1247.</p>
<p><img src="images/ZosteropsDELINEATE4_tree1.png" class="img-fluid"></p>
<p>The species limits for the top 5 delimitation schemes were recorded manually from the <code>.json</code> file and entered in a spreadsheet. Below are the summarized results for the best 5 out of 5,680,619 partitions together with a total constrained cumulative probability of 0.8473 and an unconstrained cumulative probability of 0.2524.</p>
<div class="cell" data-label="Species delimitation from the 5 best supoprted partitions in run 4.">
<div class="sourceCode cell-code" id="cb92"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb92-1"><a href="#cb92-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tab-delimited file without headers</span></span>
<span id="cb92-2"><a href="#cb92-2" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"Data/Delineate/ZosteropsDELINEATE4_summary.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb92-3"><a href="#cb92-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-4"><a href="#cb92-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert the data frame to a gt table</span></span>
<span id="cb92-5"><a href="#cb92-5" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE4_summary <span class="ot">&lt;-</span> data <span class="sc">%&gt;%</span></span>
<span id="cb92-6"><a href="#cb92-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb92-7"><a href="#cb92-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb92-8"><a href="#cb92-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Zosterops DELINEATE run4 Summary"</span></span>
<span id="cb92-9"><a href="#cb92-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb92-10"><a href="#cb92-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb92-11"><a href="#cb92-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">.list =</span> <span class="cn">NULL</span>  <span class="co"># Set column labels to NULL to remove them</span></span>
<span id="cb92-12"><a href="#cb92-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb92-13"><a href="#cb92-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb92-14"><a href="#cb92-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">vars</span>(<span class="fu">where</span>(is.numeric)), <span class="co"># Format numeric columns</span></span>
<span id="cb92-15"><a href="#cb92-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb92-16"><a href="#cb92-16" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb92-17"><a href="#cb92-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_width</span>(</span>
<span id="cb92-18"><a href="#cb92-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">everything</span>() <span class="sc">~</span> <span class="fu">px</span>(<span class="dv">130</span>)  <span class="co"># Set all columns to the same width (130 pixels)</span></span>
<span id="cb92-19"><a href="#cb92-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb92-20"><a href="#cb92-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_options</span>(</span>
<span id="cb92-21"><a href="#cb92-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.width =</span> <span class="fu">pct</span>(<span class="dv">100</span>), <span class="co"># Set table width</span></span>
<span id="cb92-22"><a href="#cb92-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.font.size =</span> <span class="dv">12</span>,   <span class="co"># Set font size</span></span>
<span id="cb92-23"><a href="#cb92-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.align =</span> <span class="st">"left"</span>     <span class="co"># Align text to the left</span></span>
<span id="cb92-24"><a href="#cb92-24" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb94"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb94-1"><a href="#cb94-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print and save the table</span></span>
<span id="cb94-2"><a href="#cb94-2" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE4_summary</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="rfomizvkxl" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#rfomizvkxl table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#rfomizvkxl thead, #rfomizvkxl tbody, #rfomizvkxl tfoot, #rfomizvkxl tr, #rfomizvkxl td, #rfomizvkxl th {
  border-style: none;
}

#rfomizvkxl p {
  margin: 0;
  padding: 0;
}

#rfomizvkxl .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: 0;
  margin-right: auto;
  color: #333333;
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#rfomizvkxl .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#rfomizvkxl .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#rfomizvkxl .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#rfomizvkxl .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#rfomizvkxl .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rfomizvkxl .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#rfomizvkxl .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#rfomizvkxl .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#rfomizvkxl .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#rfomizvkxl .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#rfomizvkxl .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#rfomizvkxl .gt_spanner_row {
  border-bottom-style: hidden;
}

#rfomizvkxl .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#rfomizvkxl .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#rfomizvkxl .gt_from_md > :first-child {
  margin-top: 0;
}

#rfomizvkxl .gt_from_md > :last-child {
  margin-bottom: 0;
}

#rfomizvkxl .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#rfomizvkxl .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#rfomizvkxl .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#rfomizvkxl .gt_row_group_first td {
  border-top-width: 2px;
}

#rfomizvkxl .gt_row_group_first th {
  border-top-width: 2px;
}

#rfomizvkxl .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#rfomizvkxl .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#rfomizvkxl .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#rfomizvkxl .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rfomizvkxl .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#rfomizvkxl .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#rfomizvkxl .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#rfomizvkxl .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#rfomizvkxl .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rfomizvkxl .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#rfomizvkxl .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#rfomizvkxl .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#rfomizvkxl .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#rfomizvkxl .gt_left {
  text-align: left;
}

#rfomizvkxl .gt_center {
  text-align: center;
}

#rfomizvkxl .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#rfomizvkxl .gt_font_normal {
  font-weight: normal;
}

#rfomizvkxl .gt_font_bold {
  font-weight: bold;
}

#rfomizvkxl .gt_font_italic {
  font-style: italic;
}

#rfomizvkxl .gt_super {
  font-size: 65%;
}

#rfomizvkxl .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#rfomizvkxl .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#rfomizvkxl .gt_indent_1 {
  text-indent: 5px;
}

#rfomizvkxl .gt_indent_2 {
  text-indent: 10px;
}

#rfomizvkxl .gt_indent_3 {
  text-indent: 15px;
}

#rfomizvkxl .gt_indent_4 {
  text-indent: 20px;
}

#rfomizvkxl .gt_indent_5 {
  text-indent: 25px;
}

#rfomizvkxl .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#rfomizvkxl div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="8" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops DELINEATE run4 Summary</th>
</tr>
<tr class="gt_col_headings even">
<th id="Tree.taxa" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tree.taxa</th>
<th id="Putative.species" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Putative.species</th>
<th id="Constraint" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Constraint</th>
<th id="X" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X</th>
<th id="X.1" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.1</th>
<th id="X.2" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.2</th>
<th id="X.3" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.3</th>
<th id="X.4" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.4</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">japonicus</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">alani2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.4">Z. meyeni</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus6</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus7</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus8</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">simplex2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">erythro1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">erythro2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">everetti2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.1">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.2">Z. meyeni</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">palpebro</td>
<td class="gt_row gt_left" headers="Putative.species">Z. palpebrosus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.1">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.2">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.3">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.4">Z. palpebrosus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa"></td>
<td class="gt_row gt_left" headers="Putative.species"></td>
<td class="gt_row gt_right" headers="Constraint">NA</td>
<td class="gt_row gt_left" headers="X">0.1247</td>
<td class="gt_row gt_left" headers="X.1">0.0823</td>
<td class="gt_row gt_left" headers="X.2">0.0192</td>
<td class="gt_row gt_left" headers="X.3">0.0158</td>
<td class="gt_row gt_left" headers="X.4">0.0104</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb95"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb95-1"><a href="#cb95-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb95-2"><a href="#cb95-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(ZosteropsDELINEATE4_summary, <span class="st">"ZosteropsDELINEATE4_summary.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="run-5-1" class="level3">
<h3 class="anchored" data-anchor-id="run-5-1">Run 5</h3>
<section id="constraints-4" class="level4">
<h4 class="anchored" data-anchor-id="constraints-4">Constraints</h4>
<p>The starting tree is provided from the same file as in Delineate run 1 (<code>ZosteropsPICL_tree.nex</code>). The taxon assignments and constraints are provided in the file <code>ZosteropsDELINEATE5_assignments.tsv</code> differ in that the lineages <code>japonicus</code>, <code>alani1</code>, <code>alani2</code>, and <code>alani3</code> are constrained to the <em>Z. japonicus</em> species while all the other lineages in the <em>Z. japonicus</em>, <em>Z. japonicus alani</em>, <em>Z. meyeni</em>, and <em>Z. montanus</em> clade are unconstrained. In addition, as in the other runs, <em>Z. nigrorum nigrorum</em> (<code>nigrorum1</code>), <em>Z. nigrorum catarmanensis</em> (<code>nigrorum5</code>) are also unconstrained. Below is the species tree <code>ZosteropsPICL_tree.nex</code> (the same tree used in all Delineate runs).</p>
<pre><code>#NEXUS
begin trees;
    tree tree_2 = [&amp;R] (palpebro:4.495118,(((((montanus1:0.000000001,((japonicus:0.000000001,(alani3:0.000000001,(alani1:0.000000001,alani2:0.000000001):0.000000001):0.000000001):0.000000001,(meyeni1:0.000000001,(meyeni2:0.000000001,meyeni3:0.000000001):0.000000001):0.000000001):0.000000001):0.122653,(montanus8:0.094776,((nigrorum5:0.000000001,(montanus2:0.000000001,montanus3:0.000000001):0.000000001):0.000003,((montanus4:0.000000001,montanus5:0.000000001):0.000000001,(montanus6:0.000000001,montanus7:0.000000001):0.000000001):0.000003):0.094772):0.027878):1.944734,(simplex1:0.000019,(simplex2:0.000000001,simplex3:0.000000001):0.000019):2.067368):0.667965,(erythro1:0.000072,erythro2:0.000072):2.73528):1.424642,((everetti1:0.000029,(everetti2:0.000000001,everetti3:0.000000001):0.000029):0.920545,(nigrorum1:0.700863,(nigrorum4:0.329469,(nigrorum2:0.329464,nigrorum3:0.329464):0.000005):0.371394):0.219711):3.239421):0.335124);
end;</code></pre>
<p>And what follows here is the taxon assignments file for run 5, <code>ZosteropsDELINEATE5_assignments.tsv</code>.</p>
<pre><code>lineage species status
japonicus   japonicus   1
alani1  japonicus   1
alani2  japonicus   1
alani3  japonicus   1
meyeni1 meyeni  0
meyeni2 meyeni  0
meyeni3 meyeni  0
montanus1   montanus    0
montanus2   montanus    0
montanus3   montanus    0
montanus4   montanus    0
montanus5   montanus    0
montanus6   montanus    0
montanus7   montanus    0
montanus8   montanus    0
simplex1    simplex 1
simplex2    simplex 1
simplex3    simplex 1
erythro1    erythropleurus  1
erythro2    erythropleurus  1
everetti1   everetti    1
everetti2   everetti    1
everetti3   everetti    1
nigrorum1   nigrorum    0
nigrorum2   nigrorum    1
nigrorum3   nigrorum    1
nigrorum4   nigrorum    1
nigrorum5   nigrorum    0
palpebro    palpebrosus 1</code></pre>
</section>
<section id="run-8" class="level4">
<h4 class="anchored" data-anchor-id="run-8">Run</h4>
<p>Run Delineate for run 5 exactly as in runs 1-4. Start the Delineate environment in Anaconda. Open a terminal session. Navigate to the directory with the <code>ZosteropsPICL_tree.nex</code> and <code>ZosteropsDELINEATE5_assignments.tsv</code> using the <code>cd</code> command. Execute the following command to run Delineate specifying the run parameters and the tree and assignment files.</p>
<pre><code>delineate-estimate partitions -t ZosteropsPICL_tree.nex -c ZosteropsDELINEATE5_assignments.tsv -P 0.95 -u</code></pre>
</section>
<section id="results-6" class="level4">
<h4 class="anchored" data-anchor-id="results-6">Results</h4>
<p>These 3 files will be created upon completion of run 5.</p>
<ul>
<li><p><code>ZosteropsDELINEATE5_assignments.delimitation-results.json</code></p></li>
<li><p><code>ZosteropsDELINEATE5_assignments.delimitation-results.trees</code></p></li>
<li><p><code>ZosteropsDELINEATE5_assignments.delimitation-results.trunc.trees</code></p></li>
</ul>
<p>The speciation completion rate for run 5 is 0.7930 and the lineage birth rate is 1.3202. There are a total of 2,168,963 partitions in run 5 of which 9 are within the confidence interval. Below is the best tree with a constrained probability of 0.4185 and an unconstrained probability of 0.1247.</p>
<p><img src="images/ZosteropsDELINEATE5_tree1.png" class="img-fluid"></p>
<p>The species limits for the top 5 delimitation schemes were recorded manually from the <code>.json</code> file and entered in a spreadsheet. Below are the summarized results for the best 5 out of 2,168,963 partitions together with a total constrained cumulative probability of 0.8473 and an unconstrained cumulative probability of 0.2524.</p>
<div class="cell" data-label="Species delimitation from the 5 best supoprted partitions in run 5.">
<div class="sourceCode cell-code" id="cb99"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb99-1"><a href="#cb99-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Read the tab-delimited file without headers</span></span>
<span id="cb99-2"><a href="#cb99-2" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"Data/Delineate/ZosteropsDELINEATE5_summary.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb99-3"><a href="#cb99-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb99-4"><a href="#cb99-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert the data frame to a gt table</span></span>
<span id="cb99-5"><a href="#cb99-5" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE5_summary <span class="ot">&lt;-</span> data <span class="sc">%&gt;%</span></span>
<span id="cb99-6"><a href="#cb99-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">gt</span>() <span class="sc">%&gt;%</span></span>
<span id="cb99-7"><a href="#cb99-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_header</span>(</span>
<span id="cb99-8"><a href="#cb99-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Zosterops DELINEATE run5 Summary"</span></span>
<span id="cb99-9"><a href="#cb99-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb99-10"><a href="#cb99-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_label</span>(</span>
<span id="cb99-11"><a href="#cb99-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">.list =</span> <span class="cn">NULL</span>  <span class="co"># Set column labels to NULL to remove them</span></span>
<span id="cb99-12"><a href="#cb99-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb99-13"><a href="#cb99-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">fmt_number</span>(</span>
<span id="cb99-14"><a href="#cb99-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">columns =</span> <span class="fu">vars</span>(<span class="fu">where</span>(is.numeric)), <span class="co"># Format numeric columns</span></span>
<span id="cb99-15"><a href="#cb99-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">decimals =</span> <span class="dv">0</span></span>
<span id="cb99-16"><a href="#cb99-16" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb99-17"><a href="#cb99-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cols_width</span>(</span>
<span id="cb99-18"><a href="#cb99-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">everything</span>() <span class="sc">~</span> <span class="fu">px</span>(<span class="dv">130</span>)  <span class="co"># Set all columns to the same width (130 pixels)</span></span>
<span id="cb99-19"><a href="#cb99-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb99-20"><a href="#cb99-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tab_options</span>(</span>
<span id="cb99-21"><a href="#cb99-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.width =</span> <span class="fu">pct</span>(<span class="dv">100</span>), <span class="co"># Set table width</span></span>
<span id="cb99-22"><a href="#cb99-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.font.size =</span> <span class="dv">12</span>,   <span class="co"># Set font size</span></span>
<span id="cb99-23"><a href="#cb99-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">table.align =</span> <span class="st">"left"</span>     <span class="co"># Align text to the left</span></span>
<span id="cb99-24"><a href="#cb99-24" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Since gt v0.3.0, `columns = vars(...)` has been deprecated.
• Please use `columns = c(...)` instead.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb101"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb101-1"><a href="#cb101-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Print and save the table</span></span>
<span id="cb101-2"><a href="#cb101-2" aria-hidden="true" tabindex="-1"></a>ZosteropsDELINEATE5_summary</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="rmvrcvlrsu" style="padding-left:0px;padding-right:0px;padding-top:10px;padding-bottom:10px;overflow-x:auto;overflow-y:auto;width:auto;height:auto;">
<style>#rmvrcvlrsu table {
  font-family: system-ui, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

#rmvrcvlrsu thead, #rmvrcvlrsu tbody, #rmvrcvlrsu tfoot, #rmvrcvlrsu tr, #rmvrcvlrsu td, #rmvrcvlrsu th {
  border-style: none;
}

#rmvrcvlrsu p {
  margin: 0;
  padding: 0;
}

#rmvrcvlrsu .gt_table {
  display: table;
  border-collapse: collapse;
  line-height: normal;
  margin-left: 0;
  margin-right: auto;
  color: #333333;
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  background-color: #FFFFFF;
  width: 100%;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #A8A8A8;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #A8A8A8;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
}

#rmvrcvlrsu .gt_caption {
  padding-top: 4px;
  padding-bottom: 4px;
}

#rmvrcvlrsu .gt_title {
  color: #333333;
  font-size: 125%;
  font-weight: initial;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-color: #FFFFFF;
  border-bottom-width: 0;
}

#rmvrcvlrsu .gt_subtitle {
  color: #333333;
  font-size: 85%;
  font-weight: initial;
  padding-top: 3px;
  padding-bottom: 5px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-color: #FFFFFF;
  border-top-width: 0;
}

#rmvrcvlrsu .gt_heading {
  background-color: #FFFFFF;
  text-align: center;
  border-bottom-color: #FFFFFF;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#rmvrcvlrsu .gt_bottom_border {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rmvrcvlrsu .gt_col_headings {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
}

#rmvrcvlrsu .gt_col_heading {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 6px;
  padding-left: 5px;
  padding-right: 5px;
  overflow-x: hidden;
}

#rmvrcvlrsu .gt_column_spanner_outer {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: normal;
  text-transform: inherit;
  padding-top: 0;
  padding-bottom: 0;
  padding-left: 4px;
  padding-right: 4px;
}

#rmvrcvlrsu .gt_column_spanner_outer:first-child {
  padding-left: 0;
}

#rmvrcvlrsu .gt_column_spanner_outer:last-child {
  padding-right: 0;
}

#rmvrcvlrsu .gt_column_spanner {
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: bottom;
  padding-top: 5px;
  padding-bottom: 5px;
  overflow-x: hidden;
  display: inline-block;
  width: 100%;
}

#rmvrcvlrsu .gt_spanner_row {
  border-bottom-style: hidden;
}

#rmvrcvlrsu .gt_group_heading {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  text-align: left;
}

#rmvrcvlrsu .gt_empty_group_heading {
  padding: 0.5px;
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  vertical-align: middle;
}

#rmvrcvlrsu .gt_from_md > :first-child {
  margin-top: 0;
}

#rmvrcvlrsu .gt_from_md > :last-child {
  margin-bottom: 0;
}

#rmvrcvlrsu .gt_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  margin: 10px;
  border-top-style: solid;
  border-top-width: 1px;
  border-top-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 1px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 1px;
  border-right-color: #D3D3D3;
  vertical-align: middle;
  overflow-x: hidden;
}

#rmvrcvlrsu .gt_stub {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
}

#rmvrcvlrsu .gt_stub_row_group {
  color: #333333;
  background-color: #FFFFFF;
  font-size: 100%;
  font-weight: initial;
  text-transform: inherit;
  border-right-style: solid;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
  padding-left: 5px;
  padding-right: 5px;
  vertical-align: top;
}

#rmvrcvlrsu .gt_row_group_first td {
  border-top-width: 2px;
}

#rmvrcvlrsu .gt_row_group_first th {
  border-top-width: 2px;
}

#rmvrcvlrsu .gt_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#rmvrcvlrsu .gt_first_summary_row {
  border-top-style: solid;
  border-top-color: #D3D3D3;
}

#rmvrcvlrsu .gt_first_summary_row.thick {
  border-top-width: 2px;
}

#rmvrcvlrsu .gt_last_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rmvrcvlrsu .gt_grand_summary_row {
  color: #333333;
  background-color: #FFFFFF;
  text-transform: inherit;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
}

#rmvrcvlrsu .gt_first_grand_summary_row {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-top-style: double;
  border-top-width: 6px;
  border-top-color: #D3D3D3;
}

#rmvrcvlrsu .gt_last_grand_summary_row_top {
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 5px;
  padding-right: 5px;
  border-bottom-style: double;
  border-bottom-width: 6px;
  border-bottom-color: #D3D3D3;
}

#rmvrcvlrsu .gt_striped {
  background-color: rgba(128, 128, 128, 0.05);
}

#rmvrcvlrsu .gt_table_body {
  border-top-style: solid;
  border-top-width: 2px;
  border-top-color: #D3D3D3;
  border-bottom-style: solid;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
}

#rmvrcvlrsu .gt_footnotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#rmvrcvlrsu .gt_footnote {
  margin: 0px;
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#rmvrcvlrsu .gt_sourcenotes {
  color: #333333;
  background-color: #FFFFFF;
  border-bottom-style: none;
  border-bottom-width: 2px;
  border-bottom-color: #D3D3D3;
  border-left-style: none;
  border-left-width: 2px;
  border-left-color: #D3D3D3;
  border-right-style: none;
  border-right-width: 2px;
  border-right-color: #D3D3D3;
}

#rmvrcvlrsu .gt_sourcenote {
  font-size: 90%;
  padding-top: 4px;
  padding-bottom: 4px;
  padding-left: 5px;
  padding-right: 5px;
}

#rmvrcvlrsu .gt_left {
  text-align: left;
}

#rmvrcvlrsu .gt_center {
  text-align: center;
}

#rmvrcvlrsu .gt_right {
  text-align: right;
  font-variant-numeric: tabular-nums;
}

#rmvrcvlrsu .gt_font_normal {
  font-weight: normal;
}

#rmvrcvlrsu .gt_font_bold {
  font-weight: bold;
}

#rmvrcvlrsu .gt_font_italic {
  font-style: italic;
}

#rmvrcvlrsu .gt_super {
  font-size: 65%;
}

#rmvrcvlrsu .gt_footnote_marks {
  font-size: 75%;
  vertical-align: 0.4em;
  position: initial;
}

#rmvrcvlrsu .gt_asterisk {
  font-size: 100%;
  vertical-align: 0;
}

#rmvrcvlrsu .gt_indent_1 {
  text-indent: 5px;
}

#rmvrcvlrsu .gt_indent_2 {
  text-indent: 10px;
}

#rmvrcvlrsu .gt_indent_3 {
  text-indent: 15px;
}

#rmvrcvlrsu .gt_indent_4 {
  text-indent: 20px;
}

#rmvrcvlrsu .gt_indent_5 {
  text-indent: 25px;
}

#rmvrcvlrsu .katex-display {
  display: inline-flex !important;
  margin-bottom: 0.75em !important;
}

#rmvrcvlrsu div.Reactable > div.rt-table > div.rt-thead > div.rt-tr.rt-tr-group-header > div.rt-th-group:after {
  height: 0px !important;
}
</style>

<table class="gt_table caption-top table table-sm table-striped small" data-quarto-postprocess="true" data-quarto-disable-processing="false" data-quarto-bootstrap="false">
<thead>
<tr class="gt_heading header">
<th colspan="8" class="gt_heading gt_title gt_font_normal gt_bottom_border">Zosterops DELINEATE run5 Summary</th>
</tr>
<tr class="gt_col_headings even">
<th id="Tree.taxa" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Tree.taxa</th>
<th id="Putative.species" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">Putative.species</th>
<th id="Constraint" class="gt_col_heading gt_columns_bottom_border gt_right" data-quarto-table-cell-role="th" scope="col">Constraint</th>
<th id="X" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X</th>
<th id="X.1" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.1</th>
<th id="X.2" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.2</th>
<th id="X.3" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.3</th>
<th id="X.4" class="gt_col_heading gt_columns_bottom_border gt_left" data-quarto-table-cell-role="th" scope="col">X.4</th>
</tr>
</thead>
<tbody class="gt_table_body">
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">japonicus</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">alani2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">alani3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. japonicus alani</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">meyeni3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. meyeni</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.4">Z. japonicus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus6</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">montanus7</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">montanus8</td>
<td class="gt_row gt_left" headers="Putative.species">Z. montanus</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">simplex2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">simplex3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. simplex</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. simplex</td>
<td class="gt_row gt_left" headers="X.1">Z. simplex</td>
<td class="gt_row gt_left" headers="X.2">Z. simplex</td>
<td class="gt_row gt_left" headers="X.3">Z. simplex</td>
<td class="gt_row gt_left" headers="X.4">Z. simplex</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">erythro1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">erythro2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. erythropleurus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.1">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.2">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.3">Z. erythropleurus</td>
<td class="gt_row gt_left" headers="X.4">Z. erythropleurus</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">everetti2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">everetti3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. everetti</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. everetti</td>
<td class="gt_row gt_left" headers="X.1">Z. everetti</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Z. everetti</td>
<td class="gt_row gt_left" headers="X.4">Z. everetti</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum1</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. everetti</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 1</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum2</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum3</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum4</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.1">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.2">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.3">Z. nigrorum</td>
<td class="gt_row gt_left" headers="X.4">Z. nigrorum</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa">nigrorum5</td>
<td class="gt_row gt_left" headers="Putative.species">Z. nigrorum</td>
<td class="gt_row gt_right" headers="Constraint">0</td>
<td class="gt_row gt_left" headers="X">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.1">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.2">Z. japonicus</td>
<td class="gt_row gt_left" headers="X.3">Delimited species 2</td>
<td class="gt_row gt_left" headers="X.4">Delimited species 1</td>
</tr>
<tr class="odd">
<td class="gt_row gt_left" headers="Tree.taxa">palpebro</td>
<td class="gt_row gt_left" headers="Putative.species">Z. palpebrosus</td>
<td class="gt_row gt_right" headers="Constraint">1</td>
<td class="gt_row gt_left" headers="X">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.1">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.2">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.3">Z. palpebrosus</td>
<td class="gt_row gt_left" headers="X.4">Z. palpebrosus</td>
</tr>
<tr class="even">
<td class="gt_row gt_left" headers="Tree.taxa"></td>
<td class="gt_row gt_left" headers="Putative.species"></td>
<td class="gt_row gt_right" headers="Constraint">NA</td>
<td class="gt_row gt_left" headers="X">0.1247</td>
<td class="gt_row gt_left" headers="X.1">0.0823</td>
<td class="gt_row gt_left" headers="X.2">0.0192</td>
<td class="gt_row gt_left" headers="X.3">0.0158</td>
<td class="gt_row gt_left" headers="X.4">0.0104</td>
</tr>
</tbody>
</table>

</div>
</div>
<div class="sourceCode cell-code" id="cb102"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb102-1"><a href="#cb102-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the gt table as a PNG file</span></span>
<span id="cb102-2"><a href="#cb102-2" aria-hidden="true" tabindex="-1"></a><span class="fu">gtsave</span>(ZosteropsDELINEATE5_summary, <span class="st">"ZosteropsDELINEATE5_summary.png"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
</section>
<section id="speciation-rate" class="level2">
<h2 class="anchored" data-anchor-id="speciation-rate">Speciation rate</h2>
<p>Prune the original tree from PICL so that only the lineages best delimited as species in Delineate remain. The delimitation for this analysis will be based on the best result for the least constrained run of Delineate (see Run 1). Also remove <em>Z. palpebrosus</em> as an outgroup so that any estimates of speciation rates will be confined to the ingroup of Asiatic Zosterops. Plot the trees in mutation units (MU, substitutions per site per generation) and coalescent units (2N<sub>e</sub>). These trees will be used to estimate the speciation rate using a birth-death model in <code>phytools</code>. See here <a href="http://www.phytools.org/Cordoba2017/ex/12/BD-models.html" class="uri">http://www.phytools.org/Cordoba2017/ex/12/BD-models.html</a>.</p>
<div class="cell" data-label="Estimating the speciation rate based on DELINEATE species tree">
<div class="sourceCode cell-code" id="cb103"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb103-1"><a href="#cb103-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Get the PICL tree with mutation units (MU) and define the collapses based on best DELINEATE result</span></span>
<span id="cb103-2"><a href="#cb103-2" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> Zosterops2_outtree[[<span class="dv">1</span>]]</span>
<span id="cb103-3"><a href="#cb103-3" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeMU_collapse1 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"nigrorum2"</span>, <span class="st">"nigrorum3"</span>)</span>
<span id="cb103-4"><a href="#cb103-4" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeMU_collapse2 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"everetti1"</span>, <span class="st">"everetti2"</span>)</span>
<span id="cb103-5"><a href="#cb103-5" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeMU_collapse3 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"simplex1"</span>, <span class="st">"simplex2"</span>)</span>
<span id="cb103-6"><a href="#cb103-6" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeMU_collapse4 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"montanus1"</span>, <span class="st">"montanus2"</span>, <span class="st">"montanus3"</span>, <span class="st">"montanus4"</span>, <span class="st">"montanus5"</span>, <span class="st">"montanus6"</span>, <span class="st">"montanus7"</span>, <span class="st">"montanus8"</span>, </span>
<span id="cb103-7"><a href="#cb103-7" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"alani1"</span>, <span class="st">"alani2"</span>, <span class="st">"alani3"</span>, </span>
<span id="cb103-8"><a href="#cb103-8" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"meyeni1"</span>, <span class="st">"meyeni2"</span>, <span class="st">"meyeni3"</span>, </span>
<span id="cb103-9"><a href="#cb103-9" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"nigrorum5"</span>)</span>
<span id="cb103-10"><a href="#cb103-10" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeMU_collapse5 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"palpebro"</span>)</span>
<span id="cb103-11"><a href="#cb103-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-12"><a href="#cb103-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of "nigrorum1" to the newly delimited Z. nigrorum PANAY</span></span>
<span id="cb103-13"><a href="#cb103-13" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum1"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. nigrorum nigrorum"</span></span>
<span id="cb103-14"><a href="#cb103-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-15"><a href="#cb103-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop the specified tips</span></span>
<span id="cb103-16"><a href="#cb103-16" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, ZSpeciesTreeMU_collapse1)</span>
<span id="cb103-17"><a href="#cb103-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-18"><a href="#cb103-18" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of "nigrorum4" to the Z. nigrorum NORTH to indicate the remaining northern populations of Z. nirgorum</span></span>
<span id="cb103-19"><a href="#cb103-19" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum4"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. nigrorum NORTH"</span></span>
<span id="cb103-20"><a href="#cb103-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-21"><a href="#cb103-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Change the name of everetti3</span></span>
<span id="cb103-22"><a href="#cb103-22" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"everetti3"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. everetti"</span></span>
<span id="cb103-23"><a href="#cb103-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-24"><a href="#cb103-24" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop remaining everetti</span></span>
<span id="cb103-25"><a href="#cb103-25" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, ZSpeciesTreeMU_collapse2)</span>
<span id="cb103-26"><a href="#cb103-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-27"><a href="#cb103-27" aria-hidden="true" tabindex="-1"></a><span class="co"># Change name of erythro2</span></span>
<span id="cb103-28"><a href="#cb103-28" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"erythro2"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. erythropleurus"</span></span>
<span id="cb103-29"><a href="#cb103-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-30"><a href="#cb103-30" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop erythro1</span></span>
<span id="cb103-31"><a href="#cb103-31" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, <span class="st">"erythro1"</span>)</span>
<span id="cb103-32"><a href="#cb103-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-33"><a href="#cb103-33" aria-hidden="true" tabindex="-1"></a><span class="co"># Change name of simplex3</span></span>
<span id="cb103-34"><a href="#cb103-34" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"simplex3"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. simplex"</span></span>
<span id="cb103-35"><a href="#cb103-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-36"><a href="#cb103-36" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop remaining simplex</span></span>
<span id="cb103-37"><a href="#cb103-37" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, ZSpeciesTreeMU_collapse3)</span>
<span id="cb103-38"><a href="#cb103-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-39"><a href="#cb103-39" aria-hidden="true" tabindex="-1"></a><span class="co"># Change name of japonicus</span></span>
<span id="cb103-40"><a href="#cb103-40" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeMU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"japonicus"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. japonicus/meyeni/montanus"</span></span>
<span id="cb103-41"><a href="#cb103-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-42"><a href="#cb103-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop remaining JMM taxa</span></span>
<span id="cb103-43"><a href="#cb103-43" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, ZSpeciesTreeMU_collapse4)</span>
<span id="cb103-44"><a href="#cb103-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-45"><a href="#cb103-45" aria-hidden="true" tabindex="-1"></a><span class="co"># Drop the outgroup Z. palpebrosus</span></span>
<span id="cb103-46"><a href="#cb103-46" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeMU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeMU, ZSpeciesTreeMU_collapse5)</span>
<span id="cb103-47"><a href="#cb103-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-48"><a href="#cb103-48" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb103-49"><a href="#cb103-49" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(ZosteropsSpeciesTreeMU, <span class="at">file =</span> <span class="st">"Data/Delineate/ZosteropsSpeciesTreeMU"</span>)</span>
<span id="cb103-50"><a href="#cb103-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-51"><a href="#cb103-51" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the mutation units [1]. </span></span>
<span id="cb103-52"><a href="#cb103-52" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(ZosteropsSpeciesTreeMU, <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.001</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb103-53"><a href="#cb103-53" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(ZosteropsSpeciesTreeMU<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="fl">0.1</span>), <span class="at">labels =</span> <span class="cn">TRUE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb103-54"><a href="#cb103-54" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"Substitutions per site per generation"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(ZosteropsSpeciesTreeMU<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">2.5</span>, <span class="at">cex =</span> <span class="fl">0.8</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb104"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb104-1"><a href="#cb104-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Get the PICL tree with coalescent units (CU) and define the collapses based on best DELINEATE result</span></span>
<span id="cb104-2"><a href="#cb104-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Note for this tree the tips were previously changed from the tip names (e.g. montanus 1, 2, 3, etc.) to the full species names</span></span>
<span id="cb104-3"><a href="#cb104-3" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> Zosterops2_outtree[[<span class="dv">2</span>]]</span>
<span id="cb104-4"><a href="#cb104-4" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeCU_collapse1 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. nigrorum innominatus"</span>, <span class="st">"Z. nigrorum meyleri"</span>)</span>
<span id="cb104-5"><a href="#cb104-5" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeCU_collapse2 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. everetti boholensis SAMAR"</span>, <span class="st">"Z. everetti basilanicus MINDANAO"</span>)</span>
<span id="cb104-6"><a href="#cb104-6" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeCU_collapse3 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. simplex CHINA, TAIWAN, VIETNAM, USA"</span>, <span class="st">"Z. simplex GUANGXI"</span>)</span>
<span id="cb104-7"><a href="#cb104-7" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeCU_collapse4 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. montanus montanus JAVA"</span>, <span class="st">"Z. montanus montanus SULAWESI"</span>, </span>
<span id="cb104-8"><a href="#cb104-8" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Z. montanus diuatae"</span>, <span class="st">"Z. montanus vulcani"</span>, <span class="st">"Z. montanus pectoralis"</span>, </span>
<span id="cb104-9"><a href="#cb104-9" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Z. montanus PANAY"</span>, <span class="st">"Z. montanus parkesi"</span>, <span class="st">"Z. montanus NORTH"</span>, </span>
<span id="cb104-10"><a href="#cb104-10" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"Z. japonicus alani ANEJIMA"</span>, <span class="st">"Z. japonicus alani HAHAJIMA"</span>, <span class="st">"Z. japonicus alani IWOJIMA"</span>, </span>
<span id="cb104-11"><a href="#cb104-11" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"Z. meyeni BATAN"</span>, <span class="st">"Z. meyeni SABTANG"</span>, <span class="st">"Z. meyeni LANYU"</span>, </span>
<span id="cb104-12"><a href="#cb104-12" aria-hidden="true" tabindex="-1"></a>                            <span class="st">"Z. nigrorum catarmanensis"</span>)</span>
<span id="cb104-13"><a href="#cb104-13" aria-hidden="true" tabindex="-1"></a>ZSpeciesTreeCU_collapse5 <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Z. palpebrosus"</span>)</span>
<span id="cb104-14"><a href="#cb104-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-15"><a href="#cb104-15" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"nigrorum1"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. nigrorum nigrorum"</span></span>
<span id="cb104-16"><a href="#cb104-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-17"><a href="#cb104-17" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, ZSpeciesTreeCU_collapse1)</span>
<span id="cb104-18"><a href="#cb104-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-19"><a href="#cb104-19" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"Z. nigrorum luzonicus"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. nigrorum NORTH"</span></span>
<span id="cb104-20"><a href="#cb104-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-21"><a href="#cb104-21" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"Z. everetti basilanicus CAMIGUINSUR"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. everetti"</span></span>
<span id="cb104-22"><a href="#cb104-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-23"><a href="#cb104-23" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, ZSpeciesTreeCU_collapse2)</span>
<span id="cb104-24"><a href="#cb104-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-25"><a href="#cb104-25" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"Z. erythropleurus RUSSIA"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. erythropleurus"</span></span>
<span id="cb104-26"><a href="#cb104-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-27"><a href="#cb104-27" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, <span class="st">"Z. erythropleurus VIETNAM"</span>)</span>
<span id="cb104-28"><a href="#cb104-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-29"><a href="#cb104-29" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"Z. simplex SINGAPORE"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. simplex"</span></span>
<span id="cb104-30"><a href="#cb104-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-31"><a href="#cb104-31" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, ZSpeciesTreeCU_collapse3)</span>
<span id="cb104-32"><a href="#cb104-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-33"><a href="#cb104-33" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label[ZosteropsSpeciesTreeCU<span class="sc">$</span>tip.label <span class="sc">==</span> <span class="st">"Z. japonicus CONTINENTAL"</span>] <span class="ot">&lt;-</span> <span class="st">"Z. japonicus/meyeni/montanus"</span></span>
<span id="cb104-34"><a href="#cb104-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-35"><a href="#cb104-35" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, ZSpeciesTreeCU_collapse4)</span>
<span id="cb104-36"><a href="#cb104-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-37"><a href="#cb104-37" aria-hidden="true" tabindex="-1"></a>ZosteropsSpeciesTreeCU <span class="ot">&lt;-</span> <span class="fu">drop.tip</span>(ZosteropsSpeciesTreeCU, ZSpeciesTreeCU_collapse5)</span>
<span id="cb104-38"><a href="#cb104-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-39"><a href="#cb104-39" aria-hidden="true" tabindex="-1"></a><span class="co"># Save the resulting tree to a file in Newick format</span></span>
<span id="cb104-40"><a href="#cb104-40" aria-hidden="true" tabindex="-1"></a><span class="fu">write.tree</span>(ZosteropsSpeciesTreeCU, <span class="at">file =</span> <span class="st">"Data/Delineate/ZosteropsSpeciesTreeCU"</span>)</span>
<span id="cb104-41"><a href="#cb104-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-42"><a href="#cb104-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot the PICL outtree with the mutation units [1]. </span></span>
<span id="cb104-43"><a href="#cb104-43" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(ZosteropsSpeciesTreeCU, <span class="at">edge.width =</span> <span class="dv">3</span>, <span class="at">label.offset =</span> <span class="fl">0.001</span>, <span class="at">cex =</span> <span class="fl">1.2</span>)</span>
<span id="cb104-44"><a href="#cb104-44" aria-hidden="true" tabindex="-1"></a><span class="fu">axis</span>(<span class="dv">1</span>, <span class="at">at =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fu">max</span>(ZosteropsSpeciesTreeCU<span class="sc">$</span>edge.length), <span class="at">by =</span> <span class="dv">2</span>), <span class="at">labels =</span> <span class="cn">TRUE</span>, <span class="at">tck =</span> <span class="sc">-</span><span class="fl">0.02</span>)</span>
<span id="cb104-45"><a href="#cb104-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mtext</span>(<span class="st">"Coalescent units (2Ne generations)"</span>, <span class="at">side =</span> <span class="dv">1</span>, <span class="at">line =</span> <span class="dv">2</span>, <span class="at">at =</span> <span class="fu">max</span>(ZosteropsSpeciesTreeCU<span class="sc">$</span>edge.length) <span class="sc">/</span> <span class="fl">2.5</span>, <span class="at">cex =</span> <span class="fl">0.7</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Zosterops_SVDQ_files/figure-html/unnamed-chunk-20-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p>Estimate the speciation rate for each tree using the <code>phytools</code> package and the birth-death model (<code>birthdeath</code>).</p>
<div class="cell" data-label="Speciation rates">
<div class="sourceCode cell-code" id="cb105"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb105-1"><a href="#cb105-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(phytools)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: maps</code></pre>
</div>
<div class="sourceCode cell-code" id="cb107"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb107-1"><a href="#cb107-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Estimate speciation rates for mutation unit (MU) tree and coalescent tree (CU) </span></span>
<span id="cb107-2"><a href="#cb107-2" aria-hidden="true" tabindex="-1"></a>fit.bdMU <span class="ot">&lt;-</span> <span class="fu">birthdeath</span>(ZosteropsSpeciesTreeMU)</span>
<span id="cb107-3"><a href="#cb107-3" aria-hidden="true" tabindex="-1"></a>fit.bdMU</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Estimation of Speciation and Extinction Rates
            with Birth-Death Models

     Phylogenetic tree: ZosteropsSpeciesTreeMU 
        Number of tips: 6 
              Deviance: -17.95391 
        Log-likelihood: 8.976954 
   Parameter estimates:
      d / b = 0   StdErr = 0 
      b - d = 7.747477   StdErr = 2.73942 
   (b: speciation rate, d: extinction rate)
   Profile likelihood 95% confidence intervals:
      d / b: [-3.361896, 0.7364605]
      b - d: [2.405837, 17.99545]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb109"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb109-1"><a href="#cb109-1" aria-hidden="true" tabindex="-1"></a><span class="fu">bd</span>(fit.bdMU)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>       b        d 
7.747477 0.000000 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb111"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb111-1"><a href="#cb111-1" aria-hidden="true" tabindex="-1"></a>fit.bdCU <span class="ot">&lt;-</span> <span class="fu">birthdeath</span>(ZosteropsSpeciesTreeCU)</span>
<span id="cb111-2"><a href="#cb111-2" aria-hidden="true" tabindex="-1"></a>fit.bdCU</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Estimation of Speciation and Extinction Rates
            with Birth-Death Models

     Phylogenetic tree: ZosteropsSpeciesTreeCU 
        Number of tips: 6 
              Deviance: 8.861443 
        Log-likelihood: -4.430721 
   Parameter estimates:
      d / b = 0   StdErr = 0 
      b - d = 0.2712932   StdErr = 0.09595199 
   (b: speciation rate, d: extinction rate)
   Profile likelihood 95% confidence intervals:
      d / b: [-3.361876, 0.7364616]
      b - d: [0.08424542, 0.6301483]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb113"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb113-1"><a href="#cb113-1" aria-hidden="true" tabindex="-1"></a><span class="fu">bd</span>(fit.bdCU)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>        b         d 
0.2712932 0.0000000 </code></pre>
</div>
</div>



</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-webshot2" class="csl-entry" role="listitem">
Chang, Winston. 2025. <span>“Webshot2: Take Screenshots of Web Pages.”</span> <a href="https://CRAN.R-project.org/package=webshot2">https://CRAN.R-project.org/package=webshot2</a>.
</div>
<div id="ref-chifman2014" class="csl-entry" role="listitem">
Chifman, Julia, and Laura Kubatko. 2014. <span>“Quartet Inference from SNP Data Under the Coalescent Model.”</span> <em>Bioinformatics</em> 30 (23): 3317–24. <a href="https://doi.org/10.1093/bioinformatics/btu530">https://doi.org/10.1093/bioinformatics/btu530</a>.
</div>
<div id="ref-deraad2024" class="csl-entry" role="listitem">
DeRaad, Devon A., Marlon E. Cobos, Natalie R. Hofmeister, Lucas H. DeCicco, Madhvi X. Venkatraman, Isao Nishiumi, Bailey McKay, et al. 2024. <span>“On the Brink of Explosion? Identifying the Source and Potential Spread of Introduced Zosterops White-Eyes in North America.”</span> <em>Biological Invasions</em> 26 (5): 1615–39. <a href="https://doi.org/10.1007/s10530-024-03268-8">https://doi.org/10.1007/s10530-024-03268-8</a>.
</div>
<div id="ref-gt" class="csl-entry" role="listitem">
Iannone, Richard, Joe Cheng, Barret Schloerke, Ellis Hughes, Alexandra Lauer, JooYoung Seo, Ken Brevoort, and Olivier Roy. 2024. <span>“Gt: Easily Create Presentation-Ready Display Tables.”</span> <a href="https://CRAN.R-project.org/package=gt">https://CRAN.R-project.org/package=gt</a>.
</div>
<div id="ref-kubatko2025" class="csl-entry" role="listitem">
Kubatko, Laura S, Sungsik Kong, Emerson Webb, and Zixuan Chen. 2025. <span>“The Promise of Composite Likelihood for Species-Level Phylogenomic Inference.”</span> <em>Evolutionary Journal of the Linnean Society</em>, May. <a href="https://doi.org/10.1093/evolinnean/kzaf008">https://doi.org/10.1093/evolinnean/kzaf008</a>.
</div>
<div id="ref-ortiz2019" class="csl-entry" role="listitem">
Ortiz, Edgardo M. 2019. <em>Vcf2phylip V2.0: Convert a VCF Matrix into Several Matrix Formats for Phylogenetic Analysis.</em> Zenodo. <a href="https://doi.org/10.5281/ZENODO.2540861">https://doi.org/10.5281/ZENODO.2540861</a>.
</div>
<div id="ref-ape" class="csl-entry" role="listitem">
Paradis, Emmanuel, and Klaus Schliep. 2019. <span>“Ape 5.0: An Environment for Modern Phylogenetics and Evolutionary Analyses in <span></span>r<span></span>”</span> 35: 526–28. <a href="https://doi.org/10.1093/bioinformatics/bty633">https://doi.org/10.1093/bioinformatics/bty633</a>.
</div>
<div id="ref-sukumaran2021" class="csl-entry" role="listitem">
Sukumaran, Jeet, Mark T. Holder, and L. Lacey Knowles. 2021. <span>“Incorporating the Speciation Process into Species Delimitation.”</span> Edited by Timothy G. Barraclough. <em>PLOS Computational Biology</em> 17 (5): e1008924. <a href="https://doi.org/10.1371/journal.pcbi.1008924">https://doi.org/10.1371/journal.pcbi.1008924</a>.
</div>
<div id="ref-swofford2023a" class="csl-entry" role="listitem">
Swofford, David L., and Laura S. Kubatko. 2023. <span>“CHAPTER 4 Species Tree Estimation Using Site Pattern Frequencies.”</span> In, 68–88. Princeton University Press. <a href="https://doi.org/10.1515/9780691245157-008">https://doi.org/10.1515/9780691245157-008</a>.
</div>
<div id="ref-dplyr" class="csl-entry" role="listitem">
Wickham, Hadley, Romain François, Lionel Henry, Kirill Müller, and Davis Vaughan. 2023. <span>“Dplyr: A Grammar of Data Manipulation.”</span> <a href="https://CRAN.R-project.org/package=dplyr">https://CRAN.R-project.org/package=dplyr</a>.
</div>
<div id="ref-tidyr" class="csl-entry" role="listitem">
Wickham, Hadley, Davis Vaughan, and Maximilian Girlich. 2024. <span>“Tidyr: Tidy Messy Data.”</span> <a href="https://CRAN.R-project.org/package=tidyr">https://CRAN.R-project.org/package=tidyr</a>.
</div>
<div id="ref-wilgenbusch2003" class="csl-entry" role="listitem">
Wilgenbusch, James C., and David Swofford. 2003. <span>“Inferring Evolutionary Trees with PAUP*.”</span> <em>Current Protocols in Bioinformatics</em> 00 (1). <a href="https://doi.org/10.1002/0471250953.bi0604s00">https://doi.org/10.1002/0471250953.bi0604s00</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>